{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "29fc4d35-f702-4fa4-beaf-4a3ba2352714",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "# Importing the necesary functions\n",
    "from pi_pubs import Researcher\n",
    "from chat import load_papers_pdf,text_splitter,vectorstore\n",
    "import os"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "83bacc40-fca8-4bdb-a5a3-8cdda25649e6",
   "metadata": {},
   "source": [
    "The steps to create a RAG according to Langchain (https://python.langchain.com/docs/use_cases/question_answering/) templates are the following:\n",
    "$$\\begin(itemize)\n",
    "\\item Get the PDFs for PIs (for our particular case)\n",
    "\\item Loading\n",
    "\\item Splitting\n",
    "\\item Storage\n",
    "\\item Retrieval\n",
    "\\item Generation\n",
    "\\item Chat-History\n",
    "\\end{itemize}$$\n",
    "\n",
    "The following code is intentend to create a first pipeline for a RAG application with the papers from different PIs. Dr. Adrian Jinich is used a test case"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "2456fb86-14f7-4cb8-82a1-b654eda1e978",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2023-11-01 17:38:08,473 - INFO - pubmed2pdf.cli - Trying to fetch pmid 37405387\n",
      "2023-11-01 17:38:10,620 - INFO - pubmed2pdf.cli - Trying to fetch pmid 36945430\n",
      "2023-11-01 17:38:10,620 - INFO - pubmed2pdf.cli - Trying to fetch pmid 36802628\n",
      "2023-11-01 17:38:11,083 - INFO - pubmed2pdf.cli - Trying to fetch pmid 36247217\n",
      "2023-11-01 17:38:13,345 - INFO - pubmed2pdf.cli - Trying to fetch pmid 35337802\n",
      "2023-11-01 17:38:14,277 - INFO - pubmed2pdf.cli - Trying to fetch pmid 34782606\n",
      "2023-11-01 17:38:14,277 - INFO - pubmed2pdf.cli - Trying to fetch pmid 33376214\n",
      "2023-11-01 17:38:14,806 - INFO - pubmed2pdf.cli - Trying to fetch pmid 33068537\n",
      "2023-11-01 17:38:16,117 - INFO - pubmed2pdf.cli - Trying to fetch pmid 31733221\n",
      "2023-11-01 17:38:17,955 - INFO - pubmed2pdf.cli - Trying to fetch pmid 31404220\n",
      "2023-11-01 17:38:18,792 - INFO - pubmed2pdf.cli - Trying to fetch pmid 30898926\n",
      "2023-11-01 17:38:19,412 - INFO - pubmed2pdf.cli - Trying to fetch pmid 30356318\n",
      "2023-11-01 17:38:19,412 - INFO - pubmed2pdf.cli - Trying to fetch pmid 28328209\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Done downloading. All downloaded can be found in /home/richiam/pubmed2pdf\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2023-11-01 17:38:19,964 - INFO - pubmed2pdf.cli - Trying to fetch pmid 25387603\n",
      "2023-11-01 17:38:19,964 - INFO - pubmed2pdf.cli - Trying to fetch pmid 24586198\n",
      "2023-11-01 17:38:19,964 - INFO - pubmed2pdf.cli - Trying to fetch pmid 22383887\n",
      "2023-11-01 17:38:19,964 - INFO - pubmed2pdf.cli - Trying to fetch pmid 19851443\n"
     ]
    }
   ],
   "source": [
    "# Get the PDFs for PIs (for our particular case)\n",
    "PI_name = \"Adrian Jinich\"\n",
    "test = Researcher(PI_name)\n",
    "test._pmdid_collection(\"richiam@uchicago.edu\") # An email is required to access PubmedId\n",
    "test._download_papers() \n",
    "# test.main() The last two lines can be replaced with this command\n",
    "# Note: Currently downloading all papers in predetermied folder ()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "e80fbf05-b6ca-4490-884d-199c1eddfe3e",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "page_content='High-Resolution Profiling of Stationary-Phase Survival\\nReveals Yeast Longevity Factors and Their Genetic\\nInteractions\\nErika Garay, Sergio E. Campos, Jorge Gonza ´lez de la Cruz, Ana P. Gaspar, Adrian Jinich,\\nAlexander DeLuna *\\nLaboratorio Nacional de Geno ´mica para la Biodiversidad, Centro de Investigacio ´n y de Estudios Avanzados del IPN, Irapuato, Guanajuato, Mexico\\nAbstract\\nLifespan is influenced by a large number of conserved proteins and gene-regulatory pathways. Here, we introduce a\\nstrategy for systematically finding such longevity factors in Saccharomyces cerevisiae and scoring the genetic interactions\\n(epistasis) among these factors. Specifically, we developed an automated competition-based assay for chronologicallifespan, defined as stationary-phase survival of yeast populations, and used it to phenotype over 5,600 single- or double-\\ngene knockouts at unprecedented quantitative resolution. We found that 14% of the viable yeast mutant strains were\\naffected in their stationary-phase survival; the extent of true-positive chronological lifespan factors was estimated byaccounting for the effects of culture aeration and adaptive regrowth. We show that lifespan extension by dietary restriction\\ndepends on the Swr1 histone-exchange complex and that a functional link between autophagy and the lipid-homeostasis\\nfactor Arv1 has an impact on cellular lifespan. Importantly, we describe the first genetic interaction network based on agingphenotypes, which successfully recapitulated the core-autophagy machinery and confirmed a role of the human tumorsuppressor PTEN homologue in yeast lifespan and phosphatidylinositol phosphate metabolism. Our quantitative analysis of\\nlongevity factors and their genetic interactions provides insights into the gene-network interactions of aging cells.\\nCitation: Garay E, Campos SE, Gonza ´lez de la Cruz J, Gaspar AP, Jinich A, et al. (2014) High-Resolution Profiling of Stationary-Phase Survival Reveals Yeast\\nLongevity Factors and Their Genetic Interactions. PLoS Genet 10(2): e1004168. doi:10.1371/journal.pgen.1004168\\nEditor: Stuart K. Kim, Stanford University Medical Center, United States of America\\nReceived May 2, 2013; Accepted December 23, 2013; Published February 27, 2014\\nCopyright: /C2232014 Garay et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits\\nunrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.\\nFunding: This work was funded by the Consejo Nacional de Ciencia y Tecnologı ´a (CONACYT CB-164889), the University of California Institute for Mexico and the\\nUnited States (UC-MEXUS CN-11-559), and a pilot grant from the Northwestern University Physical Sciences-Oncology Center. The funders had no role i n study\\ndesign, data collection and analysis, decision to publish, or preparation of the manuscript.\\nCompeting Interests: The authors have declared that no competing interests exist.\\n* E-mail: adeluna@langebio.cinvestav.mx\\nIntroduction\\nLifespan is a complex trait with clear genetic underpinnings\\n[1]. Mutations in the insulin/IGF-1 receptor gene daf-2 increase\\nthe lifespan of Caenorhabditis elegans [2] and Drosophila melanogaster\\n[3], while mice lacking the insulin receptor in adipose tissue livelonger as well [4]. Studies on the budding yeast Saccharomyces\\ncerevisiae have also been pivotal to discover conserved longevity\\nmechanisms [5–7]. Aging of dividing cells has been modeled inyeast by its replicative lifespan, defined as the number of daughtercells produced before death [8]. In addition, chronologicallifespan (CLS), the duration of yeast survival in stationary phase,provides a model of aging of non-dividing quiescent cells [9,10].Evidence to date indicates that both forms of yeast aging are\\nmodulated by the Tor/Sch9 [11,12] and Ras/cAMP/PKA [13]\\npathways. Likewise, the downstream activity of proteins withroles in autophagy [14], redox balance [15,16], and mitochon-drial function [17,18] influence lifespan from yeast to mammals.Moreover, dietary restriction delays both replicative and chro-nological aging in yeast; this non-genetic intervention has beenlinked to the down-regulation of Tor and Sch9 in response tonutrients [12,19].\\nCurrent phenotyping strategies allow the characterization of\\nyeast CLS in a high-throughput manner. Survival as a function oftime in stationary phase has been estimated by monitoring the\\noutgrowth of either individual-strain cultures in parallel [19,20] or\\nlarge pools of gene-deletion strains coupled to microarray [21,22]\\nor deep-sequencing analysis [23]. Such large-scale approaches\\nhave recently opened the door to the first genome-wide screens forlifespan factors [19,21–23]. With these catalogs in hand, the next\\ngreat challenge is to describe how different genetic factors are\\nintegrated to determine lifespan in yeast.\\nQuantitative genetic interaction (GI) analysis has great potential\\nto shed light on the genetic architecture of the lifespan phenotype\\nand the mechanisms that underlie the aging process. By depicting\\nthe ways in which individual mutations affect each other’s\\nphenotypic consequences, GIs inform whether genes act in\\ncommon or independent pathways [24]. A number of break-through studies based on the large-scale GI analysis of cell-\\nproliferation phenotypes (mainly colony growth) have provided\\nimportant insights into the genetic organization of the living cell\\n[25–31]. Multi-conditional and multi-phenotype GI maps have\\nfurther helped to elucidate cellular pathways and responses to\\nstimuli [32,33] and, notably, GI networks retain conserved\\nfeatures and overall levels of pathway crosstalk across large\\nevolutionary distances [34]. Yet, systematic double-mutant anal-\\nyses of lifespan phenotypes and their ensuing GI networks are still\\nmissing.\\nPLOS Genetics | www.plosgenetics.org 1 February 2014 | Volume 10 | Issue 2 | e1004168' metadata={'source': '/home/richiam/pubmed2pdf/24586198.pdf', 'page': 0}\n"
     ]
    }
   ],
   "source": [
    "# Loading\n",
    "\n",
    "docs = load_papers_pdf(\"/home/richiam/pubmed2pdf\")\n",
    "# Showing some of the documents\n",
    "print(docs[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "e70f691d-d747-48b1-b12f-b0a889532e17",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "page_content='High-Resolution Profiling of Stationary-Phase Survival\\nReveals Yeast Longevity Factors and Their Genetic\\nInteractions\\nErika Garay, Sergio E. Campos, Jorge Gonza ´lez de la Cruz, Ana P. Gaspar, Adrian Jinich,\\nAlexander DeLuna *\\nLaboratorio Nacional de Geno ´mica para la Biodiversidad, Centro de Investigacio ´n y de Estudios Avanzados del IPN, Irapuato, Guanajuato, Mexico\\nAbstract\\nLifespan is influenced by a large number of conserved proteins and gene-regulatory pathways. Here, we introduce a\\nstrategy for systematically finding such longevity factors in Saccharomyces cerevisiae and scoring the genetic interactions\\n(epistasis) among these factors. Specifically, we developed an automated competition-based assay for chronologicallifespan, defined as stationary-phase survival of yeast populations, and used it to phenotype over 5,600 single- or double-\\ngene knockouts at unprecedented quantitative resolution. We found that 14% of the viable yeast mutant strains were' metadata={'source': '/home/richiam/pubmed2pdf/24586198.pdf', 'page': 0}\n",
      "There are a total of 489 chunks\n"
     ]
    }
   ],
   "source": [
    "# Splitting\n",
    "splits = text_splitter(docs, 1200, 150)\n",
    "print(splits[0])\n",
    "# A place of improvemnet would be to be able to \"cut the head of the paper, ie the importanto information to use a metadata \n",
    "# and to be consistent with all papers\"\n",
    "print(f\"There are a total of {len(splits)} chunks\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "acfcf93d-c310-468f-b31b-13b2c69ac8c5",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "# Storage\n",
    "\n",
    "# First we need to define the embedding function and define OpenAI variables\n",
    "os.environ['OPENAI_API_KEY'] = \"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\"\n",
    "from langchain.embeddings.openai import OpenAIEmbeddings\n",
    "# The directory where the vector database is located, it also needs to be emptied\n",
    "!rm -rf ./docs/chroma\n",
    "persist_directory = 'docs/chroma/'\n",
    "vectordb = vectorstore(documents=splits,embedding_function=OpenAIEmbeddings(),directory=persist_directory)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "a8e210cb-f1ad-48db-ae3d-adcdc41a34e7",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The Vectordatabese has a total collection of 489 for a total of 489 chunks\n"
     ]
    }
   ],
   "source": [
    "print(f\"The Vectordatabese has a total collection of {vectordb._collection.count()} for a total of {len(splits)} chunks\" )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "413226a2-5d5b-4c61-869c-eb51bc254cde",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "# Retrieval\n",
    "\n",
    "retriever = vectordb.as_retriever()\n",
    "\n",
    "# Prompt to get information about \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "fd83c666-4ef0-453e-ba33-aa526e3fa64d",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "AIMessage(content='The main topics that Adrian Jinich has been working on include thermodynamic constraints in carbon fixation pathways, acetogenesis, redox biochemistry, biochemical reaction networks, and computational chemistry.')"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Generate\n",
    "\n",
    "# Creating a Prompot for the LLM:\n",
    "\n",
    "from langchain.prompts import PromptTemplate\n",
    "\n",
    "template = \"\"\"You are an assistant for question-answering tasks. \n",
    "The information provided are published scientific articles that were published by Adrian Jinich.\n",
    "Use this information to answer questions about the work, life, and publications of Adrian Jinich.\n",
    "If you do not know the answer, just say that you do not know the answer. Use ten sentences maximum and use technical terms in the answers\n",
    "Context:{context}\n",
    "Question: {question}\n",
    "\"\"\"\n",
    "\n",
    "rag_prompt_custom = PromptTemplate.from_template(template)\n",
    "\n",
    "from langchain.chat_models import ChatOpenAI\n",
    "from langchain.schema.runnable import RunnablePassthrough\n",
    "\n",
    "llm = ChatOpenAI(model_name = \"gpt-3.5-turbo\", temperature=0)\n",
    "\n",
    "rag_chain = ({\"context\": retriever, \"question\": RunnablePassthrough()} | rag_prompt_custom | llm)\n",
    "\n",
    "rag_chain.invoke(\"What are the main topics that Adrian Jinich has been working on ?\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "7aebfb56-2dbb-424f-a3ad-674720700d08",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "AIMessage(content=\"I'm sorry, but the provided information does not specify the total number of papers that Adrian Jinich has published.\")"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rag_chain.invoke(\"Can you give a number of the total papers that Adrian Jinich has published?\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "1aeb7f64-87a6-45d0-977b-9ee5aa6b47d1",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "AIMessage(content='Adrian Jinich has published scientific articles about Escherichia coli. One of the articles, titled \"Invariant Distribution of Promoter Activities in Escherichia coli,\" was co-authored by Alon Zaslaver, Shai Kaplan, Anat Bren, Avi Mayo, Erez Dekel, Uri Alon, and Shalev Itzkovitz. This article discusses the need for cells to allocate their limited resources to express a wide range of genes. The authors employed a robotic assay using a comprehensive reporter strain library for E. coli to measure promoter activity on a genomic scale. This allowed them to track promoter activity continuously as cells changed their growth rate from exponential to stationary phase. The article provides insights into how E. coli partitions its transcriptional resources between different promoters. The study was conducted at the Weizmann Institute of Science in Rehovot, Israel, where Adrian Jinich was affiliated with the Department of Computer Science and Applied Mathematics and the Department of Biological Chemistry.')"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rag_chain.invoke(\"What has Adrian Jinich published about Escherichia coli?, explain in detail\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "3eda548b-653e-4e16-911a-c9a49b4c1fc4",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "AIMessage(content='Adrian Jinich is a researcher who has contributed to the field of redox biochemistry. He has published several scientific articles on the topic, including \"Thermodynamic constraints shape the structure of carbon fixation pathways\" and \"Does acetogenesis really require especially low reduction potential?\". In these articles, Jinich explores the thermodynamic principles underlying metabolic reactions and the requirements for acetogenesis.\\n\\nJinich\\'s work involves the use of quantum chemical approaches to estimate the thermodynamics of metabolic reactions. He collaborates with other researchers, such as Haniu Ren, Sung-Jin Kim, Elad Noor, and Benjamin Sanchez-Lengeling, to carry out the research and analyze the data. The research is supervised by Ala´n Aspuru-Guzik and Arren Bar-Even.\\n\\nJinich\\'s research has practical implications for understanding and manipulating metabolic pathways. By studying the thermodynamic constraints of carbon fixation and acetogenesis, Jinich aims to uncover fundamental principles that govern these processes. This knowledge can potentially be applied to optimize metabolic engineering strategies and improve the efficiency of biofuel production.\\n\\nOverall, Adrian Jinich\\'s work focuses on the thermodynamics of metabolic reactions, particularly in the context of carbon fixation and acetogenesis. His research utilizes quantum chemical approaches and collaboration with other scientists to gain insights into the fundamental principles underlying these processes.')"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rag_chain.invoke(\"Can you summarize the work of Adrian Jinich ?, explain in detail\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "001f6de8-2779-41cb-9ad4-87cffbb09b8c",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "AIMessage(content='Adrian Jinich is a researcher who has published scientific articles on topics related to thermodynamics and biochemistry.')"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rag_chain.invoke(\"Who is Adrian Jinich?\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "3dc907b5-f4d3-4a7a-b690-39c5d8f353c1",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "AIMessage(content='The last paper that Adrian Jinich published is titled \"Quantum Chemical Approach to Estimating the Thermodynamics of Metabolic Reactions\" and was published in the journal Scientific Reports in 2014.')"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rag_chain.invoke(\"What was the last paper that Adrian Jinich published?\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "6ab0a763-58be-40bb-8fd0-80f295f92117",
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "# Adding Conversation history:\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "409ee93a-6660-45cf-9ba2-a3d91eacaaf1",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
