<!DOCTYPE html>
<html lang="en" class="no-js no-jr">
  <head>
    <meta charset="UTF-8">
    <title>Genome-wide screen identifies host loci that modulate Mycobacterium tuberculosis fitness in immunodivergent mice</title>
    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="939B5D01FBE02635000044A5C7F596D4.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="1.5.0.post1+55721b3" />

<meta name="ncbi_domain" content="gthree" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_report" content="reader" />

<meta name="ncbi_pcid" content="/articles/PMC10468300/?report=reader" />


    
    <meta name="apple-mobile-web-app-capable" content="no" /><meta name="viewport" content="initial-scale=1,minimum-scale=1,user-scalable=yes,width=device-width" /><meta name="format-detection" content="telephone=no" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10468300/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="G3: Genes|Genomes|Genetics" /><meta name="citation_title" content="Genome-wide screen identifies host loci that modulate Mycobacterium tuberculosis fitness in immunodivergent mice" /><meta name="citation_author" content="Rachel K Meade" /><meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology, Duke University, Durham, NC 27710, USA" /><meta name="citation_author_institution" content="University Program in Genetics and Genomics, Duke University, Durham, NC 27710, USA" /><meta name="citation_author" content="Jarukit E Long" /><meta name="citation_author_institution" content="Department of Microbiology and Physiological Systems, UMass Chan Medical School, Worcester, MA 01655, USA" /><meta name="citation_author_institution" content="Research Animal Diagnostic Services, Charles River Laboratories, Wilmington, MA 01887, USA" /><meta name="citation_author" content="Adrian Jinich" /><meta name="citation_author_institution" content="Division of Infectious Diseases, Weill Cornell Medical College, New York, NY 10021, USA" /><meta name="citation_author" content="Kyu Y Rhee" /><meta name="citation_author_institution" content="Division of Infectious Diseases, Weill Cornell Medical College, New York, NY 10021, USA" /><meta name="citation_author" content="David G Ashbrook" /><meta name="citation_author_institution" content="Department of Genetics, Genomics and Informatics, University of Tennessee Health Science Center, Memphis, TN 38163, USA" /><meta name="citation_author" content="Robert W Williams" /><meta name="citation_author_institution" content="Department of Genetics, Genomics and Informatics, University of Tennessee Health Science Center, Memphis, TN 38163, USA" /><meta name="citation_author" content="Christopher M Sassetti" /><meta name="citation_author_institution" content="Department of Microbiology and Physiological Systems, UMass Chan Medical School, Worcester, MA 01655, USA" /><meta name="citation_author" content="Clare M Smith" /><meta name="citation_author_institution" content="Department of Molecular Genetics and Microbiology, Duke University, Durham, NC 27710, USA" /><meta name="citation_author_institution" content="University Program in Genetics and Genomics, Duke University, Durham, NC 27710, USA" /><meta name="citation_publication_date" content="2023/09" /><meta name="citation_issue" content="9" /><meta name="citation_volume" content="13" /><meta name="citation_doi" content="10.1093/g3journal/jkad147" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC10468300/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC10468300/" /><meta name="citation_pmid" content="37405387" /><meta name="DC.Title" content="Genome-wide screen identifies host loci that modulate Mycobacterium tuberculosis fitness in immunodivergent mice" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Oxford University Press" /><meta name="DC.Contributor" content="Rachel K Meade" /><meta name="DC.Contributor" content="Jarukit E Long" /><meta name="DC.Contributor" content="Adrian Jinich" /><meta name="DC.Contributor" content="Kyu Y Rhee" /><meta name="DC.Contributor" content="David G Ashbrook" /><meta name="DC.Contributor" content="Robert W Williams" /><meta name="DC.Contributor" content="Christopher M Sassetti" /><meta name="DC.Contributor" content="Clare M Smith" /><meta name="DC.Date" content="2023 Sep" /><meta name="DC.Identifier" content="10.1093/g3journal/jkad147" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Genome-wide screen identifies host loci that modulate Mycobacterium tuberculosis fitness in immunodivergent mice" /><meta property="og:type" content="article" /><meta property="og:description" content="Genetic differences among mammalian hosts and among strains of Mycobacterium tuberculosis (Mtb) are well-established determinants of tuberculosis (TB) patient outcomes. The advent of recombinant inbred mouse panels and next-generation transposon mutagenesis ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10468300/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'" /><meta name="pmcaccid" content="PMC10468300" />
    <style>.hidden {display: none!important}</style><link href="//fonts.googleapis.com/css?family=Archivo+Narrow:400,700,400italic,700italic&amp;subset=latin" rel="stylesheet" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/jatsreader/ptpmc_3.22/css/libs.min.css" /><link rel="stylesheet" href="/corehtml/pmc/jatsreader/ptpmc_3.22/css/jr.min.css" /><script type="text/javascript" src="/corehtml/pmc/jatsreader/ptpmc_3.22/js/jr.boots.min.js"> </script>
  </head>
  <body>

  
      <div id="jr" data-jr-path="/corehtml/pmc/jatsreader/ptpmc_3.22/"><div class="jr-unsupported"><div id="modal"><p><span class="attn inline-block"></span></p><p>You possibly have javascript turned off.</p><p>The functionality is limited to basic scrolling.</p><p>You may switch to <a href="/pmc/articles/PMC10468300/?report=classic" class="">Article in classic view</a>.</p></div></div><div id="jr-ui" class="hidden"><nav id="jr-head"><div class="flexh tb"><div id="jr-tb1"><a id="jr-links-sw" class="btn wsprkl hidden" title="Links"><img src="/corehtml/pmc/jatsreader/ptpmc_3.22/img/pmc.logo.svg" alt="pmc logo" class="svg" /></a><a id="jr-alt-sw" class="btn wsprkl hidden" title="Alternative formats of the Article">Alt</a><a id="jr-pdf-sw" href="/pmc/articles/PMC10468300/pdf/jkad147.pdf" class="btn wsprkl">PDF</a></div><div class="jr-rhead f1"><div class="t">Genome-wide screen identifies host loci that modulate <em>Mycobacterium tuberculosis</em> fitness in immunodivergent mice</div><div class="j"><a href="#" class="ctxp">G3 (Bethesda). 2023 Sep; 13(9): jkad147. </a></div></div><div id="jr-tb2"><a id="jr-help-sw" class="btn wsprkl hidden" title="Settings, typography and Help with NLM PubReader"><img src="/corehtml/pmc/jatsreader/ptpmc_3.22/img/pmc.gear.svg" alt="settings &amp; help" class="svg" /></a><a id="jr-fip-sw" class="btn wsprkl hidden" title="Find"><img src="/corehtml/pmc/jatsreader/ptpmc_3.22/img/pmc.mg.svg" alt="Search on this page" class="svg" /></a><a id="jr-cmap-sw" class="btn wsprkl hidden" title="Article Navigation"><img src="/corehtml/pmc/jatsreader/ptpmc_3.22/img/pmc.cmap.svg" alt="Table of Content" class="svg" /></a></div></div></nav><nav id="jr-dash" class="noselect"><div id="jr-pi" class="hidden"><a id="jr-pi-prev" class="hidden" title="Previous page"><svg xmlns="http://www.w3.org/2000/svg" width="24pt" height="24pt" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M75,30 c-80,60 -80,0 0,60 c-30,-60 -30,0 0,-60"></path><text x="20" y="28" textLength="60" style="font-size:25px">Prev</text></svg></a><div class="pginfo">Page <i class="jr-pg-pn">0</i> of <i class="jr-pg-lp">0</i></div><a id="jr-pi-next" class="hidden" title="Next page"><svg xmlns="http://www.w3.org/2000/svg" width="24pt" height="24pt" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M25,30c80,60 80,0 0,60 c30,-60 30,0 0,-60"></path><text x="20" y="28" textLength="60" style="font-size:25px">Next</text></svg></a></div><div id="jr-is-tb"><a id="jr-is-sw" class="btn wsprkl hidden" title="Switch between Figures/Tables strip and Progress bar"><svg xmlns="http://www.w3.org/2000/svg" width="24pt" height="24pt" viewBox="0 0 100 100" preserveAspectRatio="none"><rect x="10" y="40" width="20" height="20"></rect><rect x="40" y="40" width="20" height="20"></rect><rect x="70" y="40" width="20" height="20"></rect></svg></a></div><nav id="jr-istrip" class="istrip hidden"><a id="jr-is-prev" href="#" class="hidden" title="Previous"><svg xmlns="http://www.w3.org/2000/svg" width="24pt" height="24pt" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M80,40 60,65 80,90 70,90 50,65 70,40z M50,40 30,65 50,90 40,90 20,65 40,40z"></path><text x="35" y="25" textLength="60" style="font-size:25px">Prev</text></svg></a><a id="jr-is-next" href="#" class="hidden" title="Next"><svg xmlns="http://www.w3.org/2000/svg" width="24pt" height="24pt" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M20,40 40,65 20,90 30,90 50,65 30,40z M50,40 70,65 50,90 60,90 80,65 60,40z"></path><text x="15" y="25" textLength="60" style="font-size:25px">Next</text></svg></a></nav><nav id="jr-progress"></nav></nav><aside id="jr-links-p" class="hidden flexv"><div class="tb sk-htbar flexh"><div><a class="jr-p-close btn wsprkl">Done</a></div><div class="title-text f1">Links</div></div><div class="cnt lol f1"><a class="navlink" href="/pmc/">PMC Home</a><a href="/pmc/journals/" class="navlink">Journal List</a><a class="navlink" href="/pmc/journals/1684/">G3 (Bethesda)</a><a class="navlink" href="/pmc/issues/444355/">v.13(9); 2023 Sep</a><a class="btn share" target="_blank" rel="noopener noreferrer" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10468300%2F"><img src="/corehtml/pmc/jatsreader/ptpmc_3.22/img/pmc.fb.svg" alt="Facebook's f logo" class="svg" style="float:right;padding:3px" />Share on Facebook</a><a class="btn share" target="_blank" rel="noopener noreferrer" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10468300%2F&amp;text=Genome-wide%20screen%20identifies%20host%20loci%20that%20modulate%20Mycobacterium%20tuberculosis%20fitness%20in%20immunodivergent%20mice%20..."><img src="/corehtml/pmc/jatsreader/ptpmc_3.22/img/pmc.twitter.svg" alt="Twitter t logo" class="svg" style="float:right;padding:3px" />Share on Twitter</a><a class="btn feedback" target="_blank" rel="noopener noreferrer nofollow" href="https://support.nlm.nih.gov/support/create-case/?from=https://www.ncbi.nlm.nih.gov/" ref="linksrc=feedback_btn">Feedback</a></div></aside><aside id="jr-alt-p" class="hidden flexv"><div class="tb sk-htbar flexh"><div><a class="jr-p-close btn wsprkl">Done</a></div><div class="title-text f1">Alternative formats</div></div><div class="cnt lol f1"><a href="/pmc/articles/PMC10468300/?report=classic" class="">Article in classic view</a><a href="/pmc/articles/PMC10468300/pdf/jkad147.pdf" class="int-view">PDF (6.5M)</a><a class="citationexporter ctxp" href="#" data-citationid="PMC10468300" role="button">Cite</a></div></aside><aside id="jr-cmap-p" class="hidden flexv"><div class="tb sk-htbar flexh"><div><a class="jr-p-close btn wsprkl">Done</a></div><div class="title-text f1">Article navigation</div></div><div class="cnt lol f1"></div></aside><aside id="jr-help-p" class="hidden flexv"><div class="tb sk-htbar flexh"><div><a class="jr-p-close btn wsprkl">Done</a></div><div class="title-text f1">Settings &amp; Help</div></div><div class="cnt f1"><div id="jr-typo-p" class="typo"><div><a class="sf btn wsprkl">A-</a><a class="lf btn wsprkl">A+</a></div><div><a class="bcol-auto btn wsprkl"><svg xmlns="http://www.w3.org/2000/svg" width="24pt" height="24pt" viewBox="0 0 200 100" preserveAspectRatio="none"><text x="10" y="70" style="font-size:60px;font-family: Trebuchet MS, ArialMT, Arial, sans-serif" textLength="180">AUTO</text></svg></a><a class="bcol-1 btn wsprkl"><svg xmlns="http://www.w3.org/2000/svg" width="24pt" height="24pt" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M15,25 85,25zM15,40 85,40zM15,55 85,55zM15,70 85,70z"></path></svg></a><a class="bcol-2 btn wsprkl"><svg xmlns="http://www.w3.org/2000/svg" width="24pt" height="24pt" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M5,25 45,25z M55,25 95,25zM5,40 45,40z M55,40 95,40zM5,55 45,55z M55,55 95,55zM5,70 45,70z M55,70 95,70z"></path></svg></a></div></div><div class="lol"><a id="jr-helpobj-sw" data-path="/corehtml/pmc/jatsreader/ptpmc_3.22/" data-href="/corehtml/pmc/jatsreader/ptpmc_3.22/img/help.xml" href="">Help with PubReader</a><a href="/pmc/articles/PMC10468300/?report=classic" class="">Switch to classic view</a><a id="jr-about-sw" data-path="/corehtml/pmc/jatsreader/ptpmc_3.22/" data-href="/corehtml/pmc/jatsreader/ptpmc_3.22/img/about.xml" href="">About PubReader</a></div></div></aside><aside id="jr-objectbox" class="thidden hidden"><div class="jr-objectbox-close wsprkl">✘</div><div class="jr-objectbox-inner cnt"><div class="jr-objectbox-drawer"></div></div></aside><nav id="jr-pm-left" class="hidden"><svg xmlns="http://www.w3.org/2000/svg" width="24pt" height="24pt" viewBox="0 0 100 1000" preserveAspectRatio="none"><text x="850" y="-20" transform="rotate(90)" textLength="150" font-size="23">Previous Page</text></svg></nav><nav id="jr-pm-right" class="hidden"><svg xmlns="http://www.w3.org/2000/svg" width="24pt" height="24pt" viewBox="0 0 100 1000" preserveAspectRatio="none"><text x="850" y="-70" transform="rotate(90)" textLength="150" font-size="23">Next Page</text></svg></nav><nav id="jr-fip" class="hidden"><nav id="jr-fip-term-p"><input type="search" placeholder="search this page" id="jr-fip-term" autocorrect="off" autocomplete="off" /><a id="jr-fip-mg" class="wsprkl btn" title="Find"><img src="/corehtml/pmc/jatsreader/ptpmc_3.22/img/pmc.mg.svg" alt="Find" class="svg" /></a><a id="jr-fip-done" class="wsprkl btn" title="Dismiss find">✘</a></nav><nav id="jr-fip-info-p"><a id="jr-fip-prev" class="wsprkl btn" title="Jump to previuos match">◀</a><button id="jr-fip-matches">no matches yet</button><a id="jr-fip-next" class="wsprkl btn" title="Jump to next match">▶</a></nav></nav></div><div id="jr-epub-interstitial" class="hidden"><h2 role="dialog">Making articles easier to read in PMC</h2><div class="contentpane"><p>
                We are experimenting with display styles that make it easier to read articles in PMC.
                The ePub format uses eBook readers, which have several "ease of reading" features
                already built in.
              </p><p>
                The ePub format is best viewed in the iBooks reader. You may notice problems with
                the display of certain parts of an article in other eReaders.
              </p><p>
                Generating an ePub file may take a long time, please be patient.
              </p></div><div class="buttonpane lol"><a id="cancelEpub" style="float: right">Cancel</a><a id="downloadEpub" style="float: left">Download article</a></div></div><a id="jr-welcome-trigger" class="hidden">t</a><div id="jr-welcome" class="hidden"><h2 role="dialog">Welcome to PubReader!</h2><div class="contentpane"><p>
                Click on <a class="btn"><img src="/corehtml/pmc/jatsreader/ptpmc_3.22/img/pmc.gear.svg" alt="settings &amp; help" class="svg" /></a> above to:
              </p><ul><li>Get help with PubReader, or </li><li>Switch to the classic article view.</li></ul></div><div class="buttonpane lol"><a id="okayWelcome" class="btn centerbutton">Okay</a></div></div><div id="jr-content"><article data-type="main" class="slang-all"><div class="jr-gthree jrb"><div class="t"><a href="//doi.org/10.1093/g3journal/jkad147">G3: Genes|Genomes|Genetics</a></div><div class="p">Oxford University Press</div></div><header class="fm-sec"><h1 class="content-title">Genome-wide screen identifies host loci that modulate <em>Mycobacterium tuberculosis</em> fitness in immunodivergent mice</h1><p class="contribs">Rachel K Meade, Jarukit E Long,  [...],  and Clare M Smith</p><p class="fm-aai"><a href="#__ffn_sectitle">Additional article information</a></p></header><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>jkad147_Supplementary_Data.</strong></div><div><a href="/pmc/articles/PMC10468300/bin/jkad147_supplementary_data.zip" data-ga-action="click_feat_suppl">jkad147_supplementary_data.zip</a><span style="color:gray"> (8.0M)</span></div><div class="small guid">GUID: BFB46883-3BFA-41B7-986E-3D8773C94746</div></div></dd></dl><dl data-length="645" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first">All relevant data to support the findings of this study are located within the paper and supplementary files. Genome sequence data is available in the NCBI Gene Expression Omnibus (GEO), accession number <a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE234093" ref="/geo/query/acc.cgi?acc=GSE234093"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE234093","term_id":"234093"}}</span>GSE234093</a>. All QTL mapping files are publicly available at: <a href="https://github.com/SmithLabDuke/Meade_et_al_BXD_TnSeq" data-ga-action="click_feat_suppl" target="_blank">https://github.com/SmithLabDuke/Meade_et_al_BXD_TnSeq</a>. Tn mutant fitness profiles within each BXD genotype and parental genotypes are publicly available on MtbTnDB (<a href="https://www.mtbtndb.app/" data-ga-action="click_feat_suppl" target="_blank">https://www.mtbtndb.app/</a>). BXD phenotypes are available on GeneNetwork (<a href="https://www.genenetwork.org" data-ga-action="click_feat_suppl" target="_blank">https://www.genenetwork.org</a>).</p><p class="p p-last">
<a href="#sup1" rid="sup1" class=" supplementary-material">Supplemental material</a> available at G3 online.</p></dd></dl></div><div id="abstract-a.m.b.r" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.m.b.rtitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">Genetic differences among mammalian hosts and among strains of <em>Mycobacterium tuberculosis</em> (<em>Mtb</em>) are well-established determinants of tuberculosis (TB) patient outcomes. The advent of recombinant inbred mouse panels and next-generation transposon mutagenesis and sequencing approaches has enabled dissection of complex host&#x02013;pathogen interactions. To identify host and pathogen genetic determinants of <em>Mtb</em> pathogenesis, we infected members of the highly diverse BXD family of strains with a comprehensive library of <em>Mtb</em> transposon mutants (TnSeq). Members of the BXD family segregate for <em>Mtb</em>-resistant C57BL/6J (B6 or <em>B</em>) and <em>Mtb</em>-susceptible DBA/2J (D2 or <em>D</em>) haplotypes. The survival of each bacterial mutant was quantified within each BXD host, and we identified those bacterial genes that were differentially required for <em>Mtb</em> fitness across BXD genotypes. Mutants that varied in survival among the host family of strains were leveraged as reporters of &#x0201c;endophenotypes,&#x0201d; each bacterial fitness profile directly probing specific components of the infection microenvironment. We conducted quantitative trait loci (QTL) mapping of these bacterial fitness endophenotypes and identified 140 <em>host&#x02013;<span class="underline">p</span>athogen</em> QTL (<em>hp</em>QTL). We located a QTL hotspot on chromosome 6 (75.97&#x02013;88.58 Mb) associated with the genetic requirement of multiple <em>Mtb</em> genes: <em>Rv0127</em> (<em>mak</em>), <em>Rv0359</em> (<em>rip2</em>), <em>Rv0955</em> (<em>perM</em>), and <em>Rv3849</em> (<em>espR</em>). Together, this screen reinforces the utility of bacterial mutant libraries as precise reporters of the host immunological microenvironment during infection and highlights specific host&#x02013;pathogen genetic interactions for further investigation. To enable downstream follow-up for both bacterial and mammalian genetic research communities, all bacterial fitness profiles have been deposited into GeneNetwork.org and added into the comprehensive collection of TnSeq libraries in MtbTnDB.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">host&#x02013;pathogen interactions, tuberculosis, mycobacteria, systems genetics, genomics, genetic diversity, natural variation, bacterial genetics, TnSeq, mouse models, BXD, QTL mapping, complex traits</span></div></div><div id="jkad147-s0" class="tsec sec"><h2 class="head no_bottom_margin" id="jkad147-s0title">Introduction</h2><p class="p p-first">Every pathogenic infection is part of an evolutionary battle between host and invader. In the case of <em>Mycobacterium tuberculosis</em> (<em>Mtb</em>), the causative agent of tuberculosis (TB), evidence tracing back at least 9,000 years tells the tale of a host&#x02013;pathogen arms race (<a href="#jkad147-B26" rid="jkad147-B26" class=" bibr popnode">Hershkovitz <em>et al</em>. 2008</a>), making <em>Mtb</em> one of the most enduring adversaries of the human species. Given the prevailing nature of this challenge, it is no surprise that weaknesses in vital host defenses provide greater opportunity for <em>Mtb</em> infection, allow <em>Mtb</em> to manipulate the nature and magnitude of the host immune response, and worsen patient prognoses (<a href="#jkad147-B16" rid="jkad147-B16" class=" bibr popnode">Cooper <em>et al</em>. 1993</a>; <a href="#jkad147-B21" rid="jkad147-B21" class=" bibr popnode">Flynn <em>et al</em>. 1993</a>; <a href="#jkad147-B17" rid="jkad147-B17" class=" bibr popnode">Cooper <em>et al</em>. 1997</a>; <a href="#jkad147-B10" rid="jkad147-B10" class=" bibr popnode">Caruso <em>et al</em>. 1999</a>; <a href="#jkad147-B32" rid="jkad147-B32" class=" bibr popnode">Kramnik <em>et al</em>. 2000</a>). In addition, the advent of drug-resistant and multidrug-resistant <em>Mtb</em> strains highlights the imminent danger posed by our microscopic rivals and underscores the importance of bacterial variation as a critical factor in the pathogenesis and continued spread of <em>Mtb</em> (<a href="#jkad147-B11" rid="jkad147-B11" class=" bibr popnode">Caws <em>et al</em>. 2008</a>; <a href="#jkad147-B25" rid="jkad147-B25" class=" bibr popnode">Hern&#x000e1;ndez-Pando <em>et al</em>. 2012</a>; <a href="#jkad147-B23" rid="jkad147-B23" class=" bibr popnode">Gopal <em>et al</em>. 2014</a>). Meanwhile, genetic diversity of both hosts and <em>Mtb</em> strains that interact during infection can give rise to a spectrum of disease outcomes (<a href="#jkad147-B56" rid="jkad147-B56" class=" bibr popnode">Smith and Sassetti 2018</a>), enhancing the complexity of TB identification and treatment. To combat a pathogen responsible for nearly 10.6 million infections and 1.6 million deaths annually (<a href="#jkad147-B65" rid="jkad147-B65" class=" bibr popnode">WHO 2022</a>), it is therefore vital to dissect the host&#x02013;pathogen interface.</p><p>Mammalian models of TB have been used successfully to identify and mechanistically interrogate TB susceptibility loci identified from human cohorts. For mouse strains that are phenotypically divergent, quantitative trait locus (QTL) mapping studies within genetically tractable and reproducible hosts have enabled the identification of genomic loci that contribute to the magnitude of clinically relevant phenotypes (<a href="#jkad147-B33" rid="jkad147-B33" class=" bibr popnode">Lavebratt <em>et al</em>. 1999</a>; <a href="#jkad147-B32" rid="jkad147-B32" class=" bibr popnode">Kramnik <em>et al</em>. 2000</a>; <a href="#jkad147-B39" rid="jkad147-B39" class=" bibr popnode">Mitsos <em>et al</em>. 2000</a>; <a href="#jkad147-B40" rid="jkad147-B40" class=" bibr popnode">Mitsos <em>et al</em>. 2003</a>; <a href="#jkad147-B69" rid="jkad147-B69" class=" bibr popnode">Yan <em>et al</em>. 2006</a>). The classic inbred strains C57BL/6J (B6) and DBA/2J (D2) lie on opposite ends of the TB susceptibility spectrum, with B6 surviving nearly a year postinfection and D2 succumbing to cachexia and morbidity within months (<a href="#jkad147-B37" rid="jkad147-B37" class=" bibr popnode">Medina and North 1998</a>). To capitalize on this diverse disease outcome and just over 6 million divergent SNPs (<a href="#jkad147-B61" rid="jkad147-B61" class=" bibr popnode">Wang <em>et al</em>. 2016</a>; <a href="#jkad147-B4" rid="jkad147-B4" class=" bibr popnode">Ashbrook <em>et al</em>. 2021</a>; <a href="#jkad147-B5" rid="jkad147-B5" class=" bibr popnode">Ashbrook <em>et al</em>. 2022</a>; <a href="#jkad147-B49" rid="jkad147-B49" class=" bibr popnode">Sasani <em>et al</em>. 2022</a>), a recombinant inbred panel generated from these 2 founders, known as the BXD, has laid the foundation for understanding host determinants of susceptibility to a variety of diseases through QTL mapping in genetically diverse mice (<a href="#jkad147-B57" rid="jkad147-B57" class=" bibr popnode">Taylor <em>et al</em>. 1973</a>; <a href="#jkad147-B58" rid="jkad147-B58" class=" bibr popnode">Taylor <em>et al</em>. 1999</a>; <a href="#jkad147-B46" rid="jkad147-B46" class=" bibr popnode">Peirce <em>et al</em>. 2004</a>; <a href="#jkad147-B4" rid="jkad147-B4" class=" bibr popnode">Ashbrook <em>et al</em>. 2021</a>).</p><p>BXD and other recombinant inbred panels have been used to explore the intricate dynamics of TB disease, but often the phenotypes leveraged to understand these dynamics in mice, such as survival time postinfection, body weight, or bacterial burden, are too complex and polygenic to reveal novel biology or enable refined mapping. These &#x0201c;macrophenotypes&#x0201d; materialize from many genetic and environmental factors that can be difficult to mechanistically dissect or precisely map. Within each unique host, bacteria face similarly diverse immunological dynamics and spatiotemporal selective pressures. To contribute to the growing body of work on genetic determinants of TB outcome, quantifying precise bacterial &#x0201c;endophenotypes&#x0201d; underlying disease outcomes is the crucial next step toward elaborating the biological intricacies that govern the host-pathogen interface. Recently employed molecular endophenotypes include expression QTL (<a href="#jkad147-B13" rid="jkad147-B13" class=" bibr popnode">Chesler <em>et al</em>. 2005</a>), metabolite QTL (<a href="#jkad147-B68" rid="jkad147-B68" class=" bibr popnode">Wu <em>et al</em>. 2014</a>), protein QTL (<a href="#jkad147-B15" rid="jkad147-B15" class=" bibr popnode">Chick <em>et al</em>. 2016</a>), and chromatin accessibility QTL (<a href="#jkad147-B53" rid="jkad147-B53" class=" bibr popnode">Skelly <em>et al</em>. 2020</a>), but for infectious disease, the most salient endophenotypes lie at the interface of the host and pathogen.</p><p>A classic approach to assess the requirement of a bacterial gene for infection is to infect a standard inbred mouse strain, such as B6, with a single knockout <em>Mtb</em> mutant. Expanding on this design, genome-wide mutagenesis enables the generation of single knockout mutants across the <em>Mtb</em> genome, thereby allowing the measurement of <em>Mtb</em> gene requirements under a variety of selective pressures. Transposon (Tn) mutant libraries (TnSeq) have been successfully used to report on variable components of the bacterial infection microenvironment under distinct host and antibiotic pressures (<a href="#jkad147-B51" rid="jkad147-B51" class=" bibr popnode">Sassetti and Rubin 2003</a>; <a href="#jkad147-B44" rid="jkad147-B44" class=" bibr popnode">Nambi <em>et al</em>. 2015</a>; <a href="#jkad147-B38" rid="jkad147-B38" class=" bibr popnode">Mishra <em>et al</em>. 2017</a>; <a href="#jkad147-B7" rid="jkad147-B7" class=" bibr popnode">Bellerose <em>et al</em>. 2019</a>). We have recently leveraged TnSeq across genetically diverse Collaborative Cross (CC) mice to map both host- and pathogen-linked loci (<a href="#jkad147-B55" rid="jkad147-B55" class=" bibr popnode">Smith <em>et al</em>. 2022</a>). Deep phenotyping of disease traits in over 50 CC strains enabled mapping of 10 <em>tuberculosis immunophenotype</em> QTL (<em>Tip</em>QTL), derived from bacterial burden and cytokine quantifications in each host. Measurement of TnSeq mutant fitness during in vivo infection yielded 46 <em>host interacting with pathogen</em> QTL (<em>Hip</em>QTL). While 8 founder lines do provide a greater extent of SNP-level diversity in the CC panel than in the BXD panel (<a href="#jkad147-B43" rid="jkad147-B43" class=" bibr popnode">Munger <em>et al</em>. 2014</a>), the additional founders can reduce statistical strength for QTL mapping and complicate determination of allele effects for mapped QTL. The 2-state model of the BXD panel allows relatively straightforward identification of significant QTL and downstream allele effects, potentially shortening the path from screen to biological discovery.</p><p class="p p-last">To further characterize the BXD phenome (<a href="#jkad147-B14" rid="jkad147-B14" class=" bibr popnode">Chesler <em>et al</em>. 2004</a>) and add to the growing repertoire of selective pressures applied to <em>Mtb</em> whole-genome mutant libraries (<a href="#jkad147-B29" rid="jkad147-B29" class=" bibr popnode">Jinich <em>et al</em>. 2021</a>), we now leverage the phenotypic divergence and reproducibility of the BXD family and the molecular phenotyping precision of an <em>Mtb</em> Tn mutant library. We infected 19 BXD strains and both parents with a saturated library of <em>Mtb</em> Tn mutants and conducted dual-genome QTL mapping to identify linkages between bacterial fitness profiles and host genetic factors, revealing 140 host genetic loci significantly associated with specific bacterial mutants. Thus, in passing a bacterial Tn mutant library through selection within the immunologically and phenotypically diverse BXD microenvironments, we have conducted a multidimensional host&#x02013;pathogen screen to identify precise host and bacterial determinants of TB disease. Overall, this study provides a blueprint for the dissection of host&#x02013;pathogen interactions, which can play a pivotal role in functional annotation of understudied host and bacterial genes.</p></div><div id="jkad147-s1" class="tsec sec"><h2 class="head no_bottom_margin" id="jkad147-s1title">Materials and methods</h2><div id="jkad147-s1.1" class="sec sec-first"><h3 id="jkad147-s1.1title">Ethics statement</h3><p class="p p-first-last">All mouse studies were conducted in accordance with the guidelines issued in the Guide for the Care and Use of Laboratory Animals of the National Institutes of Health and the Office of Laboratory Animal Welfare. Animal studies conducted at Duke University Medical School were conducted using protocols approved by the Duke Institutional Animal Care and Use Committee (IACUC) (Animal Welfare Assurance #A221-20-11) in a manner designed to minimize pain and suffering in <em>Mtb</em>-infected animals. Any animal exhibiting signs of severe disease were immediately euthanized in accordance with IACUC-approved endpoints. All mouse studies conducted at the University of Massachusetts Medical School (UMass) were performed using protocols approved by UMass IACUC (Animal Welfare Assurance #A3306-01).</p></div><div id="jkad147-s1.2" class="sec"><h3 id="jkad147-s1.2title">Mice</h3><p class="p p-first-last">Male and female C57BL/6J (#000664) and DBA/2J (#000671) mice were purchased from The Jackson Laboratory. Male mice from 19 BXD strains were imported from the colony of Robert Williams (University of Tennessee Health Science Center, Memphis, TN, USA) in 2013. The 19 BXD strains in this study include BXD9/TyJ, BXD29/Ty, BXD39/TyJ, BXD40/TyJ, BXD48a/RwwJ, BXD51/RwwJ, BXD54/RwwJ, BXD56/RwwJ, BXD60/RwwJ, BXD62/RwwJ, BXD67/RwwJ, BXD69/RwwJ, BXD73/RwwJ, BXD73b/RwwJ, BXD77/RwwJ, BXD79/RwwJ, BXD90/RwwJ, BXD93/RwwJ, and BXD102/RwwJ. All mice were housed in a specific pathogen-free facility within standardized living conditions (12-h light/dark, food and water ad libitum). Mice were matched at 8&#x02013;12 weeks of age at the time of <em>Mtb</em> infection. Male mice were used in the TnSeq screen, and both male and female mice were used in the aerosol studies.</p></div><div id="jkad147-s1.3" class="sec"><h3 id="jkad147-s1.3title">
<em>Mtb</em> strains</h3><p class="p p-first-last">All <em>Mtb</em> strains were cultured in Middlebrook 7H9 medium supplemented with oleic acid&#x02013;albumin&#x02013;dextrose catalase (OADC), 0.2% glycerol, and 0.05% Tween 80 to log-phase with shaking (200 rpm) at 37&#x000b0;C. Hygromycin (50 <em>&#x000b5;</em>g/mL) and kanamycin (20 <em>&#x000b5;</em>g/mL) were added when necessary. Prior to all in vivo infections, cultures were washed, resuspended in phosphate-buffered saline (PBS) containing 0.05% Tween 80 (hereafter PBS-T), and sonicated before diluting to desired concentration.</p></div><div id="jkad147-s1.4" class="sec"><h3 id="jkad147-s1.4title">Mouse infections</h3><p class="p p-first">For aerosol infections, B6 and D2 were infected with &#x0223c;50 colony forming units (CFUs) of <em>Mtb</em> H37Rv YFP via aerosol inhalation (Glas-Col) for 6 and 12 weeks. At each timepoint, male and female mice were euthanized in accordance with approved IACUC protocols, and lung and spleen were harvested into PBS-T and homogenized. CFU was quantified by dilution plating onto Middlebrook 7H10 agar supplemented with OADC, 0.2% glycerol, 50 mg/mL carbenicillin, 10 mg/mL amphotericin B, 25 mg/mL polymyxin B, and 20 mg/mL trimethoprim. One lung lobe per mouse was collected into 10% neutral-buffered formalin for hematoxylin and eosin (H&#x00026;E) staining by the Duke University Research Immunohistology Laboratory.</p><p class="p p-last">For TnSeq experiments, 1 &#x000d7; 10<sup>6</sup> CFU of saturated <em>Himar1</em> Tn mutants (<a href="#jkad147-B50" rid="jkad147-B50" class=" bibr popnode">Sassetti <em>et al</em>. 2003</a>) was delivered via intravenous tail vein injection, an infection route that enables consistent delivery of the complete TnSeq library to each mouse. Mice were infected in a single batch. At 4-week postinfection, mice were euthanized, and spleens and lungs were harvested and then homogenized in a FastPrep-24 (MP Biomedicals). CFU was quantified by dilution plating on 7H10 agar with 20 <em>&#x000b5;</em>g/mL kanamycin. For library recovery, &#x0223c;1 &#x000d7; 10<sup>6</sup> CFU per mouse was plated on 7H10 agar with 20 <em>&#x000b5;</em>g/mL kanamycin. After 3 weeks of growth, colonies were harvested by scraping, and genomic DNA was extracted. The relative abundance of each Tn mutant was estimated as described in <a href="#jkad147-B35" rid="jkad147-B35" class=" bibr popnode">Long <em>et al</em>. (2015)</a>.</p></div><div id="jkad147-s1.5" class="sec"><h3 id="jkad147-s1.5title">Microscopy and damage quantification</h3><p class="p p-first-last">H&#x00026;E-stained lung sections were imaged in bright field at 2&#x000d7; magnification on the Keyence BZ-X800. Images were processed identically within FIJI software (v2.3.0/1.53p) for image clarity. To quantify damage, an artificial neural network&#x02013;based damage identification model was developed within QuPath (v0.3.2) (<a href="#jkad147-B6" rid="jkad147-B6" class=" bibr popnode">Bankhead <em>et al</em>. 2017</a>), as described in <a href="#jkad147-B66" rid="jkad147-B66" class=" bibr popnode">Wilburn <em>et al</em>. (2023)</a>. Eight images (1 image per experimental group), containing a total of 68 unaffected and 65 affected manual annotations, were utilized for the single purpose of training a pixel classifier, and the remaining 3 images per group were quantified using the model. To prevent bias, lung sections presented in this report were quantified as the closest to the mean damage of each experimental group.</p></div><div id="jkad147-s1.6" class="sec"><h3 id="jkad147-s1.6title">TnSeq analysis</h3><p class="p p-first">TnSeq libraries were prepared and Tn insertion counts were estimated as previously described (<a href="#jkad147-B55" rid="jkad147-B55" class=" bibr popnode">Smith <em>et al</em>. 2022</a>). The H37Rv genome annotation used is NCBI Reference Sequence <a id="__tag_1140177341" class="tag_hotlink" href="/nuccore/NC_018143.2" ref="/nuccore/NC_018143.2"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NC_018143.2","term_id":"561108321","term_text":"NC_018143.2"}}</span>NC_018143.2</a>. In total, 40 independent Tn libraries were sequenced after recovery from the 19 BXD genotypes and the 2 parent strains, B6 and D2. Two independent Tn libraries were placed under selection within each host genotype. However, in the cases of 2 BXD genotypes, BXD48a and BXD51, only 1 library could be sufficiently recovered for sequencing. Within TRANSIT (<a href="#jkad147-B19" rid="jkad147-B19" class=" bibr popnode">DeJesus <em>et al</em>. 2015</a>), beta-geometric correction was used to normalize insertion mutant counts across all libraries, and pseudocounts were implemented. Insertion counts were totaled for each <em>Mtb</em> gene, and counts were averaged between individual replicate libraries per mouse. Per gene mean values were compared with the grand mean and then log<sub>2</sub> transformed.</p><p class="p p-last">To control for multiple hypotheses genome wide, 10,000 permutation tests were performed to generate a resampling distribution used to derive the significance of Tn mutant selection between in vitro and in vivo conditions. An <em>Mtb</em> gene was considered &#x0201c;essential&#x0201d; within a host if mutants lacking that gene experienced a log<sub>2</sub> fold change (LFC) value of &#x02212;0.5 or lower postselection in that host. To identify <em>Mtb</em> genes whose fitness was sufficiently variable to enable QTL mapping, the dynamic range in LFC across the parental and BXD strains for any given bacterial gene had to be at least 0.63. This value is the minimum LFC range of a mutant that was significantly selected in at least 1 host genotype after resampling (<em>Q</em> &#x0003c; 0.05). Mutants used for QTL mapping also needed to be sufficiently represented in the library, which was accomplished by requiring 4 or more TA Tn insertion sites to be detected for each bacterial gene. Further, each gene included needed to be significantly selected in at least 1 host genotype before resampling (<em>P</em> &#x0003c; 0.05).</p></div><div id="jkad147-s1.7" class="sec sec-last"><h3 id="jkad147-s1.7title">Genotyping and QTL mapping</h3><p class="p p-first-last">Previously published BXD genotypes containing 7,321 total markers were leveraged for QTL mapping (<a href="#jkad147-B61" rid="jkad147-B61" class=" bibr popnode">Wang <em>et al</em>. 2016</a>). Markers that were not diagnostic of genetic differences between B6 and D2 were filtered out resulting in a total of 7,314 markers. Genotype data and phenotype data, including lung and spleen burden and TnSeq mutant fitness profiles, were imported into R statistical software (version 4.1.1) and formatted for QTL mapping within R/qtl2 (version 0.28) (<a href="#jkad147-B9" rid="jkad147-B9" class=" bibr popnode">Broman <em>et al</em>. 2019</a>). The leave-one-chromosome-out (LOCO) approach was leveraged to estimate kinship between strains as a covariate for QTL mapping. Because all screened mice were male and infected in 1 batch, no other covariates were used for the initial mapping. Logarithm of odds (LOD) scores were calculated using the linear mixed model in R/qtl2 to establish phenotypic associations with marker loci across the host genome. Trait-wise significance thresholds for QTL were established by 10,000 permutation tests. The significance of gene class overrepresentation among the mapped QTL in comparison with class representation in the whole <em>Mtb</em> genome was established using Fisher's exact test. Chromosomal ideograms of QTL across the mouse genome were generated using R/karyoploteR (<a href="#jkad147-B22" rid="jkad147-B22" class=" bibr popnode">Gel and Serra 2017</a>).</p></div></div><div id="jkad147-s2" class="tsec sec"><h2 class="head no_bottom_margin" id="jkad147-s2title">Results</h2><div id="jkad147-s2.1" class="sec sec-first"><h3 id="jkad147-s2.1title">B6 and D2 mouse strains experience distinct chronic disease profiles after aerosol infection</h3><p class="p p-first">B6 and D2, the parent strains of the BXD panel, have divergent responses to <em>Mtb</em> infection (<a href="#jkad147-B12" rid="jkad147-B12" class=" bibr popnode">Chackerian and Behar 2003</a>). Canonically considered <em>Mtb</em> resistant, the T<sub>H</sub>1-skewed adaptive immune response initiated by B6 mice is capable of restraining <em>Mtb</em> growth after 25 days, resulting in a mean survival time of &#x0223c;230 days (<a href="#jkad147-B39" rid="jkad147-B39" class=" bibr popnode">Mitsos <em>et al</em>. 2000</a>). D2 mice control early infection with <em>Mtb</em> equally as well as B6 mice until &#x0223c;25 days. However, D2 mice experience low IFN-&#x003b3;-producing T-cell influx to the lung when adaptive immunity should begin, at which point B6 and D2 begin to diverge in bacterial growth restriction (<a href="/pmc/articles/PMC10468300/figure/jkad147-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F1" rid-ob="ob-jkad147-F1" co-legend-rid="lgnd_jkad147-F1"><span>Fig. 1a and b</span></a>) and lung damage (<a href="/pmc/articles/PMC10468300/figure/jkad147-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F1" rid-ob="ob-jkad147-F1" co-legend-rid="lgnd_jkad147-F1"><span>Fig. 1c&#x02013;e</span></a>) (<a href="#jkad147-B36" rid="jkad147-B36" class=" bibr popnode">Marquis <em>et al</em>. 2008</a>). D2 macrophages become foamy and dysfunctional, making it difficult to contain and control bacterial growth in the lung (<a href="#jkad147-B12" rid="jkad147-B12" class=" bibr popnode">Chackerian and Behar 2003</a>). By 12-week postaerosol infection, D2 lungs exhibit hyperinflammatory and fibrotic responses leading to overt damage (<a href="/pmc/articles/PMC10468300/figure/jkad147-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F1" rid-ob="ob-jkad147-F1" co-legend-rid="lgnd_jkad147-F1"><span>Fig. 1f&#x02013;h</span></a>) and an ultimate mean survival time of &#x0223c;110 days (<a href="#jkad147-B37" rid="jkad147-B37" class=" bibr popnode">Medina and North 1998</a>; <a href="#jkad147-B31" rid="jkad147-B31" class=" bibr popnode">Keller <em>et al</em>. 2006</a>). In addition to genotype-specific differences between these BXD parent strains, we also observe a sex effect independent of genotype, demonstrated by higher susceptibility of males of both genotypes (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Fig. 1a and b</a>) (<a href="#jkad147-B59" rid="jkad147-B59" class=" bibr popnode">Tsuyuguchi <em>et al</em>. 2001</a>; <a href="#jkad147-B20" rid="jkad147-B20" class=" bibr popnode">Dibbern <em>et al</em>. 2017</a>). Further, we find that in contrast to burden, all groups reduce lung damage by week 12 except for D2 males (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Fig. 1c&#x02013;g</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jkad147-F1" co-legend-rid="lgnd_jkad147-F1"><a href="/pmc/articles/PMC10468300/figure/jkad147-F1/" target="figure" rid-figpopup="jkad147-F1" rid-ob="ob-jkad147-F1"><img loading="lazy" src="/pmc/articles/PMC10468300/bin/jkad147f1.gif" class="small-thumb" alt="Fig. 1." title="Fig. 1." src-large="/pmc/articles/PMC10468300/bin/jkad147f1.jpg" /></a><div class="icnblk_cntnt" id="lgnd_jkad147-F1"><div><a class="figpopup" href="/pmc/articles/PMC10468300/figure/jkad147-F1/" target="figure" rid-figpopup="jkad147-F1" rid-ob="ob-jkad147-F1">Fig. 1.</a></div><!--caption a4--><div><span>B6 and D2 are phenodeviant in TB susceptibility. a) Log<sub>10</sub> transformed CFUs recovered from lung tissue throughout the course of the aerosol infection (with an infectious dose of &#x0223c;40 CFU of <em>Mtb</em> H37Rv). b) Log<sub>10</sub> transformed CFU recovered from spleen</span> ...</div></div></div></div><div id="jkad147-s2.2" class="sec"><h3 id="jkad147-s2.2title">Early clinical macrophenotypes fail to distinguish parental disease outcomes</h3><p class="p p-first">With over 100 still extant recombinant inbred strains bred from B6 and D2 parental strains, the BXD family is a well-suited mammalian resource to map the phenotypic diversion during <em>Mtb</em> infection (<a href="/pmc/articles/PMC10468300/figure/jkad147-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F2" rid-ob="ob-jkad147-F2" co-legend-rid="lgnd_jkad147-F2"><span>Fig. 2a</span></a>). To study early disease traits and bacterial endophenotypes that predict differences in TB disease outcomes during chronic infection stages, we intravenously infected the parental B6 and D2 lines and 19 BXD genotypes with a saturated TnSeq library of Tn mutants. The library contains knockouts of every <em>Mtb</em> gene that is not required in vitro, which collectively produces an infection similar to the wild-type infection strain (<a href="#jkad147-B51" rid="jkad147-B51" class=" bibr popnode">Sassetti and Rubin 2003</a>; <a href="#jkad147-B55" rid="jkad147-B55" class=" bibr popnode">Smith <em>et al</em>. 2022</a>). At 4-week postinfection, the parental B6 and D2 strains demonstrated approximately the same bacterial burden in lung and spleen (<a href="/pmc/articles/PMC10468300/figure/jkad147-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F2" rid-ob="ob-jkad147-F2" co-legend-rid="lgnd_jkad147-F2"><span>Fig. 2b</span></a>). Moreover, BXD strains did not exhibit sufficiently wide variation in bacterial burden at 4-week postinfection to precisely map the genetic cause of such a complex trait (<a href="/pmc/articles/PMC10468300/figure/jkad147-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F2" rid-ob="ob-jkad147-F2" co-legend-rid="lgnd_jkad147-F2"><span>Fig. 2c</span></a>; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Fig. 2a and b</a>). Compound macrophenotypes, such as CFUs in organ homogenate, are highly polygenic, making it difficult to identify causal genes even with large panels of mice (<a href="#jkad147-B33" rid="jkad147-B33" class=" bibr popnode">Lavebratt <em>et al</em>. 1999</a>; <a href="#jkad147-B39" rid="jkad147-B39" class=" bibr popnode">Mitsos <em>et al</em>. 2000</a>; <a href="#jkad147-B40" rid="jkad147-B40" class=" bibr popnode">Mitsos <em>et al</em>. 2003</a>; <a href="#jkad147-B69" rid="jkad147-B69" class=" bibr popnode">Yan <em>et al</em>. 2006</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jkad147-F2" co-legend-rid="lgnd_jkad147-F2"><a href="/pmc/articles/PMC10468300/figure/jkad147-F2/" target="figure" rid-figpopup="jkad147-F2" rid-ob="ob-jkad147-F2"><img loading="lazy" src="/pmc/articles/PMC10468300/bin/jkad147f2.gif" class="small-thumb" alt="Fig. 2." title="Fig. 2." src-large="/pmc/articles/PMC10468300/bin/jkad147f2.jpg" /></a><div class="icnblk_cntnt" id="lgnd_jkad147-F2"><div><a class="figpopup" href="/pmc/articles/PMC10468300/figure/jkad147-F2/" target="figure" rid-figpopup="jkad147-F2" rid-ob="ob-jkad147-F2">Fig. 2.</a></div><!--caption a4--><div><span>BXD panel exhibits natural genetic variation, but classical clinical traits cannot map genome-wide QTL at 4-week postinfection. a) The BXD panel is a biparental recombinant inbred panel bred from C57BL/6J (B6) and DBA/2J (D2) and composed of over 100</span> ...</div></div></div></div><div id="jkad147-s2.3" class="sec"><h3 id="jkad147-s2.3title">Tn mutant fitness endophenotypes report on the host immunological microenvironment</h3><p class="p p-first">To interrogate the biological mechanisms that predict immunodivergent outcomes, we quantified the abundance of each bacterial mutant both before infection with the TnSeq library and after recovery from mouse organs of each genotype, using the LFC in abundance as a quantification of relative mutant fitness in each condition. This LFC fitness value serves as a precise reporter of the host microenvironment and can be used to map host loci that impact <em>Mtb</em> mutant fitness (<a href="/pmc/articles/PMC10468300/figure/jkad147-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F3" rid-ob="ob-jkad147-F3" co-legend-rid="lgnd_jkad147-F3"><span>Fig. 3a</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jkad147-F3" co-legend-rid="lgnd_jkad147-F3"><a href="/pmc/articles/PMC10468300/figure/jkad147-F3/" target="figure" rid-figpopup="jkad147-F3" rid-ob="ob-jkad147-F3"><img loading="lazy" src="/pmc/articles/PMC10468300/bin/jkad147f3.gif" class="small-thumb" alt="Fig. 3." title="Fig. 3." src-large="/pmc/articles/PMC10468300/bin/jkad147f3.jpg" /></a><div class="icnblk_cntnt" id="lgnd_jkad147-F3"><div><a class="figpopup" href="/pmc/articles/PMC10468300/figure/jkad147-F3/" target="figure" rid-figpopup="jkad147-F3" rid-ob="ob-jkad147-F3">Fig. 3.</a></div><!--caption a4--><div><span>TnSeq mutant fitness is a sensitive reporter of the factors that drive disease outcome. a) Each BXD and parental strain was infected with 10<sup>6</sup> CFU of <em>Mtb</em> TnSeq library, representative of &#x0223c;10<sup>5</sup> independent <em>Mtb</em> Tn mutants. For each mutant, the change</span> ...</div></div></div><p>We identified 3 main classes of bacterial mutants, which are highlighted in <a href="/pmc/articles/PMC10468300/figure/jkad147-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F3" rid-ob="ob-jkad147-F3" co-legend-rid="lgnd_jkad147-F3"><span>Fig. 3b</span></a>. The first of these classes includes canonical <em>Mtb</em> genes that are essential for in vivo survival, exemplified by <em>bioA</em>. These genes remain essential for <em>Mtb</em> survival in both parental backgrounds and across the BXD strains (<a href="/pmc/articles/PMC10468300/figure/jkad147-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F3" rid-ob="ob-jkad147-F3" co-legend-rid="lgnd_jkad147-F3"><span>Fig. 3b</span></a>; &#x0201c;essential&#x0201d; profile in red). A second, &#x0201c;differentially required&#x0201d; set of mutants is exemplified in this figure by <em>mbtG</em>; the loss of <em>mbtG</em> did not significantly reduce bacterial fitness within B6 mice, but <em>mbtG</em> mutants experienced an extreme fitness reduction within D2 mice (<a href="/pmc/articles/PMC10468300/figure/jkad147-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F3" rid-ob="ob-jkad147-F3" co-legend-rid="lgnd_jkad147-F3"><span>Fig. 3b</span></a>; &#x0201c;differentially required&#x0201d; profile in yellow). Further, there is a spectrum of <em>mbtG</em> essentiality across the family, indicating that some variable host factor could modify the fitness of this mutant. In addition to in vivo essential and differentially essential <em>Mtb</em> genes, a third class of <em>Mtb</em> genes proved to be broadly dispensable for <em>Mtb</em> survival in these hosts, imparting an adaptive benefit for <em>Mtb</em> when the gene was disturbed, represented here by <em>glpK</em> (<a href="/pmc/articles/PMC10468300/figure/jkad147-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F3" rid-ob="ob-jkad147-F3" co-legend-rid="lgnd_jkad147-F3"><span>Fig. 3b</span></a>; &#x0201c;nonessential&#x0201d; profile in green).</p><p>When surveying the complete set of <em>Mtb</em> Tn mutants, we find that in comparison with B6 infection, <em>Mtb</em> requires almost twice as many genes to survive within D2 mice (<a href="/pmc/articles/PMC10468300/figure/jkad147-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F3" rid-ob="ob-jkad147-F3" co-legend-rid="lgnd_jkad147-F3"><span>Fig. 3c</span></a>). This finding suggests that there are detectable immunological pressures being exerted on <em>Mtb</em> within the highly inflammatory D2 line prior to the divergence in B6 and D2 bacterial control and disease outcome. This result replicates findings from previous work in which a greater number of <em>Mtb</em> genes were essential in T-cell&#x02013;deficient and IFN-&#x003b3;&#x02013;deficient mice than in B6 (<a href="#jkad147-B38" rid="jkad147-B38" class=" bibr popnode">Mishra <em>et al</em>. 2017</a>; <a href="#jkad147-B55" rid="jkad147-B55" class=" bibr popnode">Smith <em>et al</em>. 2022</a>). We additionally find that the BXD panel, while recapitulating most B6- and D2-essential genes, reveals additional subsets of genes that are only required within specific BXD strains (<a href="/pmc/articles/PMC10468300/figure/jkad147-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F3" rid-ob="ob-jkad147-F3" co-legend-rid="lgnd_jkad147-F3"><span>Fig. 3d</span></a>), highlighting the power of this genetically diverse panel to offer novel insights into the biology controlling TB outcome.</p><p class="p p-last">This spectrum of <em>Mtb</em> gene essentiality is further confirmed when we examine the correlation of the complete TnSeq library fitness between hosts (<a href="/pmc/articles/PMC10468300/figure/jkad147-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F3" rid-ob="ob-jkad147-F3" co-legend-rid="lgnd_jkad147-F3"><span>Fig. 3e</span></a>). Interestingly, the majority of the strains hierarchically clustered closer to D2 than to B6, indicating that they could be inheriting factors from D2 that put greater immunological pressure on <em>Mtb</em>.</p></div><div id="jkad147-s2.4" class="sec"><h3 id="jkad147-s2.4title">Tn mutant fitness profiles map <em>hp</em>QTL</h3><p class="p p-first">We aimed to use these bacterial mutants as sensitive reporters of host immunological pressures within the early phase of infection at the onset of adaptive immunity. Using the LFC values collected for each Tn mutant population in the TnSeq library as quantitative endophenotypes (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Table 1</a>), we conducted QTL mapping. After removing nondynamic genes for quality control (<a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Fig. 3a</a>; <em>Mtb</em> genes that are not sufficiently varying in essentiality across the screened BXD genotypes), we found 140 genome-wide significant QTL, defined as <em>P</em> &#x02264; 0.05 after 10,000 permutations (<a href="/pmc/articles/PMC10468300/table/jkad147-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="jkad147-T1" rid-ob="ob-jkad147-T1" co-legend-rid=""><span>Table 1</span></a>). Of these QTL, 33 of the loci had a <em>P</em>-value of &#x02264;0.01 (<a href="/pmc/articles/PMC10468300/figure/jkad147-F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F4" rid-ob="ob-jkad147-F4" co-legend-rid="lgnd_jkad147-F4"><span>Fig. 4a</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jkad147-F4" co-legend-rid="lgnd_jkad147-F4"><a href="/pmc/articles/PMC10468300/figure/jkad147-F4/" target="figure" rid-figpopup="jkad147-F4" rid-ob="ob-jkad147-F4"><img loading="lazy" src="/pmc/articles/PMC10468300/bin/jkad147f4.gif" class="small-thumb" alt="Fig. 4." title="Fig. 4." src-large="/pmc/articles/PMC10468300/bin/jkad147f4.jpg" /></a><div class="icnblk_cntnt" id="lgnd_jkad147-F4"><div><a class="figpopup" href="/pmc/articles/PMC10468300/figure/jkad147-F4/" target="figure" rid-figpopup="jkad147-F4" rid-ob="ob-jkad147-F4">Fig. 4.</a></div><!--caption a4--><div><span>TnSeq mutant fitness endophenotypes map genome-wide significant QTL. a) The QTL that reach a significance threshold of <em>P</em> &#x02264; 0.01. The color of the QTL corresponds to the Mycobrowser class of the gene absent from the Tn mutant that mapped the QTL.</span> ...</div></div></div><!--table ft1--><!--table-wrap mode=article t1--><div class="table-wrap whole_rhythm" id="jkad147-T1"><a href="/pmc/articles/PMC10468300/table/jkad147-T1/" rid-ob="ob-jkad147-T1" rid-figpopup="jkad147-T1" class="table img_link icnblk_img figpopup"><img loading="lazy" alt="Table 1." title="Table 1." class="small-thumb" src="/pmc/articles/PMC10468300/table/jkad147-T1/?report=thumb" src-large="/pmc/articles/PMC10468300/table/jkad147-T1/?report=previmg" /></a><div class="icnblk_cntnt"><div><a class="figpopup" href="/pmc/articles/PMC10468300/table/jkad147-T1/" target="table" rid-figpopup="jkad147-T1" rid-ob="ob-jkad147-T1">Table 1.</a></div><!--caption a4--><div><span>Genome-wide significant QTL (<em>P</em> &#x02264; 0.05).</span></div></div></div><p class="p p-last">Among the most significant QTL mapped in the screen, certain gene classes were marginally overrepresented in comparison with the genome-wide abundance of that class. Tn mutants lacking genes that encode known virulence factors made up 12.1% of the top hits in this screen despite only representing 5.8% of the <em>Mtb</em> genome (<em>P</em> = 0.1) (<a href="/pmc/articles/PMC10468300/figure/jkad147-F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F4" rid-ob="ob-jkad147-F4" co-legend-rid="lgnd_jkad147-F4"><span>Fig. 4b</span></a>) (<a href="#jkad147-B30" rid="jkad147-B30" class=" bibr popnode">Kapopoulou <em>et al</em>. 2011</a>). Further, lipid metabolism genes made up 15.2% of the bacterial mutants mapping highly significant QTL while representing 6.6% of the <em>Mtb</em> genome (<em>P</em> = 0.07). Certain gene classes were abundant in approximately equal proportions in the mapped QTL as they were genome wide, such as metabolism and respiration genes (22.8% among top QTL vs 24.2% genome-wide prevalence) and genes encoding PPE/PE proteins (3.0% among top QTL vs 4.1% genome-wide prevalence). Roughly half of <em>Mtb</em> protein-coding genes remain unannotated (<a href="#jkad147-B64" rid="jkad147-B64" class=" bibr popnode">Whitaker <em>et al</em>. 2020</a>), and 7 of the highly significant (<em>P</em> &#x02264; 0.01) bacterial fitness traits mapped by this screen correspond to bacterial genes with no known function. Taking these screen data in combination with previous in vivo TnSeq studies, these data will contribute to deorphaning the <em>Mtb</em> genome.</p></div><div id="jkad147-s2.5" class="sec sec-last"><h3 id="jkad147-s2.5title">Multiple highly significant QTL map to host chromosome 6 &#x0201c;hotspot&#x0201d;</h3><p class="p p-first">The nature of this screen enabled the discovery of host loci that impact the fitness of numerous bacterial mutants. Among the QTL mapped in this screen, 5 QTL mapped to chromosome 7 (<a href="/pmc/articles/PMC10468300/figure/jkad147-F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F4" rid-ob="ob-jkad147-F4" co-legend-rid="lgnd_jkad147-F4"><span>Fig. 4a</span></a>), yet the 95% confidence intervals for each QTL span such a large part of the chromosome that identifying a causal host gene would be infeasible without further experimentation. Aside from the chromosome 7 locus, there appears to be a much narrower region on chromosome 6 (75.97&#x02013;88.58 Mb) strongly linked to the essentiality of 4 bacterial genes: <em>Rv0127</em> (<em>mak</em>; sometimes annotated as <em>pep2</em>), <em>Rv0359</em> (<em>rip2</em>), <em>Rv0955</em> (<em>perM</em>), and <em>Rv3849</em> (<em>espR</em>) (<a href="/pmc/articles/PMC10468300/figure/jkad147-F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F5" rid-ob="ob-jkad147-F5" co-legend-rid="lgnd_jkad147-F5"><span>Fig. 5a&#x02013;e</span></a>). For each of the mutants lacking these bacterial genes, apart from <em>perM</em>, the B6 sequence in this region is associated with lower bacterial mutant fitness (<a href="/pmc/articles/PMC10468300/figure/jkad147-F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F5" rid-ob="ob-jkad147-F5" co-legend-rid="lgnd_jkad147-F5"><span>Fig. 5f&#x02013;i</span></a>; <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Fig. 4a&#x02013;h</a>). <em>Mtb</em> mutants lacking <em>perM</em> experience a fitness cost within genotypes that possess the <em>D</em> haplotype in this region.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jkad147-F5" co-legend-rid="lgnd_jkad147-F5"><a href="/pmc/articles/PMC10468300/figure/jkad147-F5/" target="figure" rid-figpopup="jkad147-F5" rid-ob="ob-jkad147-F5"><img loading="lazy" src="/pmc/articles/PMC10468300/bin/jkad147f5.gif" class="small-thumb" alt="Fig. 5." title="Fig. 5." src-large="/pmc/articles/PMC10468300/bin/jkad147f5.jpg" /></a><div class="icnblk_cntnt" id="lgnd_jkad147-F5"><div><a class="figpopup" href="/pmc/articles/PMC10468300/figure/jkad147-F5/" target="figure" rid-figpopup="jkad147-F5" rid-ob="ob-jkad147-F5">Fig. 5.</a></div><!--caption a4--><div><span>A chromosome 6 QTL hotspot highlights a genomic region controlling fitness of multiple bacterial mutants. QTL mapping of Tn mutants lacking a) <em>mak</em>, b) <em>espR</em>, c) <em>perM</em>, and d) <em>rip2</em>. After 10,000 permutation tests, the solid threshold represents <em>P</em> = 0.05,</span> ...</div></div></div><p>This QTL hotspot is mapped by <em>Mtb</em> genes that impact metabolism, virulence, and defense. Mak catalyzes the ATP-dependent conversion of maltose into maltose-1-phosphate as a part of the glycogen biosynthesis pathway that supports construction of the protective outer capsule of <em>Mtb</em> (<a href="#jkad147-B34" rid="jkad147-B34" class=" bibr popnode">Li <em>et al</em>. 2014</a>). Of note, TreS, which converts trehalose to maltose upstream of Mak, also maps to a genome-wide significant QTL on chromosome 11 and has done so in previous <em>Mtb</em> infection screens (<a href="#jkad147-B55" rid="jkad147-B55" class=" bibr popnode">Smith <em>et al</em>. 2022</a>). <em>rip2</em> is a putative zinc metalloprotease located in the <em>Mtb</em> cell membrane (<a href="#jkad147-B54" rid="jkad147-B54" class=" bibr popnode">Sklar <em>et al</em>. 2010</a>), although very little is known about its function. PerM is an integral membrane protein that is known to play an essential role in enabling bacterial cell division under acid stress in B6 mice (<a href="#jkad147-B60" rid="jkad147-B60" class=" bibr popnode">Wang <em>et al</em>. 2019</a>). Here, we report that <em>perM</em> mutants are even more severely attenuated within a D2 background, implying that PerM is differentially impacted by host immunological pressures. EspR is a key regulator of the ESX-1 secretion system (<a href="#jkad147-B47" rid="jkad147-B47" class=" bibr popnode">Rosenberg <em>et al</em>. 2011</a>; <a href="#jkad147-B8" rid="jkad147-B8" class=" bibr popnode">Blasco <em>et al</em>. 2012</a>), which itself plays a well-established role in mycobacterial virulence. Mutants lacking these bacterial genes map significant QTL within the interval of 75.97&#x02013;88.58 Mb on mouse chromosome 6.</p><p>The 95% Bayesian confidence intervals of the QTL mapped by <em>rip2</em> and <em>perM</em> mutants overlap while the confidence intervals of <em>mak</em> and <em>espR</em> mutants overlap separately from the <em>rip2</em> and <em>perM</em> QTL. To test for the independence of these 2 groups of nearby QTL, we remapped each of the QTL within the chromosome 6 hotspot, iteratively incorporating the haplotype possessed by each BXD strain at each QTL as an additive covariate. Dependent loci will not map if the haplotype of 1 locus is factored into the mapping pipeline as a covariate. Conversely, if a QTL achieves significance by permutation test while the haplotype state at another significant QTL nearby is factored into mapping as a covariate, it suggests that the 2 loci arise from distinct causal genetic factors within the host. These Scan2 independence tests, so named in R/qtl, suggested 2 causal host loci exist within this highly significant QTL hotspot: 1 host factor underlying the <em>rip2</em> and <em>perM</em> mutant fitness QTL and 1 underlying the <em>mak</em> and <em>espR</em> QTL (<a href="/pmc/articles/PMC10468300/figure/jkad147-F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F6" rid-ob="ob-jkad147-F6" co-legend-rid="lgnd_jkad147-F6"><span>Fig. 6a&#x02013;d</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jkad147-F6" co-legend-rid="lgnd_jkad147-F6"><a href="/pmc/articles/PMC10468300/figure/jkad147-F6/" target="figure" rid-figpopup="jkad147-F6" rid-ob="ob-jkad147-F6"><img loading="lazy" src="/pmc/articles/PMC10468300/bin/jkad147f6.gif" class="small-thumb" alt="Fig. 6." title="Fig. 6." src-large="/pmc/articles/PMC10468300/bin/jkad147f6.jpg" /></a><div class="icnblk_cntnt" id="lgnd_jkad147-F6"><div><a class="figpopup" href="/pmc/articles/PMC10468300/figure/jkad147-F6/" target="figure" rid-figpopup="jkad147-F6" rid-ob="ob-jkad147-F6">Fig. 6.</a></div><!--caption a4--><div><span>Independence tests identify 2 putative causal variants. Scan2 independence tests from R/qtl were used to identify whether more than 1 causal host factor may underlie the chromosome 6 hotspot. The QTL mapped by a) <em>mak</em>, b) <em>espR</em>, c) <em>perM</em>, and d) <em>rip2</em> Tn</span> ...</div></div></div><p class="p">To identify host gene candidates that may mediate the survival of these 2 groups of bacterial mutants, we defined a pipeline to select for protein-coding genes for which <em>D</em> haplotype has a unique SNP in comparison with <em>B</em> haplotype (<a href="/pmc/articles/PMC10468300/figure/jkad147-F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F7" rid-ob="ob-jkad147-F7" co-legend-rid="lgnd_jkad147-F7"><span>Fig. 7a</span></a>). We passed each of the genes identified by this pipeline through a list of genetic and clinical criteria using independent but complementary data sets from a variety of <em>Mtb</em>-infected mammalian cohorts (<a href="/pmc/articles/PMC10468300/figure/jkad147-F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="jkad147-F7" rid-ob="ob-jkad147-F7" co-legend-rid="lgnd_jkad147-F7"><span>Fig. 7b</span></a>) (<a href="#jkad147-B70" rid="jkad147-B70" class=" bibr popnode">Zak <em>et al</em>. 2016</a>; <a href="#jkad147-B41" rid="jkad147-B41" class=" bibr popnode">Moreira-Teixeira <em>et al</em>. 2017</a>; <a href="#jkad147-B2" rid="jkad147-B2" class=" bibr popnode">Ahmed <em>et al</em>. 2020</a>). From this comparative analysis, we identified a putative host candidate within the region mapped by <em>mak</em>: MAX dimerization protein 1 (<em>Mxd1</em>), which is known to be upregulated in murine bone marrow-derived macrophages after infection with the hypervirulent <em>Mtb</em> HN878 strain (<a href="#jkad147-B48" rid="jkad147-B48" class=" bibr popnode">Roy <em>et al</em>. 2018</a>). Further, we found a putative host candidate within the QTL mapped by <em>perM</em> mutants: ancient ubiquitous protein 1 (<em>Aup1</em>). The QTL mapped by the <em>mak</em> Tn mutant was enriched for protein-coding genes with D2-specific SNPs compared with the other QTL within the hotspot. However, the causal locus underlying these QTL may not be protein coding at all. In fact, historical studies of cardiomyopathy in B6 and D2 backcrosses have identified a QTL that was later discovered to be caused by an intronic variant in the tropin-interacting kinase gene <em>Tnni3k</em> (<a href="#jkad147-B62" rid="jkad147-B62" class=" bibr popnode">Wheeler <em>et al</em>. 2005</a>; <a href="#jkad147-B63" rid="jkad147-B63" class=" bibr popnode">Wheeler <em>et al</em>. 2009</a>), which also conferred susceptibility to viral-induced myocarditis (<a href="#jkad147-B67" rid="jkad147-B67" class=" bibr popnode">Wiltshire <em>et al</em>. 2011</a>). There are many examples in literature of noncoding causal variants (<a href="#jkad147-B24" rid="jkad147-B24" class=" bibr popnode">Hamilton and Yu 2012</a>), yet there remain compelling candidate genes within the chromosome 6 hotspot that could drive the conditional essentiality of these <em>Mtb</em> genes in vivo. Altogether, we have leveraged complementary screen data to identify plausible host gene candidates underlying these <em>host&#x02013;pathogen</em> QTL (<em>hp</em>QTL).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="jkad147-F7" co-legend-rid="lgnd_jkad147-F7"><a href="/pmc/articles/PMC10468300/figure/jkad147-F7/" target="figure" rid-figpopup="jkad147-F7" rid-ob="ob-jkad147-F7"><img loading="lazy" src="/pmc/articles/PMC10468300/bin/jkad147f7.gif" class="small-thumb" alt="Fig. 7." title="Fig. 7." src-large="/pmc/articles/PMC10468300/bin/jkad147f7.jpg" /></a><div class="icnblk_cntnt" id="lgnd_jkad147-F7"><div><a class="figpopup" href="/pmc/articles/PMC10468300/figure/jkad147-F7/" target="figure" rid-figpopup="jkad147-F7" rid-ob="ob-jkad147-F7">Fig. 7.</a></div><!--caption a4--><div><span>A bioinformatic pipeline highlights putative candidate genes underlying the QTL on chromosome 6. a) A graphic summarizing the bioinformatic pipeline used to prioritize host gene candidates within each QTL. b) A heatmap representative of the per-gene outcome</span> ...</div></div></div></div></div><div id="jkad147-s3" class="tsec sec"><h2 class="head no_bottom_margin" id="jkad147-s3title">Discussion</h2><p class="p p-first">While it is thought that nearly a quarter of the global population is infected with <em>Mtb</em> (<a href="#jkad147-B27" rid="jkad147-B27" class=" bibr popnode">Houben and Dodd 2016</a>), only 5&#x02013;10% of <em>Mtb</em> infections ultimately result in active TB (<a href="#jkad147-B23" rid="jkad147-B23" class=" bibr popnode">Gopal <em>et al</em>. 2014</a>). This paradox has motivated decades of study on human genetic variants that could predict such a lethal prognosis in affected individuals. While Genome-wide association studies (GWAS) among infected populations has identified numerous host loci associated with increased susceptibility to disease, a limited ability to dissect biological mechanism leaves many questions unanswered (<a href="#jkad147-B52" rid="jkad147-B52" class=" bibr popnode">Saul <em>et al</em>. 2019</a>). QTL mapping, a parallel systems genetics technique, has been utilized for decades to identify host loci in genetically tractable populations linked to TB susceptibility; Sst1, one of the most well-known susceptibility loci, was first discovered through a QTL mapping study in mice (<a href="#jkad147-B32" rid="jkad147-B32" class=" bibr popnode">Kramnik <em>et al</em>. 2000</a>). Despite the widespread use of disease traits, including mean survival time, body weight, and bacterial burden, to identify host determinants of disease progression (<a href="#jkad147-B33" rid="jkad147-B33" class=" bibr popnode">Lavebratt <em>et al</em>. 1999</a>; <a href="#jkad147-B32" rid="jkad147-B32" class=" bibr popnode">Kramnik <em>et al</em>. 2000</a>; <a href="#jkad147-B39" rid="jkad147-B39" class=" bibr popnode">Mitsos <em>et al</em>. 2000</a>; <a href="#jkad147-B40" rid="jkad147-B40" class=" bibr popnode">Mitsos <em>et al</em>. 2003</a>; <a href="#jkad147-B69" rid="jkad147-B69" class=" bibr popnode">Yan <em>et al</em>. 2006</a>), these phenotypes can fail to dissect immunological features that drive distinct <em>Mtb</em> pathogenesis in diverse hosts. Here, we describe a screen in which we leveraged the reproducibility, host variation, and phenotypic divergence of the BXD family and the molecular precision of the TnSeq library to identify novel axes of host&#x02013;pathogen interaction during <em>Mtb</em> infection. We demonstrate the late (12-week) phenodivergence of the parental B6 and D2 mice; however, we found that CFU serves as a weak predictor of host outcome at the onset of adaptive immunity 4-week postinfection. By utilizing TnSeq as a refined infection trait, we identified bacterial mutants that could distinguish the resistant B6 and susceptible D2 strains at the early 4-week infection timepoint. Finally, we leveraged the differentially required mutants as traits to conduct QTL mapping and identified 140 genome-wide <em>hp</em>QTL and a QTL hotspot encompassing a cluster of <em>Mtb</em> virulence and cell wall genes. Using this BXD TnSeq platform and a novel candidate prioritization pipeline, we identified viable host gene candidates for future study.</p><p>This screen highlights the importance of intentionality in design for deep phenotyping platforms to optimally identify the causal factors underlying gene associations. In contrast to complex macrophenotypes, which require hundreds of mice to sufficiently power causal locus identification for highly polygenic traits, molecular endophenotypes, such as <em>Mtb</em> Tn mutant fitness, allow us to learn more about the intricate details of <em>Mtb</em> invasion in the early stages of infection, which is particularly helpful in an experimental framework with fewer host genotypes. The strength of TnSeq as a deep phenotyping platform lies in the redundancy of each <em>Mtb</em> gene knockout; within the TnSeq library, each bacterial gene that is nonessential in vitro was perturbed at 4 or more unique Tn insertion sites, resulting in multiple subpopulations of unique replicates of each <em>Mtb</em> gene knockout. These mutant replicates taken together paint a robust picture of individual <em>Mtb</em> gene requirement in vivo, and multiple host replicates strengthen the reproducibility of each <em>Mtb</em> mutant fitness profile. In contrast with our previous TnSeq study that utilized nearly 70 host genotypes (<a href="#jkad147-B55" rid="jkad147-B55" class=" bibr popnode">Smith <em>et al</em>. 2022</a>), this TnSeq study includes 21 unique genotypes and 73 mice, including the parental strains B6 and D2. The greater number of Tn mutant <em>hp</em>QTL identified in this screen demonstrates the power of a 2-state model to segregate allele effects, as opposed to the 8-state model present in the CC recombinant inbred panel. We, however, cannot eliminate the possibility that the smaller number of host genotypes included in this study did not sufficiently limit statistical noise during QTL mapping analyses.</p><p>Using bacterial mutant profiles as endophenotypes, we identified 140 genome-wide significant (<em>P</em> &#x02264; 0.05) associations between <em>Mtb</em> Tn mutant survival and host genetic loci in a diverse cohort, which may contribute to divergence in clinical outcomes between <em>Mtb</em>-resistant and susceptible hosts. These <em>hp</em>QTL and the Tn mutant fitness profiles across the BXD panel have been shared with GeneNetwork.org (<a href="#jkad147-B14" rid="jkad147-B14" class=" bibr popnode">Chesler <em>et al</em>. 2004</a>; <a href="#jkad147-B45" rid="jkad147-B45" class=" bibr popnode">Parker <em>et al</em>. 2017</a>; <a href="#jkad147-B4" rid="jkad147-B4" class=" bibr popnode">Ashbrook <em>et al</em>. 2021</a>), an online compendium of BXD phenotypes. Increasing the precision of bacterial traits creates a rich environment for screen-based hypothesis generation in a smaller host population, thereby lowering the cost through lower animal count and less total infection time. This screen serves as a proof of concept that large host populations are not an absolute necessity to uncover novel and biologically meaningful axes of the host&#x02013;pathogen interface. Molecular endophenotyping enables a much more accessible experimental platform to assist in our ongoing struggle against <em>Mtb</em>.</p><p>From this discovery screen, we have identified several plausible host gene candidates. Variants in <em>Aup1</em> represent a possible axis for interaction between <em>Mtb</em> and the mammalian host. The acetyltransferase activity of <em>Aup1</em> has been shown to be exploited by flaviviruses to trigger autophagy of lipid droplets within the cell, which produces ATP for the virus, thereby promoting viral replication (<a href="#jkad147-B71" rid="jkad147-B71" class=" bibr popnode">Zhang <em>et al</em>. 2018</a>). Although not much is known about whether <em>Mtb</em> interacts directly with this protein, <em>Mtb</em> is one of the few bacteria capable of generating lipid droplets (<a href="#jkad147-B18" rid="jkad147-B18" class=" bibr popnode">Daniel <em>et al</em>. 2011</a>). Host lipid droplets, specifically those derived from foamy macrophages, have been shown to promote the production of defensive cytokines during <em>Mtb</em> infection (<a href="#jkad147-B28" rid="jkad147-B28" class=" bibr popnode">Jaisinghani <em>et al</em>. 2018</a>), implicating a potentially intriguing locus for host&#x02013;pathogen interaction. The other high priority candidate <em>Mxd1</em> encodes the protein MAD, which competes with MYC to bind with MAX, forming a transcriptional repressor. This mechanism has led <em>Mxd1</em> to be considered a putative tumor suppressor gene. <em>Mxd1</em> has been shown to be upregulated in murine bone marrow&#x02013;derived macrophages upon initial infection with the hypervirulent <em>Mtb</em> HN878 strain (<a href="#jkad147-B48" rid="jkad147-B48" class=" bibr popnode">Roy <em>et al</em>. 2018</a>) and to regulate the fitness of murine dendritic cells (<a href="#jkad147-B3" rid="jkad147-B3" class=" bibr popnode">Anderson <em>et al</em>. 2020</a>), yet the function of <em>Mxd1</em> in response to <em>Mtb</em> has yet to be determined. Together, these candidates provide feasible next steps for mechanistic interrogation of these loci.</p><p class="p p-last">While the functions of many mycobacterial genes remain unknown (<a href="#jkad147-B64" rid="jkad147-B64" class=" bibr popnode">Whitaker <em>et al</em>. 2020</a>), this Tn mutant screen within the BXD family builds directly upon a previous screen in the CC panel (<a href="#jkad147-B55" rid="jkad147-B55" class=" bibr popnode">Smith <em>et al</em>. 2022</a>) to further articulate the conditional necessity of many canonically &#x0201c;nonessential&#x0201d; <em>Mtb</em> genes within a spectrum of host microenvironments. A substantial number of <em>Mtb</em> genes known to be nonessential in B6 mice prove to be essential in a subset of these genetically diverse hosts, suggesting that these bacterial genes play an active yet undiscovered role in pathogenesis under specific immunological conditions. These new host conditions have been shared with MtbTnDB (<a href="#jkad147-B29" rid="jkad147-B29" class=" bibr popnode">Jinich <em>et al</em>. 2021</a>), a central repository encompassing over 60 published <em>Mtb</em> TnSeq screens for ease of browsing and interscreen comparison. For mycobacterial researchers exploring <em>Mtb</em> genes traditionally thought to be nonessential in vivo, this work challenges such a notion and offers a new model that will provide insight into the host&#x02013;pathogen dynamic underlying this conditional in vivo essentiality.</p></div><div id="sec-a.n.e" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.n.etitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="sup1"><h4>jkad147_Supplementary_Data</h4><div class="sup-box half_rhythm" id="media-a.n.e.b.b"><a href="/pmc/articles/PMC10468300/bin/jkad147_supplementary_data.zip" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(8.0M, zip)</sup></div></div></div><div id="ack1" class="tsec sec"><h2 class="head no_bottom_margin" id="ack1title">Acknowledgments</h2><div class="sec"><p>We thank Anna Tyler, Matthew Mahoney, and Greg Carter for their assistance in concept development; Emily Hunt, Kris Riebe, and Summer Harris for technical assistance; Erin Curtis for graphic design and thoughtful discussions; Tobin and Smith Lab members for constructive feedback; Arthur Centeno for BXD phenotype import to WebQTL; Anisha Zaveri for TnSeq library import to MtbTnDB; and Douglas Marchuk for insightful comments on the manuscript. We are especially grateful to Richard Baker for rigorous biostatistical suggestions, meticulous data management and analysis, and conceptual development. Biocontainment work was partially performed in the Duke Regional Biocontainment Laboratory, which received partial support for construction from the National Institutes of Health, National Institute of Allergy and Infectious Diseases (UC6-AI058607; G20-AI167200).</p></div></div><div id="article-aaff-info" class="tsec sec"><h2 class="head no_bottom_margin" id="article-aaff-infotitle">Contributor Information</h2><p><span class="fm-affl">Rachel K Meade, </span><span class="fm-affl" id="aff-a.m.b.f.a.b"> 
Department of Molecular Genetics and Microbiology, Duke University, Durham, NC 27710, USA.</span><span class="fm-affl" id="aff-a.m.b.f.a.c"> 
University Program in Genetics and Genomics, Duke University, Durham, NC 27710, USA.</span></p><p><span class="fm-affl">Jarukit E Long, </span><span class="fm-affl" id="aff-a.m.b.f.b.b"> 
Department of Microbiology and Physiological Systems, UMass Chan Medical School, Worcester, MA 01655, USA.</span><span class="fm-affl" id="aff-a.m.b.f.b.c"> 
Research Animal Diagnostic Services, Charles River Laboratories, Wilmington, MA 01887, USA.</span></p><p><span class="fm-affl">Adrian Jinich, </span><span class="fm-affl" id="aff-a.m.b.f.c.b"> 
Division of Infectious Diseases, Weill Cornell Medical College, New York, NY 10021, USA.</span></p><p><span class="fm-affl">Kyu Y Rhee, </span><span class="fm-affl" id="aff-a.m.b.f.d.b"> 
Division of Infectious Diseases, Weill Cornell Medical College, New York, NY 10021, USA.</span></p><p><span class="fm-affl">David G Ashbrook, </span><span class="fm-affl" id="aff-a.m.b.f.e.b"> 
Department of Genetics, Genomics and Informatics, University of Tennessee Health Science Center, Memphis, TN 38163, USA.</span></p><p><span class="fm-affl">Robert W Williams, </span><span class="fm-affl" id="aff-a.m.b.f.f.b"> 
Department of Genetics, Genomics and Informatics, University of Tennessee Health Science Center, Memphis, TN 38163, USA.</span></p><p><span class="fm-affl">Christopher M Sassetti, </span><span class="fm-affl" id="aff-a.m.b.f.g.b"> 
Department of Microbiology and Physiological Systems, UMass Chan Medical School, Worcester, MA 01655, USA.</span></p><p><span class="fm-affl">Clare M Smith, </span><span class="fm-affl" id="aff-a.m.b.f.h.b"> 
Department of Molecular Genetics and Microbiology, Duke University, Durham, NC 27710, USA.</span><span class="fm-affl" id="aff-a.m.b.f.h.c"> 
University Program in Genetics and Genomics, Duke University, Durham, NC 27710, USA.</span></p></div><div id="jkad147-s4" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="jkad147-s4title">Data availability</h2><!--/article/back/sec/--><p class="p p-first">All relevant data to support the findings of this study are located within the paper and supplementary files. Genome sequence data is available in the NCBI Gene Expression Omnibus (GEO), accession number <a id="__tag_1140177333" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE234093" ref="/geo/query/acc.cgi?acc=GSE234093"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE234093","term_id":"234093"}}</span>GSE234093</a>. All QTL mapping files are publicly available at: <a href="https://github.com/SmithLabDuke/Meade_et_al_BXD_TnSeq" data-ga-action="click_feat_suppl" target="_blank">https://github.com/SmithLabDuke/Meade_et_al_BXD_TnSeq</a>. Tn mutant fitness profiles within each BXD genotype and parental genotypes are publicly available on MtbTnDB (<a href="https://www.mtbtndb.app/" data-ga-action="click_feat_suppl" target="_blank">https://www.mtbtndb.app/</a>). BXD phenotypes are available on GeneNetwork (<a href="https://www.genenetwork.org" data-ga-action="click_feat_suppl" target="_blank">https://www.genenetwork.org</a>).</p><p class="p p-last">
<a href="#sup1" rid="sup1" class=" supplementary-material">Supplemental material</a> available at G3 online.</p></div><div id="jkad147-s5" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="jkad147-s5title">Funding</h2><!--/article/back/sec/--><p class="p p-first-last">This work was funded by a Whitehead Scholar Award and a NIH (1DP2-GM146458-01) to C.M. Smith; AI132130 to C.M. Sassetti; R01GM123489 to R.W.W.</p></div><div id="__ffn_sec" class="tsec sec"><h2 class="head no_bottom_margin" id="__ffn_sectitle">Article information</h2><div class="fm-sec"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">G3 (Bethesda).</span> 2023 Sep; 13(9): jkad147. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2023 Jul 5. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fg3journal%2Fjkad147" target="_blank" rel="noopener noreferrer">10.1093/g3journal/jkad147</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC10468300</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/37405387">37405387</a></div></div></div><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Meade%20RK%5BAuthor%5D" class="affpopup" co-rid="_co_idm140614519405200" co-class="co-affbox">Rachel K Meade</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Long%20JE%5BAuthor%5D" class="affpopup" co-rid="_co_idm140614519400912" co-class="co-affbox">Jarukit E Long</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jinich%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140614519396624" co-class="co-affbox">Adrian Jinich</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rhee%20KY%5BAuthor%5D" class="affpopup" co-rid="_co_idm140614519393856" co-class="co-affbox">Kyu Y Rhee</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ashbrook%20DG%5BAuthor%5D" class="affpopup" co-rid="_co_idm140614519391088" co-class="co-affbox">David G Ashbrook</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Williams%20RW%5BAuthor%5D" class="affpopup" co-rid="_co_idm140614517678288" co-class="co-affbox">Robert W Williams</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sassetti%20CM%5BAuthor%5D" class="affpopup" co-rid="_co_idm140614517675408" co-class="co-affbox">Christopher M Sassetti</a>, and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Smith%20CM%5BAuthor%5D" class="affpopup" co-rid="_co_idm140614517672640" co-class="co-affbox">Clare M Smith</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup><sup></sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140614519405200"><h3 class="no_margin">Rachel K Meade</h3><p>
Department of Molecular Genetics and Microbiology, Duke University, Durham, NC 27710, USA</p><p>
University Program in Genetics and Genomics, Duke University, Durham, NC 27710, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Meade%20RK%5BAuthor%5D">Rachel K Meade</a></div></div><div id="_co_idm140614519400912"><h3 class="no_margin">Jarukit E Long</h3><p>
Department of Microbiology and Physiological Systems, UMass Chan Medical School, Worcester, MA 01655, USA</p><p>
Research Animal Diagnostic Services, Charles River Laboratories, Wilmington, MA 01887, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Long%20JE%5BAuthor%5D">Jarukit E Long</a></div></div><div id="_co_idm140614519396624"><h3 class="no_margin">Adrian Jinich</h3><p>
Division of Infectious Diseases, Weill Cornell Medical College, New York, NY 10021, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jinich%20A%5BAuthor%5D">Adrian Jinich</a></div></div><div id="_co_idm140614519393856"><h3 class="no_margin">Kyu Y Rhee</h3><p>
Division of Infectious Diseases, Weill Cornell Medical College, New York, NY 10021, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rhee%20KY%5BAuthor%5D">Kyu Y Rhee</a></div></div><div id="_co_idm140614519391088"><h3 class="no_margin">David G Ashbrook</h3><p>
Department of Genetics, Genomics and Informatics, University of Tennessee Health Science Center, Memphis, TN 38163, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Ashbrook%20DG%5BAuthor%5D">David G Ashbrook</a></div></div><div id="_co_idm140614517678288"><h3 class="no_margin">Robert W Williams</h3><p>
Department of Genetics, Genomics and Informatics, University of Tennessee Health Science Center, Memphis, TN 38163, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Williams%20RW%5BAuthor%5D">Robert W Williams</a></div></div><div id="_co_idm140614517675408"><h3 class="no_margin">Christopher M Sassetti</h3><p>
Department of Microbiology and Physiological Systems, UMass Chan Medical School, Worcester, MA 01655, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sassetti%20CM%5BAuthor%5D">Christopher M Sassetti</a></div></div><div id="_co_idm140614517672640"><h3 class="no_margin">Clare M Smith</h3><p>
Department of Molecular Genetics and Microbiology, Duke University, Durham, NC 27710, USA</p><p>
University Program in Genetics and Genomics, Duke University, Durham, NC 27710, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Smith%20CM%5BAuthor%5D">Clare M Smith</a></div></div></div></div><div class="contrib-group half_rhythm fm-editor">D Threadgill, <span class="fm-role">Editor</span></div><div class="half_rhythm"><div class="fm-authors-info half_rhythm"><div><span class="fm-affl">Rachel K Meade, </span><span class="fm-affl" id="aff-a.m.b.f.a.b"> 
Department of Molecular Genetics and Microbiology, Duke University, Durham, NC 27710, USA;</span><span class="fm-affl" id="aff-a.m.b.f.a.c"> 
University Program in Genetics and Genomics, Duke University, Durham, NC 27710, USA;</span></div><div><a href="#article-aaff-info">Contributor Information</a>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author" /></sup>Corresponding author.</div><div id="jkad147-cor1">Corresponding author: Department of Molecular Genetics and Microbiology, Duke University, 207 Research Drive, Durham NC 27710, USA. Email: <a href="mailto:dev@null" data-email="ude.ekud@htims.m.eralc" class="oemail">ude.ekud@htims.m.eralc</a></div><div id="jkad147-FM1">
<strong>Conflicts of interest</strong> The author(s) declare no conflicts of interest.</div></div><div class="fm-article-notes half_rhythm"><div class="fm-pubdate half_rhythm">Received 2023 Mar 7; Accepted 2023 Jun 27.</div></div><div class="permissions half_rhythm"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2023. Published by Oxford University Press on behalf of The Genetics Society of America.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article."><div><strong>Previous version available:</strong> This article is based on a previously available preprint posted on <em>bioRxiv</em> on March 6, 2023: "<a href="/pmc/articles/PMC10028809/">Genome-wide screen identifies host loci that modulate <em>M. tuberculosis</em> fitness in immunodivergent mice</a>".</div></div><div class="courtesy-note whole_rhythm small"><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">G3: Genes|Genomes|Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div></div></div><div id="ref1" class="tsec sec"><h2 class="head no_bottom_margin" id="ref1title">Literature cited</h2><div class="ref-list-sec sec" id="reference-list"><ul class="first-line-outdent"><li id="jkad147-B1"><span class="mixed-citation">
Adams DJ, Doran AG, Lilue J, Keane TM. <span class="ref-title">The mouse genomes project: a repository of inbred laboratory mouse strain genomes</span>. <span class="ref-journal">Mamm Genome</span>. 2015;<span class="ref-vol">26</span>(<span class="ref-iss">9&#x02013;10</span>):403&#x02013;412. doi: 10.1007/s00335-015-9579-6. [<a href="https://pubmed.ncbi.nlm.nih.gov/26123534">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00335-015-9579-6" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mamm+Genome&amp;title=The+mouse+genomes+project:+a+repository+of+inbred+laboratory+mouse+strain+genomes&amp;volume=26&amp;issue=9&#x02013;10&amp;publication_year=2015&amp;pages=403-412&amp;pmid=26123534&amp;doi=10.1007/s00335-015-9579-6&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B2"><span class="mixed-citation">
Ahmed M, Thirunavukkarasu S, Rosa BA, Thomas KA, Das S, Rangel-Moreno J, Lu L, Mehra S, Mbandi SK, Thackray LB, et al.. 
<span class="ref-title">Immune correlates of tuberculosis disease and risk translate across species</span>. <span class="ref-journal">Sci Transl Med</span>. 2020;<span class="ref-vol">12</span>(<span class="ref-iss">528</span>):eaay0233. . doi: 10.1126/scitranslmed.aay0233. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7354419/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31996462">PubMed</a>] [<a href="//doi.org/10.1126%2Fscitranslmed.aay0233" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Transl+Med&amp;title=Immune+correlates+of+tuberculosis+disease+and+risk+translate+across+species&amp;volume=12&amp;issue=528&amp;publication_year=2020&amp;pages=eaay0233&amp;pmid=31996462&amp;doi=10.1126/scitranslmed.aay0233&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B3"><span class="mixed-citation">
Anderson DA III, Murphy TL, Eisenman RN, Murphy KM. <span class="ref-title">The MYCL and MXD1 transcription factors regulate the fitness of murine dendritic cells</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2020;<span class="ref-vol">117</span>(<span class="ref-iss">9</span>):4885&#x02013;4893. doi: 10.1073/pnas.1915060117. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7060746/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32071205">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1915060117" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=The+MYCL+and+MXD1+transcription+factors+regulate+the+fitness+of+murine+dendritic+cells&amp;volume=117&amp;issue=9&amp;publication_year=2020&amp;pages=4885-4893&amp;pmid=32071205&amp;doi=10.1073/pnas.1915060117&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B4"><span class="mixed-citation">
Ashbrook DG, Arends D, Prins P, Mulligan MK, Roy S, Williams EG, Lutz CM, Valenzuela A, Bohl CJ, Ingels JF, et al.. 
<span class="ref-title">A platform for experimental precision medicine: the extended BXD mouse family</span>. <span class="ref-journal">Cell Syst</span>. 2021;<span class="ref-vol">12</span>(<span class="ref-iss">3</span>):235&#x02013;247.e9. . doi: 10.1016/j.cels.2020.12.002. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7979527/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33472028">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cels.2020.12.002" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Syst&amp;title=A+platform+for+experimental+precision+medicine:+the+extended+BXD+mouse+family&amp;volume=12&amp;issue=3&amp;publication_year=2021&amp;pages=235-247.e9&amp;pmid=33472028&amp;doi=10.1016/j.cels.2020.12.002&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B5"><span class="mixed-citation">
Ashbrook DG, Sasani T, Maksimov M, Gunturkun MH, Ma N, Villani F, Ren Y, Rothschild D, Chen H, Lu L, et al.. 
2022. <span class="ref-title">Private and sub-family specific mutations of founder haplotypes in the BXD family reveal phenotypic consequences relevant to health and disease</span>. <span class="ref-journal">bioRxiv 2022.04.21.489063</span>. 10.1101/2022.04.21.48906 [<a href="//doi.org/10.1101%2F2022.04.21.48906" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv+2022.04.21.489063&amp;title=Private+and+sub-family+specific+mutations+of+founder+haplotypes+in+the+BXD+family+reveal+phenotypic+consequences+relevant+to+health+and+disease&amp;publication_year=2022&amp;doi=10.1101/2022.04.21.48906&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B6"><span class="mixed-citation">
Bankhead P, Loughrey MB, Fern&#x000e1;ndez JA, Dombrowski Y, McArt DG, Dunne PD, McQuaid S, Gray RT, Murray LJ, Coleman HG, et al.. 
<span class="ref-title">Qupath: open source software for digital pathology image analysis</span>. <span class="ref-journal">Sci Rep</span>. 2017;<span class="ref-vol">7</span>(<span class="ref-iss">1</span>):16878. doi: 10.1038/s41598-017-17204-5. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5715110/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29203879">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-017-17204-5" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=Qupath:+open+source+software+for+digital+pathology+image+analysis&amp;volume=7&amp;issue=1&amp;publication_year=2017&amp;pages=16878&amp;pmid=29203879&amp;doi=10.1038/s41598-017-17204-5&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B7"><span class="mixed-citation">
Bellerose MM, Baek S-H, Huang C-C, Moss CE, Koh E-I, Proulx MK, Smith CM, Baker RE, Lee JS, Eum S, et al.. 
<span class="ref-title">Common variants in the glycerol kinase gene reduce tuberculosis drug efficacy</span>. <span class="ref-journal">mBio</span>. 2019;<span class="ref-vol">10</span>(<span class="ref-iss">4</span>):e00663-19. . doi: 10.1128/mBio.00663-19. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6667613/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31363023">PubMed</a>] [<a href="//doi.org/10.1128%2FmBio.00663-19" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Common+variants+in+the+glycerol+kinase+gene+reduce+tuberculosis+drug+efficacy&amp;volume=10&amp;issue=4&amp;publication_year=2019&amp;pages=e00663-19&amp;pmid=31363023&amp;doi=10.1128/mBio.00663-19&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B8"><span class="mixed-citation">
Blasco B, Chen JM, Hartkoorn R, Sala C, Uplekar S, Rougemont J, Pojer F, Cole ST. <span class="ref-title">Virulence regulator EspR of <em>Mycobacterium tuberculosis</em> is a nucleoid-associated protein</span>. <span class="ref-journal">PLoS Pathog</span>. 2012;<span class="ref-vol">8</span>(<span class="ref-iss">3</span>):e1002621. . doi: 10.1371/journal.ppat.1002621. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3315491/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22479184">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.ppat.1002621" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Virulence+regulator+EspR+of+Mycobacterium+tuberculosis+is+a+nucleoid-associated+protein&amp;volume=8&amp;issue=3&amp;publication_year=2012&amp;pages=e1002621&amp;pmid=22479184&amp;doi=10.1371/journal.ppat.1002621&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B9"><span class="mixed-citation">
Broman KW, Gatti DM, Simecek P, Furlotte NA, Prins P, Sen &#x0015a;, Yandell BS, Churchill GA. <span class="ref-title">R/qtl2: software for mapping quantitative trait loci with high-dimensional data and multiparent populations</span>. <span class="ref-journal">Genetics</span>. 2019;<span class="ref-vol">211</span>(<span class="ref-iss">2</span>):495&#x02013;502. doi: 10.1534/genetics.118.301595. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6366910/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30591514">PubMed</a>] [<a href="//doi.org/10.1534%2Fgenetics.118.301595" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=R/qtl2:+software+for+mapping+quantitative+trait+loci+with+high-dimensional+data+and+multiparent+populations&amp;volume=211&amp;issue=2&amp;publication_year=2019&amp;pages=495-502&amp;pmid=30591514&amp;doi=10.1534/genetics.118.301595&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B10"><span class="mixed-citation">
Caruso AM, Serbina N, Klein E, Triebold K, Bloom BR, Flynn JL. <span class="ref-title">Mice deficient in CD4 T cells have only transiently diminished levels of IFN-&#x003b3;, yet succumb to tuberculosis</span>. <span class="ref-journal">J Immunol</span>. 1999;<span class="ref-vol">162</span>(<span class="ref-iss">9</span>):5407&#x02013;5416. doi: 10.4049/jimmunol.162.9.5407. [<a href="https://pubmed.ncbi.nlm.nih.gov/10228018">PubMed</a>] [<a href="//doi.org/10.4049%2Fjimmunol.162.9.5407" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Mice+deficient+in+CD4+T+cells+have+only+transiently+diminished+levels+of+IFN-&#x003b3;,+yet+succumb+to+tuberculosis&amp;volume=162&amp;issue=9&amp;publication_year=1999&amp;pages=5407-5416&amp;pmid=10228018&amp;doi=10.4049/jimmunol.162.9.5407&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B11"><span class="mixed-citation">
Caws M, Thwaites G, Dunstan S, Hawn TR, Lan NTN, Thuong NTT, Stepniewska K, Huyen MNT, Bang ND, Tran HL, et al.. 
<span class="ref-title">The influence of host and bacterial genotype on the development of disseminated disease with <em>Mycobacterium tuberculosis</em></span>. <span class="ref-journal">PLoS Pathog</span>. 2008;<span class="ref-vol">4</span>(<span class="ref-iss">3</span>):e1000034. doi: 10.1371/journal.ppat.1000034. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2268004/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18369480">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.ppat.1000034" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=The+influence+of+host+and+bacterial+genotype+on+the+development+of+disseminated+disease+with+Mycobacterium+tuberculosis&amp;volume=4&amp;issue=3&amp;publication_year=2008&amp;pages=e1000034&amp;pmid=18369480&amp;doi=10.1371/journal.ppat.1000034&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B12"><span class="mixed-citation">
Chackerian AA, Behar SM. <span class="ref-title">Susceptibility to <em>Mycobacterium tuberculosis</em>: lessons from inbred strains of mice</span>. <span class="ref-journal">Tuberculosis(Edinb)</span>. 2003;<span class="ref-vol">83</span>(<span class="ref-iss">5</span>):279&#x02013;285. doi: 10.1016/S1472-9792(03)00017-9. [<a href="https://pubmed.ncbi.nlm.nih.gov/12972341">PubMed</a>] [<a href="//doi.org/10.1016%2FS1472-9792(03)00017-9" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Tuberculosis(Edinb)&amp;title=Susceptibility+to+Mycobacterium+tuberculosis:+lessons+from+inbred+strains+of+mice&amp;volume=83&amp;issue=5&amp;publication_year=2003&amp;pages=279-285&amp;pmid=12972341&amp;doi=10.1016/S1472-9792(03)00017-9&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B13"><span class="mixed-citation">
Chesler EJ, Lu L, Shou S, Qu Y, Gu J, Wang J, Hsu HC, Mountz JD, Baldwin NE, Langston MA, et al.. 
<span class="ref-title">Complex trait analysis of gene expression uncovers polygenic and pleiotropic networks that modulate nervous system function</span>. <span class="ref-journal">Nat Genet</span>. 2005;<span class="ref-vol">37</span>(<span class="ref-iss">3</span>):233&#x02013;242. doi: 10.1038/ng1518. [<a href="https://pubmed.ncbi.nlm.nih.gov/15711545">PubMed</a>] [<a href="//doi.org/10.1038%2Fng1518" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Complex+trait+analysis+of+gene+expression+uncovers+polygenic+and+pleiotropic+networks+that+modulate+nervous+system+function&amp;volume=37&amp;issue=3&amp;publication_year=2005&amp;pages=233-242&amp;pmid=15711545&amp;doi=10.1038/ng1518&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B14"><span class="mixed-citation">
Chesler EJ, Lu L, Wang J, Williams RW, Manly KF. <span class="ref-title">WebQTL: rapid exploratory analysis of gene expression and genetic networks for brain and behavior</span>. <span class="ref-journal">Nat Neurosci</span>. 2004;<span class="ref-vol">7</span>(<span class="ref-iss">5</span>):485&#x02013;486. doi: 10.1038/nn0504-485. [<a href="https://pubmed.ncbi.nlm.nih.gov/15114364">PubMed</a>] [<a href="//doi.org/10.1038%2Fnn0504-485" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=WebQTL:+rapid+exploratory+analysis+of+gene+expression+and+genetic+networks+for+brain+and+behavior&amp;volume=7&amp;issue=5&amp;publication_year=2004&amp;pages=485-486&amp;pmid=15114364&amp;doi=10.1038/nn0504-485&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B15"><span class="mixed-citation">
Chick JM, Munger SC, Simecek P, Huttlin EL, Choi K, Gatti DM, Raghupathy N, Svenson KL, Churchill GA, Gygi SP. <span class="ref-title">Defining the consequences of genetic variation on a proteome-wide scale</span>. <span class="ref-journal">Nature</span>. 2016;<span class="ref-vol">534</span>(<span class="ref-iss">7608</span>):500&#x02013;505. doi: 10.1038/nature18270. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5292866/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27309819">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature18270" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Defining+the+consequences+of+genetic+variation+on+a+proteome-wide+scale&amp;volume=534&amp;issue=7608&amp;publication_year=2016&amp;pages=500-505&amp;pmid=27309819&amp;doi=10.1038/nature18270&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B16"><span class="mixed-citation">
Cooper AM, Dalton DK, Stewart TA, Griffin JP, Russell DG, Orme IM. <span class="ref-title">Disseminated tuberculosis in interferon gamma gene-disrupted mice</span>. <span class="ref-journal">J Exp Med</span>. 1993;<span class="ref-vol">178</span>(<span class="ref-iss">6</span>):2243&#x02013;2247. doi: 10.1084/jem.178.6.2243. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2191280/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/8245795">PubMed</a>] [<a href="//doi.org/10.1084%2Fjem.178.6.2243" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=Disseminated+tuberculosis+in+interferon+gamma+gene-disrupted+mice&amp;volume=178&amp;issue=6&amp;publication_year=1993&amp;pages=2243-2247&amp;pmid=8245795&amp;doi=10.1084/jem.178.6.2243&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B17"><span class="mixed-citation">
Cooper AM, Magram J, Ferrante J, Orme IM. <span class="ref-title">Interleukin 12 (IL-12) is crucial to the development of protective immunity in mice intravenously infected with <em>Mycobacterium tuberculosis</em></span>. <span class="ref-journal">J Exp Med</span>. 1997;<span class="ref-vol">186</span>(<span class="ref-iss">1</span>):39&#x02013;45. doi: 10.1084/jem.186.1.39. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2198958/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9206995">PubMed</a>] [<a href="//doi.org/10.1084%2Fjem.186.1.39" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=Interleukin+12+(IL-12)+is+crucial+to+the+development+of+protective+immunity+in+mice+intravenously+infected+with+Mycobacterium+tuberculosis&amp;volume=186&amp;issue=1&amp;publication_year=1997&amp;pages=39-45&amp;pmid=9206995&amp;doi=10.1084/jem.186.1.39&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B18"><span class="mixed-citation">
Daniel J, Maamar H, Deb C, Sirakova TD, Kolattukudy PE. <span class="ref-title"><em>Mycobacterium tuberculosis</em> uses host triacylglycerol to accumulate lipid droplets and acquires a dormancy-like phenotype in lipid-loaded macrophages</span>. <span class="ref-journal">PLoS Pathog</span>. 2011;<span class="ref-vol">7</span>(<span class="ref-iss">6</span>):e1002093. doi: 10.1371/journal.ppat.1002093. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3121879/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21731490">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.ppat.1002093" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Mycobacterium+tuberculosis+uses+host+triacylglycerol+to+accumulate+lipid+droplets+and+acquires+a+dormancy-like+phenotype+in+lipid-loaded+macrophages&amp;volume=7&amp;issue=6&amp;publication_year=2011&amp;pages=e1002093&amp;pmid=21731490&amp;doi=10.1371/journal.ppat.1002093&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B19"><span class="mixed-citation">
DeJesus MA, Ambadipudi C, Baker R, Sassetti C, Ioerger TR. <span class="ref-title">TRANSIT&#x02014;a software tool for Himar1 TnSeq analysis</span>. <span class="ref-journal">PLoS Comput Biol</span>. 2015;<span class="ref-vol">11</span>(<span class="ref-iss">10</span>):e1004401. doi: 10.1371/journal.pcbi.1004401. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4598096/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26447887">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pcbi.1004401" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput+Biol&amp;title=TRANSIT&#x02014;a+software+tool+for+Himar1+TnSeq+analysis&amp;volume=11&amp;issue=10&amp;publication_year=2015&amp;pages=e1004401&amp;pmid=26447887&amp;doi=10.1371/journal.pcbi.1004401&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B20"><span class="mixed-citation">
Dibbern J, Eggers L, Schneider BE. <span class="ref-title">Sex differences in the C57BL/6 model of Mycobacterium tuberculosis infection</span>. <span class="ref-journal">Sci Rep</span>. 2017;<span class="ref-vol">7</span>(<span class="ref-iss">1</span>):10957. doi: 10.1038/s41598-017-11438-z. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5591305/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28887521">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-017-11438-z" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=Sex+differences+in+the+C57BL/6+model+of+Mycobacterium+tuberculosis+infection&amp;volume=7&amp;issue=1&amp;publication_year=2017&amp;pages=10957&amp;pmid=28887521&amp;doi=10.1038/s41598-017-11438-z&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B21"><span class="mixed-citation">
Flynn JL, Chan J, Triebold KJ, Dalton DK, Stewart TA, Bloom BR. <span class="ref-title">An essential role for interferon &#x003b3; in resistance to <em>Mycobacterium tuberculosis</em> infection</span>. <span class="ref-journal">J Exp Med</span>. 1993;<span class="ref-vol">178</span>(<span class="ref-iss">6</span>):2249&#x02013;2254. doi: 10.1084/jem.178.6.2249. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2191274/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/7504064">PubMed</a>] [<a href="//doi.org/10.1084%2Fjem.178.6.2249" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Exp+Med&amp;title=An+essential+role+for+interferon+&#x003b3;+in+resistance+to+Mycobacterium+tuberculosis+infection&amp;volume=178&amp;issue=6&amp;publication_year=1993&amp;pages=2249-2254&amp;pmid=7504064&amp;doi=10.1084/jem.178.6.2249&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B22"><span class="mixed-citation">
Gel B, Serra E. <span class="ref-title">Karyoploter: an R/Bioconductor package to plot customizable genomes displaying arbitrary data</span>. <span class="ref-journal">Bioinformatics</span>. 2017;<span class="ref-vol">33</span>(<span class="ref-iss">19</span>):3088&#x02013;3090. doi: 10.1093/bioinformatics/btx346. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5870550/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28575171">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtx346" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Karyoploter:+an+R/Bioconductor+package+to+plot+customizable+genomes+displaying+arbitrary+data&amp;volume=33&amp;issue=19&amp;publication_year=2017&amp;pages=3088-3090&amp;pmid=28575171&amp;doi=10.1093/bioinformatics/btx346&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B23"><span class="mixed-citation">
Gopal R, Monin L, Slight S, Uche U, Blanchard E, Fallert Junecko BA, Ramos-Payan R, Stallings CL, Reinhart TA, Kolls JK, et al.. 
<span class="ref-title">Unexpected role for IL-17 in protective immunity against hypervirulent <em>Mycobacterium tuberculosis</em> HN878 infection.</span>
<span class="ref-journal">PLoS Pathog</span>. 2014;<span class="ref-vol">10</span>(<span class="ref-iss">5</span>):e1004099. doi: 10.1371/journal.ppat.1004099. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4022785/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24831696">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.ppat.1004099" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Unexpected+role+for+IL-17+in+protective+immunity+against+hypervirulent+Mycobacterium+tuberculosis+HN878+infection.&amp;volume=10&amp;issue=5&amp;publication_year=2014&amp;pages=e1004099&amp;pmid=24831696&amp;doi=10.1371/journal.ppat.1004099&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B24"><span class="mixed-citation">
Hamilton BA, Yu BD. <span class="ref-title">Modifier genes and the plasticity of genetic networks in mice</span>. <span class="ref-journal">PLoS Genet</span>. 2012;<span class="ref-vol">8</span>(<span class="ref-iss">4</span>):e1002644. doi: 10.1371/journal.pgen.1002644. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3325199/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22511884">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pgen.1002644" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Modifier+genes+and+the+plasticity+of+genetic+networks+in+mice&amp;volume=8&amp;issue=4&amp;publication_year=2012&amp;pages=e1002644&amp;pmid=22511884&amp;doi=10.1371/journal.pgen.1002644&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B25"><span class="mixed-citation">
Hern&#x000e1;ndez-Pando R, Marquina-Castillo B, Barrios-Pay&#x000e1;n J, Mata-Espinosa D. <span class="ref-title">Use of mouse models to study the variability in virulence associated with specific genotypic lineages of <em>Mycobacterium tuberculosis</em></span>. <span class="ref-journal">Infect Genet Evol</span>. 2012;<span class="ref-vol">12</span>(<span class="ref-iss">4</span>):725&#x02013;731. doi: 10.1016/j.meegid.2012.02.013. [<a href="https://pubmed.ncbi.nlm.nih.gov/22426109">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.meegid.2012.02.013" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect+Genet+Evol&amp;title=Use+of+mouse+models+to+study+the+variability+in+virulence+associated+with+specific+genotypic+lineages+of+Mycobacterium+tuberculosis&amp;volume=12&amp;issue=4&amp;publication_year=2012&amp;pages=725-731&amp;pmid=22426109&amp;doi=10.1016/j.meegid.2012.02.013&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B26"><span class="mixed-citation">
Hershkovitz I, Donoghue HD, Minnikin DE, Besra GS, Lee OY-C, Gernaey AM, Galili E, Eshed V, Greenblatt CL, Lemma E, et al.. 
<span class="ref-title">Detection and molecular characterization of 9000-year-old <em>Mycobacterium tuberculosis</em> from a neolithic settlement in the Eastern Mediterranean</span>. <span class="ref-journal">PLoS One</span>. 2008;<span class="ref-vol">3</span>(<span class="ref-iss">10</span>):e3426. doi: 10.1371/journal.pone.0003426. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2565837/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18923677">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0003426" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Detection+and+molecular+characterization+of+9000-year-old+Mycobacterium+tuberculosis+from+a+neolithic+settlement+in+the+Eastern+Mediterranean&amp;volume=3&amp;issue=10&amp;publication_year=2008&amp;pages=e3426&amp;pmid=18923677&amp;doi=10.1371/journal.pone.0003426&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B27"><span class="mixed-citation">
Houben RMGJ, Dodd PJ. <span class="ref-title">The global burden of latent tuberculosis infection: a Re-estimation using mathematical modelling</span>. <span class="ref-journal">PLOS Med</span>. 2016;<span class="ref-vol">13</span>(<span class="ref-iss">10</span>):e1002152. doi: 10.1371/journal.pmed.1002152. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5079585/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27780211">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pmed.1002152" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLOS+Med&amp;title=The+global+burden+of+latent+tuberculosis+infection:+a+Re-estimation+using+mathematical+modelling&amp;volume=13&amp;issue=10&amp;publication_year=2016&amp;pages=e1002152&amp;pmid=27780211&amp;doi=10.1371/journal.pmed.1002152&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B28"><span class="mixed-citation">
Jaisinghani N, Dawa S, Singh K, Nandy A, Menon D, Bhandari PD, Khare G, Tyagi A, Gandotra S. <span class="ref-title">Necrosis driven triglyceride synthesis primes macrophages for inflammation during <em>Mycobacterium tuberculosis</em> infection</span>. <span class="ref-journal">Front Immunol</span>. 2018;<span class="ref-vol">9</span>:1490. doi: 10.3389/fimmu.2018.014903. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6037689/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30018616">PubMed</a>] [<a href="//doi.org/10.3389%2Ffimmu.2018.01490" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Immunol&amp;title=Necrosis+driven+triglyceride+synthesis+primes+macrophages+for+inflammation+during+Mycobacterium+tuberculosis+infection&amp;volume=9&amp;publication_year=2018&amp;pages=1490&amp;pmid=30018616&amp;doi=10.3389/fimmu.2018.01490&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B29"><span class="mixed-citation">
Jinich A, Zaveri A, DeJesus MA, Flores-Bautista E, Smith CM, Sassetti CM, Rock JM, Ehrt S, Schnappinger D, Ioerger TR, et al.. 
2021. <span class="ref-title">The <em>Mycobacterium tuberculosis</em> transposon sequencing database (MtbTnDB): a large-scale guide to genetic conditional essentiality</span>. <span class="ref-journal">bioRxiv 2021.03.05.434127</span>. 10.1101/2021.03.05.434127. [<a href="//doi.org/10.1101%2F2021.03.05.434127" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv+2021.03.05.434127&amp;title=The+Mycobacterium+tuberculosis+transposon+sequencing+database+(MtbTnDB):+a+large-scale+guide+to+genetic+conditional+essentiality&amp;publication_year=2021&amp;doi=10.1101/2021.03.05.434127&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B30"><span class="mixed-citation">
Kapopoulou A, Lew JM, Cole ST. <span class="ref-title">The MycoBrowser portal: a comprehensive and manually annotated resource for mycobacterial genomes</span>. <span class="ref-journal">Tuberculosis</span>. 2011;<span class="ref-vol">91</span>(<span class="ref-iss">1</span>):8&#x02013;13. doi: 10.1016/j.tube.2010.09.006. [<a href="https://pubmed.ncbi.nlm.nih.gov/20980200">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.tube.2010.09.006" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Tuberculosis&amp;title=The+MycoBrowser+portal:+a+comprehensive+and+manually+annotated+resource+for+mycobacterial+genomes&amp;volume=91&amp;issue=1&amp;publication_year=2011&amp;pages=8-13&amp;pmid=20980200&amp;doi=10.1016/j.tube.2010.09.006&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B31"><span class="mixed-citation">
Keller C, Hoffmann R, Lang R, Brandau S, Hermann C, Ehlers S. <span class="ref-title">Genetically determined susceptibility to tuberculosis in mice causally involves accelerated and enhanced recruitment of granulocytes</span>. <span class="ref-journal">Infect Immun</span>. 2006;<span class="ref-vol">74</span>(<span class="ref-iss">7</span>):4295&#x02013;4309. doi: 10.1128/IAI.00057-06. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1489748/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16790804">PubMed</a>] [<a href="//doi.org/10.1128%2FIAI.00057-06" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect+Immun&amp;title=Genetically+determined+susceptibility+to+tuberculosis+in+mice+causally+involves+accelerated+and+enhanced+recruitment+of+granulocytes&amp;volume=74&amp;issue=7&amp;publication_year=2006&amp;pages=4295-4309&amp;pmid=16790804&amp;doi=10.1128/IAI.00057-06&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B32"><span class="mixed-citation">
Kramnik I, Dietrich WF, Demant P, Bloom BR. <span class="ref-title">Genetic control of resistance to experimental infection with virulent <em>Mycobacterium tuberculosis</em></span>. <span class="ref-journal">Proc Natl Acad Sci</span>. 2000;<span class="ref-vol">97</span>(<span class="ref-iss">15</span>):8560&#x02013;8565. doi: 10.1073/pnas.150227197. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC26987/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10890913">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.150227197" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci&amp;title=Genetic+control+of+resistance+to+experimental+infection+with+virulent+Mycobacterium+tuberculosis&amp;volume=97&amp;issue=15&amp;publication_year=2000&amp;pages=8560-8565&amp;pmid=10890913&amp;doi=10.1073/pnas.150227197&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B33"><span class="mixed-citation">
Lavebratt C, Apt AS, Nikonenko B V, Schalling M, Schurr E. <span class="ref-title">Severity of tuberculosis in mice is linked to distal chromosome 3 and proximal chromosome 9</span>. <span class="ref-journal">J Infect Dis</span>. 1999;<span class="ref-vol">180</span>(<span class="ref-iss">1</span>):150&#x02013;155. doi: 10.1086/314843. [<a href="https://pubmed.ncbi.nlm.nih.gov/10353873">PubMed</a>] [<a href="//doi.org/10.1086%2F314843" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Severity+of+tuberculosis+in+mice+is+linked+to+distal+chromosome+3+and+proximal+chromosome+9&amp;volume=180&amp;issue=1&amp;publication_year=1999&amp;pages=150-155&amp;pmid=10353873&amp;doi=10.1086/314843&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B34"><span class="mixed-citation">
Li J, Guan X, Shaw N, Chen W, Dong Y, Xu X, Li X, Rao Z. <span class="ref-title">Homotypic dimerization of a maltose kinase for molecular scaffolding</span>. <span class="ref-journal">Sci Rep</span>. 2014;<span class="ref-vol">4</span>(<span class="ref-iss">1</span>):6418. doi: 10.1038/srep06418. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4171701/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25245657">PubMed</a>] [<a href="//doi.org/10.1038%2Fsrep06418" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=Homotypic+dimerization+of+a+maltose+kinase+for+molecular+scaffolding&amp;volume=4&amp;issue=1&amp;publication_year=2014&amp;pages=6418&amp;pmid=25245657&amp;doi=10.1038/srep06418&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B35"><span class="mixed-citation">
Long JE, DeJesus M, Ward D, Baker RE, Ioerger T, Sassetti CM. Identifying essential genes in <em>Mycobacterium tuberculosis</em> by global phenotypic profiling. <span class="ref-journal">Methods in Molecular Biology</span>. New York: Humana Press Inc; 2015. p. 79&#x02013;95. [<a href="https://pubmed.ncbi.nlm.nih.gov/25636614">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Methods+in+Molecular+Biology&amp;publication_year=2015&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B36"><span class="mixed-citation">
Marquis JF, Nantel A, LaCourse R, Ryan L, North RJ, Gros P. <span class="ref-title">Fibrotic response as a distinguishing feature of resistance and susceptibility to pulmonary infection with <em>Mycobacterium tuberculosis</em> in mice</span>. <span class="ref-journal">Infect Immun</span>. 2008;<span class="ref-vol">76</span>(<span class="ref-iss">1</span>):78&#x02013;88. doi: 10.1128/IAI.00369-07. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2223652/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17938213">PubMed</a>] [<a href="//doi.org/10.1128%2FIAI.00369-07" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect+Immun&amp;title=Fibrotic+response+as+a+distinguishing+feature+of+resistance+and+susceptibility+to+pulmonary+infection+with+Mycobacterium+tuberculosis+in+mice&amp;volume=76&amp;issue=1&amp;publication_year=2008&amp;pages=78-88&amp;pmid=17938213&amp;doi=10.1128/IAI.00369-07&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B37"><span class="mixed-citation">
Medina E, North RJ. <span class="ref-title">Resistance ranking of some common inbred mouse strains to <em>Mycobacterium tuberculosis</em> and relationship to major histocompatibility complex haplotype and Nramp1 genotype</span>. <span class="ref-journal">Immunology</span>. 1998;<span class="ref-vol">93</span>(<span class="ref-iss">2</span>):270&#x02013;274. doi: 10.1046/j.1365-2567.1998.00419.x. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1364188/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9616378">PubMed</a>] [<a href="//doi.org/10.1046%2Fj.1365-2567.1998.00419.x" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunology&amp;title=Resistance+ranking+of+some+common+inbred+mouse+strains+to+Mycobacterium+tuberculosis+and+relationship+to+major+histocompatibility+complex+haplotype+and+Nramp1+genotype&amp;volume=93&amp;issue=2&amp;publication_year=1998&amp;pages=270-274&amp;pmid=9616378&amp;doi=10.1046/j.1365-2567.1998.00419.x&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B38"><span class="mixed-citation">
Mishra BB, Lovewell RR, Olive AJ, Zhang G, Wang W, Eugenin E, Smith CM, Phuah JY, Long JE, Dubuke ML, et al.. 
<span class="ref-title">Nitric oxide prevents a pathogen-permissive granulocytic inflammation during tuberculosis</span>. <span class="ref-journal">Nat Microbiol</span>. 2017;<span class="ref-vol">2</span>(<span class="ref-iss">7</span>):17072. doi: 10.1038/nmicrobiol.2017.72. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5461879/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28504669">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmicrobiol.2017.72" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Microbiol&amp;title=Nitric+oxide+prevents+a+pathogen-permissive+granulocytic+inflammation+during+tuberculosis&amp;volume=2&amp;issue=7&amp;publication_year=2017&amp;pages=17072&amp;pmid=28504669&amp;doi=10.1038/nmicrobiol.2017.72&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B39"><span class="mixed-citation">
Mitsos L-M, Cardon LR, Fortin A, Ryan L, LaCourse R, North RJ, Gros P. <span class="ref-title">Genetic control of susceptibility to infection with <em>Mycobacterium tuberculosis</em> in mice</span>. <span class="ref-journal">Genes Immun</span>. 2000;<span class="ref-vol">1</span>(<span class="ref-iss">8</span>):467&#x02013;477. doi: 10.1038/sj.gene.6363712. [<a href="https://pubmed.ncbi.nlm.nih.gov/11197687">PubMed</a>] [<a href="//doi.org/10.1038%2Fsj.gene.6363712" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=Genetic+control+of+susceptibility+to+infection+with+Mycobacterium+tuberculosis+in+mice&amp;volume=1&amp;issue=8&amp;publication_year=2000&amp;pages=467-477&amp;pmid=11197687&amp;doi=10.1038/sj.gene.6363712&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B40"><span class="mixed-citation">
Mitsos L-M, Cardon LR, Ryan L, LaCourse R, North RJ, Gros P. <span class="ref-title">Susceptibility to tuberculosis: a locus on mouse chromosome 19 (<em>Trl-4</em>) regulates <em>Mycobacterium tuberculosis</em> replication in the lungs</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2003;<span class="ref-vol">100</span>(<span class="ref-iss">11</span>):6610&#x02013;6615.. doi: 10.1073/pnas.1031727100. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC164495/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12740444">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1031727100" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Susceptibility+to+tuberculosis:+a+locus+on+mouse+chromosome+19+(Trl-4)+regulates+Mycobacterium+tuberculosis+replication+in+the+lungs&amp;volume=100&amp;issue=11&amp;publication_year=2003&amp;pages=6610-6615&amp;pmid=12740444&amp;doi=10.1073/pnas.1031727100&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B41"><span class="mixed-citation">
Moreira-Teixeira L, Redford PS, Stavropoulos E, Ghilardi N, Maynard CL, Weaver CT, Freitas do Ros&#x000e1;rio AP, Wu X, Langhorne J, O&#x02019;Garra A. <span class="ref-title">T cell&#x02013;derived IL-10 impairs host resistance to <em>Mycobacterium tuberculosis</em> infection</span>. <span class="ref-journal">J Immunol</span>. 2017;<span class="ref-vol">199</span>(<span class="ref-iss">2</span>):613&#x02013;623. doi: 10.4049/jimmunol.1601340. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5502318/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28584007">PubMed</a>] [<a href="//doi.org/10.4049%2Fjimmunol.1601340" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=T+cell&#x02013;derived+IL-10+impairs+host+resistance+to+Mycobacterium+tuberculosis+infection&amp;volume=199&amp;issue=2&amp;publication_year=2017&amp;pages=613-623&amp;pmid=28584007&amp;doi=10.4049/jimmunol.1601340&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B42"><span class="mixed-citation">
Moreira-Teixeira L, Tabone O, Graham CM, Singhania A, Stavropoulos E, Redford PS, Chakravarty P, Priestnall SL, Suarez-Bonnet A, Herbert E, et al.. 
<span class="ref-title">Mouse transcriptome reveals potential signatures of protection and pathogenesis in human tuberculosis</span>. <span class="ref-journal">Nat Immunol</span>. 2020;<span class="ref-vol">21</span>(<span class="ref-iss">4</span>):464&#x02013;476. doi: 10.1038/s41590-020-0610-z. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7116040/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32205882">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41590-020-0610-z" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=Mouse+transcriptome+reveals+potential+signatures+of+protection+and+pathogenesis+in+human+tuberculosis&amp;volume=21&amp;issue=4&amp;publication_year=2020&amp;pages=464-476&amp;pmid=32205882&amp;doi=10.1038/s41590-020-0610-z&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B43"><span class="mixed-citation">
Munger SC, Raghupathy N, Choi K, Simons AK, Gatti DM, Hinerfeld DA, Svenson KL, Keller MP, Attie AD, Hibbs MA, et al.. 
<span class="ref-title">RNA-Seq alignment to individualized genomes improves transcript abundance estimates in multiparent populations</span>. <span class="ref-journal">Genetics</span>. 2014;<span class="ref-vol">198</span>(<span class="ref-iss">1</span>):59&#x02013;73. doi: 10.1534/genetics.114.165886. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4174954/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25236449">PubMed</a>] [<a href="//doi.org/10.1534%2Fgenetics.114.165886" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=RNA-Seq+alignment+to+individualized+genomes+improves+transcript+abundance+estimates+in+multiparent+populations&amp;volume=198&amp;issue=1&amp;publication_year=2014&amp;pages=59-73&amp;pmid=25236449&amp;doi=10.1534/genetics.114.165886&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B44"><span class="mixed-citation">
Nambi S, Long JE, Mishra BB, Baker R, Murphy KC, Olive AJ, Nguyen HP, Shaffer SA, Sassetti CM. <span class="ref-title">The oxidative stress network of <em>Mycobacterium tuberculosis</em> reveals coordination between radical detoxification systems</span>. <span class="ref-journal">Cell Host Microbe</span>. 2015;<span class="ref-vol">17</span>(<span class="ref-iss">6</span>):829&#x02013;837. doi: 10.1016/j.chom.2015.05.008. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4465913/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26067605">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.chom.2015.05.008" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Host+Microbe&amp;title=The+oxidative+stress+network+of+Mycobacterium+tuberculosis+reveals+coordination+between+radical+detoxification+systems&amp;volume=17&amp;issue=6&amp;publication_year=2015&amp;pages=829-837&amp;pmid=26067605&amp;doi=10.1016/j.chom.2015.05.008&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B45"><span class="mixed-citation">
Parker CC, Dickson PE, Philip VM, Thomas M, Chesler EJ. <span class="ref-title">Systems genetic analysis in GeneNetwork.org</span>. <span class="ref-journal">Curr Protoc Neurosci</span>. 2017;<span class="ref-vol">79</span>(<span class="ref-iss">1</span>):8.39.1&#x02013;8.39.20. doi: 10.1002/cpns.23. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5548442/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28398643">PubMed</a>] [<a href="//doi.org/10.1002%2Fcpns.23" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Protoc+Neurosci&amp;title=Systems+genetic+analysis+in+GeneNetwork.org&amp;volume=79&amp;issue=1&amp;publication_year=2017&amp;pages=8.39.1-8.39.20&amp;doi=10.1002/cpns.23&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B46"><span class="mixed-citation">
Peirce JL, Lu L, Gu J, Silver LM, Williams RW. <span class="ref-title">A new set of BXD recombinant inbred lines from advanced intercross populations in mice</span>. <span class="ref-journal">BMC Genet</span>. 2004;<span class="ref-vol">5</span>(<span class="ref-iss">1</span>):7. doi: 10.1186/1471-2156-5-7. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC420238/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15117419">PubMed</a>] [<a href="//doi.org/10.1186%2F1471-2156-5-7" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genet&amp;title=A+new+set+of+BXD+recombinant+inbred+lines+from+advanced+intercross+populations+in+mice&amp;volume=5&amp;issue=1&amp;publication_year=2004&amp;pages=7&amp;pmid=15117419&amp;doi=10.1186/1471-2156-5-7&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B47"><span class="mixed-citation">
Rosenberg OS, Dovey C, Tempesta M, Robbins RA, Finer-Moore JS, Stroud RM, Cox JS. <span class="ref-title">Espr, a key regulator of <em>Mycobacterium tuberculosis</em> virulence, adopts a unique dimeric structure among helix-turn-helix proteins</span>. <span class="ref-journal">Proc Natl Acad Sci</span>. 2011;<span class="ref-vol">108</span>(<span class="ref-iss">33</span>):13450&#x02013;13455. doi: 10.1073/pnas.1110242108. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3158157/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21795602">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1110242108" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci&amp;title=Espr,+a+key+regulator+of+Mycobacterium+tuberculosis+virulence,+adopts+a+unique+dimeric+structure+among+helix-turn-helix+proteins&amp;volume=108&amp;issue=33&amp;publication_year=2011&amp;pages=13450-13455&amp;pmid=21795602&amp;doi=10.1073/pnas.1110242108&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B48"><span class="mixed-citation">
Roy S, Schmeier S, Kaczkowski B, Arner E, Alam T, Ozturk M, Tamgue O, Parihar SP, Kawaji H, Itoh M, et al.. 
<span class="ref-title">Transcriptional landscape of <em>Mycobacterium tuberculosis</em> infection in macrophages</span>. <span class="ref-journal">Sci Rep</span>. 2018;<span class="ref-vol">8</span>(<span class="ref-iss">1</span>):6758. doi: 10.1038/s41598-018-24509-6. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5928056/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29712924">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-018-24509-6" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Rep&amp;title=Transcriptional+landscape+of+Mycobacterium+tuberculosis+infection+in+macrophages&amp;volume=8&amp;issue=1&amp;publication_year=2018&amp;pages=6758&amp;pmid=29712924&amp;doi=10.1038/s41598-018-24509-6&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B49"><span class="mixed-citation">
Sasani TA, Ashbrook DG, Beichman AC, Lu L, Palmer AA, Williams RW, Pritchard JK, Harris K. <span class="ref-title">A natural mutator allele shapes mutation spectrum variation in mice</span>. <span class="ref-journal">Nature</span>. 2022;<span class="ref-vol">605</span>(<span class="ref-iss">7910</span>):497&#x02013;502. doi: 10.1038/s41586-022-04701-5. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9272728/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35545679">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-022-04701-5" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+natural+mutator+allele+shapes+mutation+spectrum+variation+in+mice&amp;volume=605&amp;issue=7910&amp;publication_year=2022&amp;pages=497-502&amp;pmid=35545679&amp;doi=10.1038/s41586-022-04701-5&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B50"><span class="mixed-citation">
Sassetti CM, Boyd DH, Rubin EJ. <span class="ref-title">Genes required for mycobacterial growth defined by high density mutagenesis</span>. <span class="ref-journal">Mol Microbiol</span>. 2003;<span class="ref-vol">48</span>(<span class="ref-iss">1</span>):77&#x02013;84. doi: 10.1046/j.1365-2958.2003.03425.x. [<a href="https://pubmed.ncbi.nlm.nih.gov/12657046">PubMed</a>] [<a href="//doi.org/10.1046%2Fj.1365-2958.2003.03425.x" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=Genes+required+for+mycobacterial+growth+defined+by+high+density+mutagenesis&amp;volume=48&amp;issue=1&amp;publication_year=2003&amp;pages=77-84&amp;pmid=12657046&amp;doi=10.1046/j.1365-2958.2003.03425.x&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B51"><span class="mixed-citation">
Sassetti CM, Rubin EJ. <span class="ref-title">Genetic requirements for mycobacterial survival during infection</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>. 2003;<span class="ref-vol">100</span>(<span class="ref-iss">22</span>):12989&#x02013;12994. doi: 10.1073/pnas.2134250100. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC240732/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14569030">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.2134250100" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Genetic+requirements+for+mycobacterial+survival+during+infection&amp;volume=100&amp;issue=22&amp;publication_year=2003&amp;pages=12989-12994&amp;pmid=14569030&amp;doi=10.1073/pnas.2134250100&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B52"><span class="mixed-citation">
Saul MC, Philip VM, Reinholdt LG, Center for Systems Neurogenetics of Addiction, Chesler EJ. <span class="ref-title">High-diversity mouse populations for complex traits</span>. <span class="ref-journal">Trends Genet</span>. 2019;<span class="ref-vol">35</span>(<span class="ref-iss">7</span>):501&#x02013;514. doi: 10.1016/j.tig.2019.04.003. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6571031/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31133439">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.tig.2019.04.003" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;title=High-diversity+mouse+populations+for+complex+traits&amp;volume=35&amp;issue=7&amp;publication_year=2019&amp;pages=501-514&amp;pmid=31133439&amp;doi=10.1016/j.tig.2019.04.003&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B53"><span class="mixed-citation">
Skelly DA, Czechanski A, Byers C, Aydin S, Spruce C, Olivier C, Choi K, Gatti DM, Raghupathy N, Keele GR, et al.. 
<span class="ref-title">Mapping the effects of genetic variation on chromatin state and gene expression reveals loci that control ground state pluripotency</span>. <span class="ref-journal">Cell Stem Cell</span>. 2020;<span class="ref-vol">27</span>(<span class="ref-iss">3</span>):459&#x02013;469.e8. doi: 10.1016/j.stem.2020.07.005. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7484384/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32795400">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.stem.2020.07.005" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Stem+Cell&amp;title=Mapping+the+effects+of+genetic+variation+on+chromatin+state+and+gene+expression+reveals+loci+that+control+ground+state+pluripotency&amp;volume=27&amp;issue=3&amp;publication_year=2020&amp;pages=459-469.e8&amp;pmid=32795400&amp;doi=10.1016/j.stem.2020.07.005&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B54"><span class="mixed-citation">
Sklar JG, Makinoshima H, Schneider JS, Glickman MS. <span class="ref-title"><em>M. tuberculosis</em> intramembrane protease Rip1 controls transcription through three anti-sigma factor substrates</span>. <span class="ref-journal">Mol Microbiol</span>. 2010;<span class="ref-vol">77</span>(<span class="ref-iss">3</span>):605&#x02013;617. doi: 10.1111/j.1365-2958.2010.07232.x. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3008510/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20545848">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-2958.2010.07232.x" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Microbiol&amp;title=M.+tuberculosis+intramembrane+protease+Rip1+controls+transcription+through+three+anti-sigma+factor+substrates&amp;volume=77&amp;issue=3&amp;publication_year=2010&amp;pages=605-617&amp;pmid=20545848&amp;doi=10.1111/j.1365-2958.2010.07232.x&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B55"><span class="mixed-citation">
Smith CM, Baker RE, Proulx MK, Mishra BB, Long JE, Park SW, Lee H-N, Kiritsy MC, Bellerose MM, Olive AJ, et al.. 
<span class="ref-title">Host&#x02013;pathogen genetic interactions underlie tuberculosis susceptibility in genetically diverse mice</span>. <span class="ref-journal">eLife</span>. 2022;<span class="ref-vol">11</span>:e74419. doi: 10.7554/eLife.74419. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8846590/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35112666">PubMed</a>] [<a href="//doi.org/10.7554%2FeLife.74419" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Host&#x02013;pathogen+genetic+interactions+underlie+tuberculosis+susceptibility+in+genetically+diverse+mice&amp;volume=11&amp;publication_year=2022&amp;pages=e74419&amp;pmid=35112666&amp;doi=10.7554/eLife.74419&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B56"><span class="mixed-citation">
Smith CM, Sassetti CM. <span class="ref-title">Modeling diversity: do homogeneous laboratory strains limit discovery? </span><span class="ref-journal">Trends Microbiol</span>. 2018;<span class="ref-vol">26</span>(<span class="ref-iss">11</span>):892&#x02013;895. doi: 10.1016/j.tim.2018.08.002. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6610874/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30166218">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.tim.2018.08.002" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Microbiol&amp;title=Modeling+diversity:+do+homogeneous+laboratory+strains+limit+discovery?&amp;volume=26&amp;issue=11&amp;publication_year=2018&amp;pages=892-895&amp;pmid=30166218&amp;doi=10.1016/j.tim.2018.08.002&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B57"><span class="mixed-citation">
Taylor B, Heiniger H, Meier H. <span class="ref-title">Genetic analysis of resistance to cadmium-induced testicular damage in mice</span>. <span class="ref-journal">Proc Soc Exp Biol Med</span>. 1973;<span class="ref-vol">143</span>(<span class="ref-iss">3</span>):629&#x02013;633. doi: 10.3181/00379727-143-37380. [<a href="https://pubmed.ncbi.nlm.nih.gov/4719448">PubMed</a>] [<a href="//doi.org/10.3181%2F00379727-143-37380" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Soc+Exp+Biol+Med&amp;title=Genetic+analysis+of+resistance+to+cadmium-induced+testicular+damage+in+mice&amp;volume=143&amp;issue=3&amp;publication_year=1973&amp;pages=629-633&amp;pmid=4719448&amp;doi=10.3181/00379727-143-37380&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B58"><span class="mixed-citation">
Taylor BA, Wnek C, Kotlus BS, Roemer N, MacTaggart T, Phillips SJ. <span class="ref-title">Genotyping new BXD recombinant inbred mouse strains and comparison of BXD and consensus maps</span>. <span class="ref-journal">Mamm Genome</span>. 1999;<span class="ref-vol">10</span>(<span class="ref-iss">4</span>):335&#x02013;348. doi: 10.1007/s003359900998. [<a href="https://pubmed.ncbi.nlm.nih.gov/10087289">PubMed</a>] [<a href="//doi.org/10.1007%2Fs003359900998" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mamm+Genome&amp;title=Genotyping+new+BXD+recombinant+inbred+mouse+strains+and+comparison+of+BXD+and+consensus+maps&amp;volume=10&amp;issue=4&amp;publication_year=1999&amp;pages=335-348&amp;pmid=10087289&amp;doi=10.1007/s003359900998&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B59"><span class="mixed-citation">
Tsuyuguchi K, Suzuki K, Matsumoto H, Tanaka E, Amitani R, Kuze F. <span class="ref-title">Effect of oestrogen on <em>Mycobacterium avium</em> complex pulmonary infection in mice</span>. <span class="ref-journal">Clin Exp Immunol</span>. 2001;<span class="ref-vol">123</span>(<span class="ref-iss">3</span>):428&#x02013;434. doi: 10.1046/j.1365-2249.2001.01474.x. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1906003/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11298130">PubMed</a>] [<a href="//doi.org/10.1046%2Fj.1365-2249.2001.01474.x" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Exp+Immunol&amp;title=Effect+of+oestrogen+on+Mycobacterium+avium+complex+pulmonary+infection+in+mice&amp;volume=123&amp;issue=3&amp;publication_year=2001&amp;pages=428-434&amp;pmid=11298130&amp;doi=10.1046/j.1365-2249.2001.01474.x&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B60"><span class="mixed-citation">
Wang R, Kreutzfeldt K, Botella H, Vaubourgeix J, Schnappinger D, Ehrt S. <span class="ref-title">Persistent <em>Mycobacterium tuberculosis</em> infection in mice requires PerM for successful cell division</span>. <span class="ref-journal">eLife</span>. 2019;<span class="ref-vol">8</span>:e49570. doi: 10.7554/eLife.49570. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6872210/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31751212">PubMed</a>] [<a href="//doi.org/10.7554%2FeLife.49570" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Persistent+Mycobacterium+tuberculosis+infection+in+mice+requires+PerM+for+successful+cell+division&amp;volume=8&amp;publication_year=2019&amp;pages=e49570&amp;pmid=31751212&amp;doi=10.7554/eLife.49570&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B61"><span class="mixed-citation">
Wang X, Pandey AK, Mulligan MK, Williams EG, Mozhui K, Li Z, Jovaisaite V, Quarles LD, Xiao Z, Huang J, et al.. 
<span class="ref-title">Joint mouse&#x02013;human phenome-wide association to test gene function and disease risk</span>. <span class="ref-journal">Nat Commun</span>. 2016;<span class="ref-vol">7</span>(<span class="ref-iss">1</span>):10464. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4740880/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26833085">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Joint+mouse&#x02013;human+phenome-wide+association+to+test+gene+function+and+disease+risk&amp;volume=7&amp;issue=1&amp;publication_year=2016&amp;pages=10464&amp;pmid=26833085&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B62"><span class="mixed-citation">
Wheeler FC, Fernandez L, Carlson KM, Wolf MJ, Rockman HA, Marchuk DA. <span class="ref-title">QTL Mapping in a mouse model of cardiomyopathy reveals an ancestral modifier allele affecting heart function and survival</span>. <span class="ref-journal">Mamm Genome</span>. 2005;<span class="ref-vol">16</span>(<span class="ref-iss">6</span>):414&#x02013;423. doi: 10.1007/s00335-005-2468-7. [<a href="https://pubmed.ncbi.nlm.nih.gov/16075368">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00335-005-2468-7" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mamm+Genome&amp;title=QTL+Mapping+in+a+mouse+model+of+cardiomyopathy+reveals+an+ancestral+modifier+allele+affecting+heart+function+and+survival&amp;volume=16&amp;issue=6&amp;publication_year=2005&amp;pages=414-423&amp;pmid=16075368&amp;doi=10.1007/s00335-005-2468-7&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B63"><span class="mixed-citation">
Wheeler FC, Tang H, Marks OA, Hadnott TN, Chu P-L, Mao L, Rockman HA, Marchuk DA. <span class="ref-title"><em>Tnni3k</em> modifies disease progression in murine models of cardiomyopathy</span>. <span class="ref-journal">PLoS Genet</span>. 2009;<span class="ref-vol">5</span>(<span class="ref-iss">9</span>):e1000647. doi: 10.1371/journal.pgen.1000647. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2731170/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19763165">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pgen.1000647" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Tnni3k+modifies+disease+progression+in+murine+models+of+cardiomyopathy&amp;volume=5&amp;issue=9&amp;publication_year=2009&amp;pages=e1000647&amp;pmid=19763165&amp;doi=10.1371/journal.pgen.1000647&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B64"><span class="mixed-citation">
Whitaker M, Ruecker N, Hartman T, Klevorn T, Andres J, Kim J, Rhee K, Ehrt S. <span class="ref-title">Two interacting ATPases protect <em>Mycobacterium tuberculosis</em> from glycerol and nitric oxide toxicity</span>. <span class="ref-journal">J Bacteriol</span>. 2020;<span class="ref-vol">202</span>(<span class="ref-iss">16</span>):e00202-20. doi: 10.1128/JB.00202-20. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8404711/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32482725">PubMed</a>] [<a href="//doi.org/10.1128%2FJB.00202-20" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Two+interacting+ATPases+protect+Mycobacterium+tuberculosis+from+glycerol+and+nitric+oxide+toxicity&amp;volume=202&amp;issue=16&amp;publication_year=2020&amp;pages=e00202-20&amp;pmid=32482725&amp;doi=10.1128/JB.00202-20&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B65"><span class="mixed-citation">
WHO . Global Tuberculosis Report 2022. Geneva. 2022.</span></li><li id="jkad147-B66"><span class="mixed-citation">
Wilburn KM, Meade RK, Heckenberg EM, Dockterman J, Coers J, Sassetti CM, Olive AJ, Smith CM. <span class="ref-title">Differential requirement for IRGM proteins during tuberculosis infection in mice</span>. <span class="ref-journal">Infect Immun</span>. 2023;<span class="ref-vol">91</span>(<span class="ref-iss">2</span>):e0051022. doi: 10.1128/iai.00510-22. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9933630/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36629440">PubMed</a>] [<a href="//doi.org/10.1128%2Fiai.00510-22" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect+Immun&amp;title=Differential+requirement+for+IRGM+proteins+during+tuberculosis+infection+in+mice&amp;volume=91&amp;issue=2&amp;publication_year=2023&amp;pages=e0051022&amp;pmid=36629440&amp;doi=10.1128/iai.00510-22&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B67"><span class="mixed-citation">
Wiltshire SA, Leiva-Torres GA, Vidal SM. <span class="ref-title">Quantitative trait locus analysis, pathway analysis, and consomic mapping show genetic variants of Tnni3k, Fpgt, or H28 control susceptibility to viral myocarditis</span>. <span class="ref-journal">J Immunol.</span> 2011;<span class="ref-vol">186</span>(<span class="ref-iss">11</span>):6398&#x02013;6405. doi: 10.4049/jimmunol.1100159. [<a href="https://pubmed.ncbi.nlm.nih.gov/21525387">PubMed</a>] [<a href="//doi.org/10.4049%2Fjimmunol.1100159" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol.&amp;title=Quantitative+trait+locus+analysis,+pathway+analysis,+and+consomic+mapping+show+genetic+variants+of+Tnni3k,+Fpgt,+or+H28+control+susceptibility+to+viral+myocarditis&amp;volume=186&amp;issue=11&amp;publication_year=2011&amp;pages=6398-6405&amp;pmid=21525387&amp;doi=10.4049/jimmunol.1100159&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B68"><span class="mixed-citation">
Wu Y, Williams EG, Dubuis S, Mottis A, Jovaisaite V, Houten SM, Argmann CA, Faridi P, Wolski W, Kutalik Z, et al.. 
<span class="ref-title">Multilayered genetic and omics dissection of mitochondrial activity in a mouse reference population</span>. <span class="ref-journal">Cell</span>. 2014;<span class="ref-vol">158</span>(<span class="ref-iss">6</span>):1415&#x02013;1430. doi: 10.1016/j.cell.2014.07.039. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4179868/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25215496">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2014.07.039" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Multilayered+genetic+and+omics+dissection+of+mitochondrial+activity+in+a+mouse+reference+population&amp;volume=158&amp;issue=6&amp;publication_year=2014&amp;pages=1415-1430&amp;pmid=25215496&amp;doi=10.1016/j.cell.2014.07.039&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B69"><span class="mixed-citation">
Yan B-S, Kirby A, Shebzukhov YV, Daly MJ, Kramnik I. <span class="ref-title">Genetic architecture of tuberculosis resistance in a mouse model of infection</span>. <span class="ref-journal">Genes Immun</span>. 2006;<span class="ref-vol">7</span>(<span class="ref-iss">3</span>):201&#x02013;210. doi: 10.1038/sj.gene.6364288. [<a href="https://pubmed.ncbi.nlm.nih.gov/16452998">PubMed</a>] [<a href="//doi.org/10.1038%2Fsj.gene.6364288" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=Genetic+architecture+of+tuberculosis+resistance+in+a+mouse+model+of+infection&amp;volume=7&amp;issue=3&amp;publication_year=2006&amp;pages=201-210&amp;pmid=16452998&amp;doi=10.1038/sj.gene.6364288&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B70"><span class="mixed-citation">
Zak DE, Penn-Nicholson A, Scriba TJ, Thompson E, Suliman S, Amon LM, Mahomed H, Erasmus M, Whatney W, Hussey GD, et al.. 
<span class="ref-title">A blood RNA signature for tuberculosis disease risk: a prospective cohort study</span>. <span class="ref-journal">Lancet</span>. 2016;<span class="ref-vol">387</span>(<span class="ref-iss">10035</span>):2312&#x02013;2322. doi: 10.1016/S0140-6736(15)01316-1. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5392204/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27017310">PubMed</a>] [<a href="//doi.org/10.1016%2FS0140-6736(15)01316-1" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=A+blood+RNA+signature+for+tuberculosis+disease+risk:+a+prospective+cohort+study&amp;volume=387&amp;issue=10035&amp;publication_year=2016&amp;pages=2312-2322&amp;pmid=27017310&amp;doi=10.1016/S0140-6736(15)01316-1&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li><li id="jkad147-B71"><span class="mixed-citation">
Zhang J, Lan Y, Li MY, Lamers MM, Fusade-Boyer M, Klemm E, Thiele C, Ashour J, Sanyal S. <span class="ref-title">Flaviviruses exploit the lipid droplet protein AUP1 to trigger lipophagy and drive virus production</span>. <span class="ref-journal">Cell Host Microbe</span>. 2018;<span class="ref-vol">23</span>(<span class="ref-iss">6</span>):819&#x02013;831.e5. doi: 10.1016/j.chom.2018.05.005. [<a href="https://pubmed.ncbi.nlm.nih.gov/29902443">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.chom.2018.05.005" target="_blank" rel="noopener noreferrer">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Host+Microbe&amp;title=Flaviviruses+exploit+the+lipid+droplet+protein+AUP1+to+trigger+lipophagy+and+drive+virus+production&amp;volume=23&amp;issue=6&amp;publication_year=2018&amp;pages=819-831.e5&amp;pmid=29902443&amp;doi=10.1016/j.chom.2018.05.005&amp;" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</span></span></li></ul></div></div></article><article data-type="fig" id="ob-jkad147-F1"><div class="fm-sec"><div class="fm-flexbox"><div class="fm-citation"><div>G3 (Bethesda). 2023 Sep; 13(9): jkad147. </div><div><span class="fm-vol-iss-date">Published online 2023 Jul 5. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fg3journal%2Fjkad147" target="_blank" rel="noopener noreferrer">10.1093/g3journal/jkad147</a></span></div></div></div><div class="half_rhythm permissions"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140614559725968_cl">Copyright/License</a><span class="fm-reuse"><a href="/pmc/about/copyright/" target="_blank" rel="noopener noreferrer">Request permission to reuse</a></span><div class="hide" id="idm140614559725968_cl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2023. Published by Oxford University Press on behalf of The Genetics Society of America.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div></div></div><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="jkad147-F1"><h3>Fig. 1.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140614559716640"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10468300_jkad147f1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jkad147f1.jpg" title="Click on image to zoom" class="tileshop" data-src="/pmc/articles/PMC10468300/bin/jkad147f1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140614559716640"><a target="object" rel="noopener" href="/pmc/articles/PMC10468300/figure/jkad147-F1/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><p>B6 and D2 are phenodeviant in TB susceptibility. a) Log<sub>10</sub> transformed CFUs recovered from lung tissue throughout the course of the aerosol infection (with an infectious dose of &#x0223c;40 CFU of <em>Mtb</em> H37Rv). b) Log<sub>10</sub> transformed CFU recovered from spleen throughout infection. c and d) Male B6 and D2 H&#x00026;E-stained lung sections taken 6-week postinfection, 2&#x000d7; magnification, representative of <em>n</em> = 4 per strain. Corresponding female sections can be found in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Fig. 1c and d</a>. e) Lung damage was quantified by QuPath v0.3.2 using an artificial neural network&#x02013;based damage identification model and reported as a percent of total lung section area (<em>n</em> = 4 per genotype and sex). Lung damage is not significantly different between B6 and D2 at 6-week postinfection by unpaired Student's <em>t</em>-test. f and g) Male B6 and D2 H&#x00026;E-stained lung sections taken 12-week postinfection, 2&#x000d7; magnification, representative of <em>n</em> = 4 per strain. Corresponding female sections can be found in <a href="#sup1" rid="sup1" class=" supplementary-material">Supplementary Fig. 1e and f</a>. h) Lung damage, quantified as in e), is significantly different between B6 and D2 at 12-week postinfection by unpaired Student's <em>t</em>-test.</p></div></div></article><article data-type="fig" id="ob-jkad147-F2"><div class="fm-sec"><div class="fm-flexbox"><div class="fm-citation"><div>G3 (Bethesda). 2023 Sep; 13(9): jkad147. </div><div><span class="fm-vol-iss-date">Published online 2023 Jul 5. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fg3journal%2Fjkad147" target="_blank" rel="noopener noreferrer">10.1093/g3journal/jkad147</a></span></div></div></div><div class="half_rhythm permissions"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140614517164544_cl">Copyright/License</a><span class="fm-reuse"><a href="/pmc/about/copyright/" target="_blank" rel="noopener noreferrer">Request permission to reuse</a></span><div class="hide" id="idm140614517164544_cl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2023. Published by Oxford University Press on behalf of The Genetics Society of America.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div></div></div><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="jkad147-F2"><h3>Fig. 2.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140614517160128"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10468300_jkad147f2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jkad147f2.jpg" title="Click on image to zoom" class="tileshop" data-src="/pmc/articles/PMC10468300/bin/jkad147f2.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140614517160128"><a target="object" rel="noopener" href="/pmc/articles/PMC10468300/figure/jkad147-F2/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><p>BXD panel exhibits natural genetic variation, but classical clinical traits cannot map genome-wide QTL at 4-week postinfection. a) The BXD panel is a biparental recombinant inbred panel bred from C57BL/6J (B6) and DBA/2J (D2) and composed of over 100 mosaic strains. b) Lung and spleen burden by host genotype (<em>n</em> = 1&#x02013;5 per genotype). c) QTL mapping of lung and spleen burden traits across the BXD cohort. Solid threshold represents a genome-wide significance of <em>P</em> = 0.05. Dashed threshold represents <em>P</em> = 0.20. Thresholds were calculated from 10,000 permutation tests.</p></div></div></article><article data-type="fig" id="ob-jkad147-F3"><div class="fm-sec"><div class="fm-flexbox"><div class="fm-citation"><div>G3 (Bethesda). 2023 Sep; 13(9): jkad147. </div><div><span class="fm-vol-iss-date">Published online 2023 Jul 5. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fg3journal%2Fjkad147" target="_blank" rel="noopener noreferrer">10.1093/g3journal/jkad147</a></span></div></div></div><div class="half_rhythm permissions"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140614517155920_cl">Copyright/License</a><span class="fm-reuse"><a href="/pmc/about/copyright/" target="_blank" rel="noopener noreferrer">Request permission to reuse</a></span><div class="hide" id="idm140614517155920_cl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2023. Published by Oxford University Press on behalf of The Genetics Society of America.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div></div></div><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="jkad147-F3"><h3>Fig. 3.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140614522457808"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10468300_jkad147f3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jkad147f3.jpg" title="Click on image to zoom" class="tileshop" data-src="/pmc/articles/PMC10468300/bin/jkad147f3.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140614522457808"><a target="object" rel="noopener" href="/pmc/articles/PMC10468300/figure/jkad147-F3/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><p>TnSeq mutant fitness is a sensitive reporter of the factors that drive disease outcome. a) Each BXD and parental strain was infected with 10<sup>6</sup> CFU of <em>Mtb</em> TnSeq library, representative of &#x0223c;10<sup>5</sup> independent <em>Mtb</em> Tn mutants. For each mutant, the change between the initial abundance in the TnSeq library and the abundance recovered after 4 weeks of in vivo infection was compared with calculate a quantitative fitness metric (LFC), which could be used downstream for QTL mapping. b) Example plot showing an essential <em>Mtb</em> gene (<em>bioA</em>), a conditionally essential gene (<em>mbtG</em>), and a nonessential gene (<em>glpK</em>) across the BXD panel. c) A Venn diagram depicting the overlap of essential genes between B6, D2, and the screened BXD strains. <em>Mtb</em> genes were deemed essential if the Tn mutant experienced a 1-fold reduction (LFC &#x0003c; &#x02212;0.5) within a given host strain and were significantly different from in vitro conditions (<em>P</em> &#x0003c; 0.05) in at least 1 host strain across the panel. d) The number of <em>Mtb</em> genes essential (LFC &#x0003c; &#x02212;0.5) for growth or survival in each diverse mouse strain across the panel (<em>P</em> &#x0003c; 0.05). Salmon (smallest circle) indicates the mutants uniquely required for each host additional strain, and purple (medium circle) shows the cumulative requirement as each new host strain is added. e) A heatmap depicting the total correlation of the library-wide bacterial fitness within each screened host strain.</p></div></div></article><article data-type="fig" id="ob-jkad147-F4"><div class="fm-sec"><div class="fm-flexbox"><div class="fm-citation"><div>G3 (Bethesda). 2023 Sep; 13(9): jkad147. </div><div><span class="fm-vol-iss-date">Published online 2023 Jul 5. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fg3journal%2Fjkad147" target="_blank" rel="noopener noreferrer">10.1093/g3journal/jkad147</a></span></div></div></div><div class="half_rhythm permissions"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140614523105536_cl">Copyright/License</a><span class="fm-reuse"><a href="/pmc/about/copyright/" target="_blank" rel="noopener noreferrer">Request permission to reuse</a></span><div class="hide" id="idm140614523105536_cl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2023. Published by Oxford University Press on behalf of The Genetics Society of America.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div></div></div><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="jkad147-F4"><h3>Fig. 4.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140614523101120"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10468300_jkad147f4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jkad147f4.jpg" title="Click on image to zoom" class="tileshop" data-src="/pmc/articles/PMC10468300/bin/jkad147f4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140614523101120"><a target="object" rel="noopener" href="/pmc/articles/PMC10468300/figure/jkad147-F4/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><p>TnSeq mutant fitness endophenotypes map genome-wide significant QTL. a) The QTL that reach a significance threshold of <em>P</em> &#x02264; 0.01. The color of the QTL corresponds to the Mycobrowser class of the gene absent from the Tn mutant that mapped the QTL. The width of each segment corresponds to the size of the 95% Bayesian confidence interval of the QTL as calculated by R/qtl2. b) A pie chart tabulating the Mycobrowser classes of the mutants that mapped highly significant QTL (<em>P</em> &#x02264; 0.01), beyond the genome-wide significance threshold of <em>P</em> &#x02264; 0.05.</p></div></div></article><article data-type="table-wrap" id="ob-jkad147-T1"><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="jkad147-T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p>Genome-wide significant QTL (<em>P</em> &#x02264; 0.05).</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="1" colspan="1">
<em>hp</em>QTL</th><th align="center" rowspan="1" colspan="1">Trait</th><th align="center" rowspan="1" colspan="1">Gene</th><th align="center" rowspan="1" colspan="1">Chr</th><th align="center" rowspan="1" colspan="1">Position (Mb)</th><th align="center" rowspan="1" colspan="1">LOD</th><th align="center" rowspan="1" colspan="1">Start (Mb)</th><th align="center" rowspan="1" colspan="1">End (Mb)</th><th align="center" rowspan="1" colspan="1">
<em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">hp001</td><td rowspan="1" colspan="1">Rv0913c</td><td rowspan="1" colspan="1">
<em>Rv0913c</em>
</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">22.773</td><td rowspan="1" colspan="1">3.52</td><td rowspan="1" colspan="1">5.022</td><td rowspan="1" colspan="1">73.222</td><td rowspan="1" colspan="1">2.24E-02</td></tr><tr><td rowspan="1" colspan="1">hp002</td><td rowspan="1" colspan="1">Rv0778</td><td rowspan="1" colspan="1">
<em>cyp126</em>
</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">121.148</td><td rowspan="1" colspan="1">3.98</td><td rowspan="1" colspan="1">115.474</td><td rowspan="1" colspan="1">126.340</td><td rowspan="1" colspan="1">1.67E-02</td></tr><tr><td rowspan="1" colspan="1">hp003</td><td rowspan="1" colspan="1">Rv2468c</td><td rowspan="1" colspan="1">
<em>Rv2468c</em>
</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">152.965</td><td rowspan="1" colspan="1">4.54</td><td rowspan="1" colspan="1">147.055</td><td rowspan="1" colspan="1">166.299</td><td rowspan="1" colspan="1">2.33E-02</td></tr><tr><td rowspan="1" colspan="1">hp004</td><td rowspan="1" colspan="1">Rv1260</td><td rowspan="1" colspan="1">
<em>Rv1260</em>
</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">155.701</td><td rowspan="1" colspan="1">4.29</td><td rowspan="1" colspan="1">143.711</td><td rowspan="1" colspan="1">160.273</td><td rowspan="1" colspan="1">3.94E-02</td></tr><tr><td rowspan="1" colspan="1">hp005</td><td rowspan="1" colspan="1">Rv0103c</td><td rowspan="1" colspan="1">
<em>ctpB</em>
</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">189.196</td><td rowspan="1" colspan="1">3.09</td><td rowspan="1" colspan="1">3.010</td><td rowspan="1" colspan="1">195.308</td><td rowspan="1" colspan="1">4.60E-02</td></tr><tr><td rowspan="1" colspan="1">hp006</td><td rowspan="1" colspan="1">Rv1525</td><td rowspan="1" colspan="1">
<em>wbbL2</em>
</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">10.808</td><td rowspan="1" colspan="1">3.56</td><td rowspan="1" colspan="1">5.653</td><td rowspan="1" colspan="1">14.395</td><td rowspan="1" colspan="1">3.32E-02</td></tr><tr><td rowspan="1" colspan="1">hp007</td><td rowspan="1" colspan="1">Rv3400</td><td rowspan="1" colspan="1">
<em>Rv3400</em>
</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">24.076</td><td rowspan="1" colspan="1">4.12</td><td rowspan="1" colspan="1">17.898</td><td rowspan="1" colspan="1">42.375</td><td rowspan="1" colspan="1">8.38E-03</td></tr><tr><td rowspan="1" colspan="1">hp008</td><td rowspan="1" colspan="1">Rv0625c</td><td rowspan="1" colspan="1">
<em>Rv0625c</em>
</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">45.533</td><td rowspan="1" colspan="1">3.37</td><td rowspan="1" colspan="1">25.645</td><td rowspan="1" colspan="1">161.538</td><td rowspan="1" colspan="1">5.01E-02</td></tr><tr><td rowspan="1" colspan="1">hp009</td><td rowspan="1" colspan="1">Rv0599c</td><td rowspan="1" colspan="1">
<em>vapB27</em>
</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">46.212</td><td rowspan="1" colspan="1">3.74</td><td rowspan="1" colspan="1">42.557</td><td rowspan="1" colspan="1">54.975</td><td rowspan="1" colspan="1">7.86E-03</td></tr><tr><td rowspan="1" colspan="1">hp010</td><td rowspan="1" colspan="1">Rv3294c</td><td rowspan="1" colspan="1">
<em>Rv3294c</em>
</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">138.850</td><td rowspan="1" colspan="1">3.56</td><td rowspan="1" colspan="1">136.128</td><td rowspan="1" colspan="1">158.741</td><td rowspan="1" colspan="1">3.73E-02</td></tr><tr><td rowspan="1" colspan="1">hp011</td><td rowspan="1" colspan="1">Rv3565</td><td rowspan="1" colspan="1">
<em>aspB</em>
</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">159.316</td><td rowspan="1" colspan="1">3.88</td><td rowspan="1" colspan="1">42.557</td><td rowspan="1" colspan="1">174.820</td><td rowspan="1" colspan="1">1.47E-02</td></tr><tr><td rowspan="1" colspan="1">hp012</td><td rowspan="1" colspan="1">Rv2546</td><td rowspan="1" colspan="1">
<em>vapC18</em>
</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">169.114</td><td rowspan="1" colspan="1">4.90</td><td rowspan="1" colspan="1">168.446</td><td rowspan="1" colspan="1">169.887</td><td rowspan="1" colspan="1">1.80E-03</td></tr><tr><td rowspan="1" colspan="1">hp013</td><td rowspan="1" colspan="1">Rv1965</td><td rowspan="1" colspan="1">
<em>yrbE3B</em>
</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">170.362</td><td rowspan="1" colspan="1">3.50</td><td rowspan="1" colspan="1">15.435</td><td rowspan="1" colspan="1">180.835</td><td rowspan="1" colspan="1">1.43E-02</td></tr><tr><td rowspan="1" colspan="1">hp014</td><td rowspan="1" colspan="1">Rv1367c</td><td rowspan="1" colspan="1">
<em>Rv1367c</em>
</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">173.385</td><td rowspan="1" colspan="1">3.77</td><td rowspan="1" colspan="1">46.133</td><td rowspan="1" colspan="1">176.951</td><td rowspan="1" colspan="1">1.85E-02</td></tr><tr><td rowspan="1" colspan="1">hp015</td><td rowspan="1" colspan="1">Rv1290A</td><td rowspan="1" colspan="1">
<em>Rv1290A</em>
</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">181.009</td><td rowspan="1" colspan="1">2.96</td><td rowspan="1" colspan="1">14.698</td><td rowspan="1" colspan="1">181.991</td><td rowspan="1" colspan="1">2.36E-02</td></tr><tr><td rowspan="1" colspan="1">hp016</td><td rowspan="1" colspan="1">Rv3047c</td><td rowspan="1" colspan="1">
<em>Rv3047c</em>
</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">41.212</td><td rowspan="1" colspan="1">3.92</td><td rowspan="1" colspan="1">39.644</td><td rowspan="1" colspan="1">51.561</td><td rowspan="1" colspan="1">4.49E-02</td></tr><tr><td rowspan="1" colspan="1">hp017</td><td rowspan="1" colspan="1">Rv0101</td><td rowspan="1" colspan="1">
<em>nrp</em>
</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">57.642</td><td rowspan="1" colspan="1">3.69</td><td rowspan="1" colspan="1">42.035</td><td rowspan="1" colspan="1">61.219</td><td rowspan="1" colspan="1">6.19E-03</td></tr><tr><td rowspan="1" colspan="1">hp018</td><td rowspan="1" colspan="1">Rv2051c</td><td rowspan="1" colspan="1">
<em>ppm1</em>
</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">62.392</td><td rowspan="1" colspan="1">4.25</td><td rowspan="1" colspan="1">61.219</td><td rowspan="1" colspan="1">70.185</td><td rowspan="1" colspan="1">1.94E-02</td></tr><tr><td rowspan="1" colspan="1">hp019</td><td rowspan="1" colspan="1">Rv1935c</td><td rowspan="1" colspan="1">
<em>echA13</em>
</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">129.496</td><td rowspan="1" colspan="1">4.23</td><td rowspan="1" colspan="1">10.997</td><td rowspan="1" colspan="1">157.792</td><td rowspan="1" colspan="1">4.20E-02</td></tr><tr><td rowspan="1" colspan="1">hp020</td><td rowspan="1" colspan="1">Rv1735c</td><td rowspan="1" colspan="1">
<em>Rv1735c</em>
</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">129.508</td><td rowspan="1" colspan="1">3.64</td><td rowspan="1" colspan="1">117.504</td><td rowspan="1" colspan="1">157.140</td><td rowspan="1" colspan="1">4.20E-02</td></tr><tr><td rowspan="1" colspan="1">hp021</td><td rowspan="1" colspan="1">Rv2602</td><td rowspan="1" colspan="1">
<em>vapC41</em>
</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">132.174</td><td rowspan="1" colspan="1">4.01</td><td rowspan="1" colspan="1">129.509</td><td rowspan="1" colspan="1">138.245</td><td rowspan="1" colspan="1">1.12E-02</td></tr><tr><td rowspan="1" colspan="1">hp022</td><td rowspan="1" colspan="1">Rv0773c</td><td rowspan="1" colspan="1">
<em>ggtA</em>
</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">146.239</td><td rowspan="1" colspan="1">4.56</td><td rowspan="1" colspan="1">143.570</td><td rowspan="1" colspan="1">156.034</td><td rowspan="1" colspan="1">2.75E-02</td></tr><tr><td rowspan="1" colspan="1">hp023</td><td rowspan="1" colspan="1">Rv0077c</td><td rowspan="1" colspan="1">
<em>Rv0077c</em>
</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">144.942</td><td rowspan="1" colspan="1">4.31</td><td rowspan="1" colspan="1">144.913</td><td rowspan="1" colspan="1">148.864</td><td rowspan="1" colspan="1">1.04E-02</td></tr><tr><td rowspan="1" colspan="1">hp024</td><td rowspan="1" colspan="1">Rv0588</td><td rowspan="1" colspan="1">
<em>yrbE2B</em>
</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">153.793</td><td rowspan="1" colspan="1">3.76</td><td rowspan="1" colspan="1">54.951</td><td rowspan="1" colspan="1">155.498</td><td rowspan="1" colspan="1">3.23E-02</td></tr><tr><td rowspan="1" colspan="1">hp025</td><td rowspan="1" colspan="1">Rv0093c</td><td rowspan="1" colspan="1">
<em>Rv0093c</em>
</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">28.411</td><td rowspan="1" colspan="1">3.05</td><td rowspan="1" colspan="1">19.137</td><td rowspan="1" colspan="1">132.038</td><td rowspan="1" colspan="1">2.29E-02</td></tr><tr><td rowspan="1" colspan="1">hp026</td><td rowspan="1" colspan="1">Rv0768</td><td rowspan="1" colspan="1">
<em>aldA</em>
</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">43.085</td><td rowspan="1" colspan="1">4.19</td><td rowspan="1" colspan="1">29.537</td><td rowspan="1" colspan="1">52.408</td><td rowspan="1" colspan="1">3.09E-03</td></tr><tr><td rowspan="1" colspan="1">hp027</td><td rowspan="1" colspan="1">Rv2957</td><td rowspan="1" colspan="1">
<em>Rv2957</em>
</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">117.550</td><td rowspan="1" colspan="1">3.27</td><td rowspan="1" colspan="1">3.143</td><td rowspan="1" colspan="1">139.501</td><td rowspan="1" colspan="1">3.26E-02</td></tr><tr><td rowspan="1" colspan="1">hp028</td><td rowspan="1" colspan="1">Rv3848</td><td rowspan="1" colspan="1">
<em>Rv3848</em>
</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">136.015</td><td rowspan="1" colspan="1">5.69</td><td rowspan="1" colspan="1">133.911</td><td rowspan="1" colspan="1">138.657</td><td rowspan="1" colspan="1">1.55E-02</td></tr><tr><td rowspan="1" colspan="1">hp029</td><td rowspan="1" colspan="1">Rv0359</td><td rowspan="1" colspan="1">
<em>rip2</em>
</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">76.492</td><td rowspan="1" colspan="1">4.18</td><td rowspan="1" colspan="1">75.970</td><td rowspan="1" colspan="1">82.500</td><td rowspan="1" colspan="1">5.63E-03</td></tr><tr><td rowspan="1" colspan="1">hp030</td><td rowspan="1" colspan="1">Rv0955</td><td rowspan="1" colspan="1">
<em>perM</em>
</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">82.500</td><td rowspan="1" colspan="1">4.80</td><td rowspan="1" colspan="1">80.286</td><td rowspan="1" colspan="1">83.092</td><td rowspan="1" colspan="1">1.29E-02</td></tr><tr><td rowspan="1" colspan="1">hp031</td><td rowspan="1" colspan="1">Rv2140c</td><td rowspan="1" colspan="1">
<em>TB18.6</em>
</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">84.277</td><td rowspan="1" colspan="1">4.62</td><td rowspan="1" colspan="1">80.286</td><td rowspan="1" colspan="1">85.650</td><td rowspan="1" colspan="1">2.28E-02</td></tr><tr><td rowspan="1" colspan="1">hp032</td><td rowspan="1" colspan="1">Rv3849</td><td rowspan="1" colspan="1">
<em>espR</em>
</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">84.869</td><td rowspan="1" colspan="1">6.77</td><td rowspan="1" colspan="1">83.684</td><td rowspan="1" colspan="1">85.566</td><td rowspan="1" colspan="1">6.23E-03</td></tr><tr><td rowspan="1" colspan="1">hp033</td><td rowspan="1" colspan="1">Rv0127</td><td rowspan="1" colspan="1">
<em>mak</em>
</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">85.650</td><td rowspan="1" colspan="1">7.83</td><td rowspan="1" colspan="1">84.277</td><td rowspan="1" colspan="1">88.578</td><td rowspan="1" colspan="1">7.04E-03</td></tr><tr><td rowspan="1" colspan="1">hp034</td><td rowspan="1" colspan="1">Rv1056</td><td rowspan="1" colspan="1">
<em>Rv1056</em>
</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">93.600</td><td rowspan="1" colspan="1">4.10</td><td rowspan="1" colspan="1">89.371</td><td rowspan="1" colspan="1">95.127</td><td rowspan="1" colspan="1">4.40E-02</td></tr><tr><td rowspan="1" colspan="1">hp035</td><td rowspan="1" colspan="1">Rv2671</td><td rowspan="1" colspan="1">
<em>ribD</em>
</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">148.818</td><td rowspan="1" colspan="1">3.53</td><td rowspan="1" colspan="1">12.797</td><td rowspan="1" colspan="1">149.266</td><td rowspan="1" colspan="1">1.60E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp036</td><td rowspan="1" colspan="1">Rv3711c</td><td rowspan="1" colspan="1">
<em>dnaQ</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">27.300</td><td rowspan="1" colspan="1">3.31</td><td rowspan="1" colspan="1">24.735</td><td rowspan="1" colspan="1">145.321</td><td rowspan="1" colspan="1">3.80E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp037</td><td rowspan="1" colspan="1">Rv3299c</td><td rowspan="1" colspan="1">
<em>atsB</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">36.692</td><td rowspan="1" colspan="1">3.01</td><td rowspan="1" colspan="1">3.078</td><td rowspan="1" colspan="1">144.446</td><td rowspan="1" colspan="1">5.34E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp038</td><td rowspan="1" colspan="1">Rv0065</td><td rowspan="1" colspan="1">
<em>vapC1</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">37.008</td><td rowspan="1" colspan="1">3.41</td><td rowspan="1" colspan="1">3.078</td><td rowspan="1" colspan="1">144.445</td><td rowspan="1" colspan="1">4.72E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp039</td><td rowspan="1" colspan="1">Rv1530</td><td rowspan="1" colspan="1">
<em>adh</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">37.008</td><td rowspan="1" colspan="1">3.22</td><td rowspan="1" colspan="1">3.078</td><td rowspan="1" colspan="1">145.321</td><td rowspan="1" colspan="1">3.74E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp040</td><td rowspan="1" colspan="1">Rv2482c</td><td rowspan="1" colspan="1">
<em>plsB2</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">37.008</td><td rowspan="1" colspan="1">3.55</td><td rowspan="1" colspan="1">24.619</td><td rowspan="1" colspan="1">144.445</td><td rowspan="1" colspan="1">2.35E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp041</td><td rowspan="1" colspan="1">Rv0026</td><td rowspan="1" colspan="1">
<em>Rv0026</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">37.760</td><td rowspan="1" colspan="1">3.34</td><td rowspan="1" colspan="1">24.764</td><td rowspan="1" colspan="1">128.756</td><td rowspan="1" colspan="1">2.82E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp042</td><td rowspan="1" colspan="1">Rv1913</td><td rowspan="1" colspan="1">
<em>Rv1913</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">39.822</td><td rowspan="1" colspan="1">3.63</td><td rowspan="1" colspan="1">30.689</td><td rowspan="1" colspan="1">145.321</td><td rowspan="1" colspan="1">2.96E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp043</td><td rowspan="1" colspan="1">Rv1236</td><td rowspan="1" colspan="1">
<em>sugA</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">40.289</td><td rowspan="1" colspan="1">3.13</td><td rowspan="1" colspan="1">24.822</td><td rowspan="1" colspan="1">140.297</td><td rowspan="1" colspan="1">5.28E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp044</td><td rowspan="1" colspan="1">Rv1528c</td><td rowspan="1" colspan="1">
<em>papA4</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">40.289</td><td rowspan="1" colspan="1">4.97</td><td rowspan="1" colspan="1">40.226</td><td rowspan="1" colspan="1">46.357</td><td rowspan="1" colspan="1">2.11E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp045</td><td rowspan="1" colspan="1">Rv1542c</td><td rowspan="1" colspan="1">
<em>glbN</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">40.289</td><td rowspan="1" colspan="1">3.71</td><td rowspan="1" colspan="1">3.078</td><td rowspan="1" colspan="1">145.321</td><td rowspan="1" colspan="1">5.77E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp046</td><td rowspan="1" colspan="1">Rv1992c</td><td rowspan="1" colspan="1">
<em>ctpG</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">40.289</td><td rowspan="1" colspan="1">3.24</td><td rowspan="1" colspan="1">6.015</td><td rowspan="1" colspan="1">145.321</td><td rowspan="1" colspan="1">3.20E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp047</td><td rowspan="1" colspan="1">Rv3702c</td><td rowspan="1" colspan="1">
<em>Rv3702c</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">40.289</td><td rowspan="1" colspan="1">4.55</td><td rowspan="1" colspan="1">24.793</td><td rowspan="1" colspan="1">140.297</td><td rowspan="1" colspan="1">2.75E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp048</td><td rowspan="1" colspan="1">Rv2388c</td><td rowspan="1" colspan="1">
<em>hemN</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">84.014</td><td rowspan="1" colspan="1">3.39</td><td rowspan="1" colspan="1">39.894</td><td rowspan="1" colspan="1">127.779</td><td rowspan="1" colspan="1">2.54E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp049</td><td rowspan="1" colspan="1">Rv0852</td><td rowspan="1" colspan="1">
<em>fadD16</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">97.465</td><td rowspan="1" colspan="1">3.71</td><td rowspan="1" colspan="1">83.879</td><td rowspan="1" colspan="1">128.058</td><td rowspan="1" colspan="1">3.89E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp050</td><td rowspan="1" colspan="1">Rv0878c</td><td rowspan="1" colspan="1">
<em>PPE13</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">97.685</td><td rowspan="1" colspan="1">4.10</td><td rowspan="1" colspan="1">89.870</td><td rowspan="1" colspan="1">127.500</td><td rowspan="1" colspan="1">2.49E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp051</td><td rowspan="1" colspan="1">Rv0089</td><td rowspan="1" colspan="1">
<em>Rv0089</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">116.709</td><td rowspan="1" colspan="1">4.08</td><td rowspan="1" colspan="1">98.124</td><td rowspan="1" colspan="1">127.989</td><td rowspan="1" colspan="1">3.64E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp052</td><td rowspan="1" colspan="1">Rv1912c</td><td rowspan="1" colspan="1">
<em>fadB5</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">134.484</td><td rowspan="1" colspan="1">4.49</td><td rowspan="1" colspan="1">51.060</td><td rowspan="1" colspan="1">134.831</td><td rowspan="1" colspan="1">1.14E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp053</td><td rowspan="1" colspan="1">Rv3868</td><td rowspan="1" colspan="1">
<em>eccA1</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">134.598</td><td rowspan="1" colspan="1">3.80</td><td rowspan="1" colspan="1">17.136</td><td rowspan="1" colspan="1">144.445</td><td rowspan="1" colspan="1">4.40E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp054</td><td rowspan="1" colspan="1">Rv2637</td><td rowspan="1" colspan="1">
<em>dedA</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">140.297</td><td rowspan="1" colspan="1">4.63</td><td rowspan="1" colspan="1">138.137</td><td rowspan="1" colspan="1">140.586</td><td rowspan="1" colspan="1">1.63E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp055</td><td rowspan="1" colspan="1">Rv0271c</td><td rowspan="1" colspan="1">
<em>fadE6</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">140.586</td><td rowspan="1" colspan="1">5.74</td><td rowspan="1" colspan="1">140.586</td><td rowspan="1" colspan="1">143.684</td><td rowspan="1" colspan="1">3.53E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp056</td><td rowspan="1" colspan="1">Rv1984c</td><td rowspan="1" colspan="1">
<em>cfp21</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">140.586</td><td rowspan="1" colspan="1">3.44</td><td rowspan="1" colspan="1">24.938</td><td rowspan="1" colspan="1">144.445</td><td rowspan="1" colspan="1">1.68E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp057</td><td rowspan="1" colspan="1">Rv1104</td><td rowspan="1" colspan="1">
<em>Rv1104</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">143.809</td><td rowspan="1" colspan="1">3.29</td><td rowspan="1" colspan="1">24.735</td><td rowspan="1" colspan="1">145.321</td><td rowspan="1" colspan="1">2.49E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp058</td><td rowspan="1" colspan="1">Rv3179</td><td rowspan="1" colspan="1">
<em>Rv3179</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">143.809</td><td rowspan="1" colspan="1">3.78</td><td rowspan="1" colspan="1">140.586</td><td rowspan="1" colspan="1">144.445</td><td rowspan="1" colspan="1">5.34E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp059</td><td rowspan="1" colspan="1">Rv1748</td><td rowspan="1" colspan="1">
<em>Rv1748</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">144.184</td><td rowspan="1" colspan="1">5.42</td><td rowspan="1" colspan="1">37.760</td><td rowspan="1" colspan="1">144.446</td><td rowspan="1" colspan="1">7.25E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp060</td><td rowspan="1" colspan="1">Rv3288c</td><td rowspan="1" colspan="1">
<em>usfY</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">144.184</td><td rowspan="1" colspan="1">3.97</td><td rowspan="1" colspan="1">30.593</td><td rowspan="1" colspan="1">145.321</td><td rowspan="1" colspan="1">3.46E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp061</td><td rowspan="1" colspan="1">Rv3698</td><td rowspan="1" colspan="1">
<em>Rv3698</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">144.309</td><td rowspan="1" colspan="1">3.41</td><td rowspan="1" colspan="1">12.026</td><td rowspan="1" colspan="1">145.321</td><td rowspan="1" colspan="1">3.60E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp062</td><td rowspan="1" colspan="1">Rv1245c</td><td rowspan="1" colspan="1">
<em>Rv1245c</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">144.446</td><td rowspan="1" colspan="1">3.97</td><td rowspan="1" colspan="1">40.289</td><td rowspan="1" colspan="1">145.321</td><td rowspan="1" colspan="1">2.62E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp063</td><td rowspan="1" colspan="1">Rv2092c</td><td rowspan="1" colspan="1">
<em>helY</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">144.687</td><td rowspan="1" colspan="1">5.34</td><td rowspan="1" colspan="1">144.533</td><td rowspan="1" colspan="1">145.321</td><td rowspan="1" colspan="1">2.50E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp064</td><td rowspan="1" colspan="1">Rv0458</td><td rowspan="1" colspan="1">
<em>Rv0458</em>
</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">145.321</td><td rowspan="1" colspan="1">3.92</td><td rowspan="1" colspan="1">144.448</td><td rowspan="1" colspan="1">145.321</td><td rowspan="1" colspan="1">2.93E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp065</td><td rowspan="1" colspan="1">Rv0538</td><td rowspan="1" colspan="1">
<em>Rv0538</em>
</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">11.279</td><td rowspan="1" colspan="1">4.57</td><td rowspan="1" colspan="1">11.279</td><td rowspan="1" colspan="1">13.680</td><td rowspan="1" colspan="1">1.47E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp066</td><td rowspan="1" colspan="1">Rv2066</td><td rowspan="1" colspan="1">
<em>cobI</em>
</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">14.819</td><td rowspan="1" colspan="1">4.31</td><td rowspan="1" colspan="1">12.880</td><td rowspan="1" colspan="1">15.343</td><td rowspan="1" colspan="1">4.00E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp067</td><td rowspan="1" colspan="1">Rv1039c</td><td rowspan="1" colspan="1">
<em>PPE15</em>
</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">16.687</td><td rowspan="1" colspan="1">4.55</td><td rowspan="1" colspan="1">13.680</td><td rowspan="1" colspan="1">16.946</td><td rowspan="1" colspan="1">1.31E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp068</td><td rowspan="1" colspan="1">Rv3855</td><td rowspan="1" colspan="1">
<em>ethR</em>
</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">87.897</td><td rowspan="1" colspan="1">4.62</td><td rowspan="1" colspan="1">87.219</td><td rowspan="1" colspan="1">89.367</td><td rowspan="1" colspan="1">1.44E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp069</td><td rowspan="1" colspan="1">Rv2161c</td><td rowspan="1" colspan="1">
<em>Rv2161c</em>
</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">113.708</td><td rowspan="1" colspan="1">4.09</td><td rowspan="1" colspan="1">109.407</td><td rowspan="1" colspan="1">128.590</td><td rowspan="1" colspan="1">5.93E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp070</td><td rowspan="1" colspan="1">Rv2384</td><td rowspan="1" colspan="1">
<em>mbtA</em>
</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">29.939</td><td rowspan="1" colspan="1">4.02</td><td rowspan="1" colspan="1">4.359</td><td rowspan="1" colspan="1">107.777</td><td rowspan="1" colspan="1">3.24E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp071</td><td rowspan="1" colspan="1">Rv1954c</td><td rowspan="1" colspan="1">
<em>Rv1954c</em>
</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">34.618</td><td rowspan="1" colspan="1">4.70</td><td rowspan="1" colspan="1">29.941</td><td rowspan="1" colspan="1">58.854</td><td rowspan="1" colspan="1">2.24E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp072</td><td rowspan="1" colspan="1">Rv3899c</td><td rowspan="1" colspan="1">
<em>Rv3899c</em>
</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">94.905</td><td rowspan="1" colspan="1">4.16</td><td rowspan="1" colspan="1">87.462</td><td rowspan="1" colspan="1">101.324</td><td rowspan="1" colspan="1">3.61E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp073</td><td rowspan="1" colspan="1">Rv0950c</td><td rowspan="1" colspan="1">
<em>Rv0950c</em>
</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">101.365</td><td rowspan="1" colspan="1">3.92</td><td rowspan="1" colspan="1">4.359</td><td rowspan="1" colspan="1">106.019</td><td rowspan="1" colspan="1">2.64E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp074</td><td rowspan="1" colspan="1">Rv0538</td><td rowspan="1" colspan="1">
<em>Rv0538</em>
</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">103.686</td><td rowspan="1" colspan="1">4.05</td><td rowspan="1" colspan="1">80.003</td><td rowspan="1" colspan="1">106.019</td><td rowspan="1" colspan="1">4.83E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp075</td><td rowspan="1" colspan="1">Rv3087</td><td rowspan="1" colspan="1">
<em>Rv3087</em>
</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">5.031</td><td rowspan="1" colspan="1">4.52</td><td rowspan="1" colspan="1">3.181</td><td rowspan="1" colspan="1">9.092</td><td rowspan="1" colspan="1">3.38E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp076</td><td rowspan="1" colspan="1">Rv2529</td><td rowspan="1" colspan="1">
<em>Rv2529</em>
</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">9.413</td><td rowspan="1" colspan="1">4.33</td><td rowspan="1" colspan="1">3.181</td><td rowspan="1" colspan="1">11.991</td><td rowspan="1" colspan="1">9.41E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp077</td><td rowspan="1" colspan="1">Rv1450c</td><td rowspan="1" colspan="1">
<em>PE_PGRS27</em>
</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">19.780</td><td rowspan="1" colspan="1">4.04</td><td rowspan="1" colspan="1">13.853</td><td rowspan="1" colspan="1">21.509</td><td rowspan="1" colspan="1">3.92E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp078</td><td rowspan="1" colspan="1">Rv1507c</td><td rowspan="1" colspan="1">
<em>Rv1507c</em>
</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">55.483</td><td rowspan="1" colspan="1">4.97</td><td rowspan="1" colspan="1">55.356</td><td rowspan="1" colspan="1">59.293</td><td rowspan="1" colspan="1">1.73E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp079</td><td rowspan="1" colspan="1">Rv1366</td><td rowspan="1" colspan="1">
<em>Rv1366</em>
</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">64.014</td><td rowspan="1" colspan="1">3.25</td><td rowspan="1" colspan="1">57.276</td><td rowspan="1" colspan="1">72.506</td><td rowspan="1" colspan="1">2.31E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp080</td><td rowspan="1" colspan="1">Rv1552</td><td rowspan="1" colspan="1">
<em>frdA</em>
</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">96.749</td><td rowspan="1" colspan="1">3.97</td><td rowspan="1" colspan="1">95.639</td><td rowspan="1" colspan="1">103.562</td><td rowspan="1" colspan="1">3.45E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp081</td><td rowspan="1" colspan="1">Rv1753c</td><td rowspan="1" colspan="1">
<em>PPE24</em>
</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">109.165</td><td rowspan="1" colspan="1">3.52</td><td rowspan="1" colspan="1">13.688</td><td rowspan="1" colspan="1">127.327</td><td rowspan="1" colspan="1">1.75E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp082</td><td rowspan="1" colspan="1">Rv2261c</td><td rowspan="1" colspan="1">
<em>Rv2261c</em>
</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">111.074</td><td rowspan="1" colspan="1">4.75</td><td rowspan="1" colspan="1">110.998</td><td rowspan="1" colspan="1">126.607</td><td rowspan="1" colspan="1">1.16E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp083</td><td rowspan="1" colspan="1">Rv0044c</td><td rowspan="1" colspan="1">
<em>Rv0044c</em>
</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">115.897</td><td rowspan="1" colspan="1">3.42</td><td rowspan="1" colspan="1">3.181</td><td rowspan="1" colspan="1">127.808</td><td rowspan="1" colspan="1">6.21E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp084</td><td rowspan="1" colspan="1">Rv2326c</td><td rowspan="1" colspan="1">
<em>Rv2326c</em>
</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">120.610</td><td rowspan="1" colspan="1">4.29</td><td rowspan="1" colspan="1">116.134</td><td rowspan="1" colspan="1">121.419</td><td rowspan="1" colspan="1">1.26E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp085</td><td rowspan="1" colspan="1">Rv0769</td><td rowspan="1" colspan="1">
<em>Rv0769</em>
</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">126.740</td><td rowspan="1" colspan="1">4.12</td><td rowspan="1" colspan="1">123.469</td><td rowspan="1" colspan="1">126.808</td><td rowspan="1" colspan="1">2.36E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp086</td><td rowspan="1" colspan="1">Rv2993c</td><td rowspan="1" colspan="1">
<em>Rv2993c</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">11.620</td><td rowspan="1" colspan="1">4.35</td><td rowspan="1" colspan="1">8.814</td><td rowspan="1" colspan="1">15.810</td><td rowspan="1" colspan="1">3.65E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp087</td><td rowspan="1" colspan="1">Rv3759c</td><td rowspan="1" colspan="1">
<em>proX</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">21.073</td><td rowspan="1" colspan="1">4.70</td><td rowspan="1" colspan="1">19.834</td><td rowspan="1" colspan="1">34.416</td><td rowspan="1" colspan="1">4.71E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp088</td><td rowspan="1" colspan="1">Rv0086</td><td rowspan="1" colspan="1">
<em>hycQ</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">24.850</td><td rowspan="1" colspan="1">3.37</td><td rowspan="1" colspan="1">18.284</td><td rowspan="1" colspan="1">35.583</td><td rowspan="1" colspan="1">1.22E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp089</td><td rowspan="1" colspan="1">Rv3841</td><td rowspan="1" colspan="1">
<em>bfrB</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">47.049</td><td rowspan="1" colspan="1">3.28</td><td rowspan="1" colspan="1">35.903</td><td rowspan="1" colspan="1">80.586</td><td rowspan="1" colspan="1">1.67E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp090</td><td rowspan="1" colspan="1">Rv0126</td><td rowspan="1" colspan="1">
<em>treS</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">65.541</td><td rowspan="1" colspan="1">3.56</td><td rowspan="1" colspan="1">64.895</td><td rowspan="1" colspan="1">81.165</td><td rowspan="1" colspan="1">3.85E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp091</td><td rowspan="1" colspan="1">Rv2116</td><td rowspan="1" colspan="1">
<em>lppK</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">73.243</td><td rowspan="1" colspan="1">4.46</td><td rowspan="1" colspan="1">73.194</td><td rowspan="1" colspan="1">81.165</td><td rowspan="1" colspan="1">1.67E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp092</td><td rowspan="1" colspan="1">Rv0765c</td><td rowspan="1" colspan="1">
<em>Rv0765c</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">106.661</td><td rowspan="1" colspan="1">4.70</td><td rowspan="1" colspan="1">106.040</td><td rowspan="1" colspan="1">107.457</td><td rowspan="1" colspan="1">2.58E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp093</td><td rowspan="1" colspan="1">Rv1850</td><td rowspan="1" colspan="1">
<em>ureC</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">107.742</td><td rowspan="1" colspan="1">4.25</td><td rowspan="1" colspan="1">107.422</td><td rowspan="1" colspan="1">110.228</td><td rowspan="1" colspan="1">1.61E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp094</td><td rowspan="1" colspan="1">Rv2802c</td><td rowspan="1" colspan="1">
<em>Rv2802c</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">107.742</td><td rowspan="1" colspan="1">4.87</td><td rowspan="1" colspan="1">107.133</td><td rowspan="1" colspan="1">109.751</td><td rowspan="1" colspan="1">9.90E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp095</td><td rowspan="1" colspan="1">Rv1452c</td><td rowspan="1" colspan="1">
<em>PE_PGRS28</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">107.964</td><td rowspan="1" colspan="1">3.29</td><td rowspan="1" colspan="1">103.707</td><td rowspan="1" colspan="1">111.341</td><td rowspan="1" colspan="1">4.10E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp096</td><td rowspan="1" colspan="1">Rv0599c</td><td rowspan="1" colspan="1">
<em>vapB27</em>
</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">113.557</td><td rowspan="1" colspan="1">3.58</td><td rowspan="1" colspan="1">110.546</td><td rowspan="1" colspan="1">118.682</td><td rowspan="1" colspan="1">1.76E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp097</td><td rowspan="1" colspan="1">Rv1969</td><td rowspan="1" colspan="1">
<em>mce3D</em>
</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">46.628</td><td rowspan="1" colspan="1">4.14</td><td rowspan="1" colspan="1">27.823</td><td rowspan="1" colspan="1">52.908</td><td rowspan="1" colspan="1">4.09E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp098</td><td rowspan="1" colspan="1">Rv3554</td><td rowspan="1" colspan="1">
<em>fdxB</em>
</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">71.873</td><td rowspan="1" colspan="1">3.22</td><td rowspan="1" colspan="1">58.970</td><td rowspan="1" colspan="1">114.110</td><td rowspan="1" colspan="1">4.64E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp099</td><td rowspan="1" colspan="1">Rv1448c</td><td rowspan="1" colspan="1">
<em>tal</em>
</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">79.511</td><td rowspan="1" colspan="1">4.90</td><td rowspan="1" colspan="1">76.378</td><td rowspan="1" colspan="1">86.909</td><td rowspan="1" colspan="1">1.38E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp100</td><td rowspan="1" colspan="1">Rv2922c</td><td rowspan="1" colspan="1">
<em>smc</em>
</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">105.499</td><td rowspan="1" colspan="1">4.57</td><td rowspan="1" colspan="1">104.281</td><td rowspan="1" colspan="1">107.191</td><td rowspan="1" colspan="1">3.70E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp101</td><td rowspan="1" colspan="1">Rv2268c</td><td rowspan="1" colspan="1">
<em>cyp128</em>
</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">107.369</td><td rowspan="1" colspan="1">3.62</td><td rowspan="1" colspan="1">5.594</td><td rowspan="1" colspan="1">107.848</td><td rowspan="1" colspan="1">3.59E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp102</td><td rowspan="1" colspan="1">Rv1775</td><td rowspan="1" colspan="1">
<em>Rv1775</em>
</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">108.105</td><td rowspan="1" colspan="1">3.71</td><td rowspan="1" colspan="1">107.369</td><td rowspan="1" colspan="1">108.898</td><td rowspan="1" colspan="1">3.33E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp103</td><td rowspan="1" colspan="1">Rv3164c</td><td rowspan="1" colspan="1">
<em>moxR3</em>
</td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">114.166</td><td rowspan="1" colspan="1">4.13</td><td rowspan="1" colspan="1">111.180</td><td rowspan="1" colspan="1">116.685</td><td rowspan="1" colspan="1">3.60E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp104</td><td rowspan="1" colspan="1">Rv2019</td><td rowspan="1" colspan="1">
<em>Rv2019</em>
</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">6.304</td><td rowspan="1" colspan="1">3.82</td><td rowspan="1" colspan="1">4.432</td><td rowspan="1" colspan="1">115.537</td><td rowspan="1" colspan="1">4.09E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp105</td><td rowspan="1" colspan="1">Rv2379c</td><td rowspan="1" colspan="1">
<em>mbtF</em>
</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">18.035</td><td rowspan="1" colspan="1">3.66</td><td rowspan="1" colspan="1">4.432</td><td rowspan="1" colspan="1">45.238</td><td rowspan="1" colspan="1">3.69E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp106</td><td rowspan="1" colspan="1">Rv0066c</td><td rowspan="1" colspan="1">
<em>icd2</em>
</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">39.372</td><td rowspan="1" colspan="1">4.40</td><td rowspan="1" colspan="1">34.110</td><td rowspan="1" colspan="1">39.515</td><td rowspan="1" colspan="1">1.60E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp107</td><td rowspan="1" colspan="1">Rv1250</td><td rowspan="1" colspan="1">
<em>Rv1250</em>
</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">50.000</td><td rowspan="1" colspan="1">3.82</td><td rowspan="1" colspan="1">45.902</td><td rowspan="1" colspan="1">54.614</td><td rowspan="1" colspan="1">2.04E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp108</td><td rowspan="1" colspan="1">Rv2282c</td><td rowspan="1" colspan="1">
<em>Rv2282c</em>
</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">54.862</td><td rowspan="1" colspan="1">3.93</td><td rowspan="1" colspan="1">54.843</td><td rowspan="1" colspan="1">72.958</td><td rowspan="1" colspan="1">3.30E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp109</td><td rowspan="1" colspan="1">Rv2074</td><td rowspan="1" colspan="1">
<em>Rv2074</em>
</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">63.704</td><td rowspan="1" colspan="1">5.16</td><td rowspan="1" colspan="1">61.332</td><td rowspan="1" colspan="1">71.002</td><td rowspan="1" colspan="1">6.80E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp110</td><td rowspan="1" colspan="1">Rv0552</td><td rowspan="1" colspan="1">
<em>Rv0552</em>
</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">67.497</td><td rowspan="1" colspan="1">5.30</td><td rowspan="1" colspan="1">63.792</td><td rowspan="1" colspan="1">67.497</td><td rowspan="1" colspan="1">3.26E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp111</td><td rowspan="1" colspan="1">Rv1009</td><td rowspan="1" colspan="1">
<em>rpfB</em>
</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">75.762</td><td rowspan="1" colspan="1">4.75</td><td rowspan="1" colspan="1">72.969</td><td rowspan="1" colspan="1">84.394</td><td rowspan="1" colspan="1">2.23E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp112</td><td rowspan="1" colspan="1">Rv0223c</td><td rowspan="1" colspan="1">
<em>Rv0223c</em>
</td><td rowspan="1" colspan="1">14</td><td rowspan="1" colspan="1">73.646</td><td rowspan="1" colspan="1">4.05</td><td rowspan="1" colspan="1">73.198</td><td rowspan="1" colspan="1">79.374</td><td rowspan="1" colspan="1">2.12E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp113</td><td rowspan="1" colspan="1">Rv3063</td><td rowspan="1" colspan="1">
<em>cstA</em>
</td><td rowspan="1" colspan="1">14</td><td rowspan="1" colspan="1">79.403</td><td rowspan="1" colspan="1">3.63</td><td rowspan="1" colspan="1">61.301</td><td rowspan="1" colspan="1">118.199</td><td rowspan="1" colspan="1">1.55E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp114</td><td rowspan="1" colspan="1">Rv1525</td><td rowspan="1" colspan="1">
<em>wbbL2</em>
</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">27.520</td><td rowspan="1" colspan="1">3.75</td><td rowspan="1" colspan="1">11.145</td><td rowspan="1" colspan="1">102.347</td><td rowspan="1" colspan="1">1.79E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp115</td><td rowspan="1" colspan="1">Rv3013</td><td rowspan="1" colspan="1">
<em>Rv3013</em>
</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">28.265</td><td rowspan="1" colspan="1">3.98</td><td rowspan="1" colspan="1">12.673</td><td rowspan="1" colspan="1">28.312</td><td rowspan="1" colspan="1">1.65E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp116</td><td rowspan="1" colspan="1">Rv3354</td><td rowspan="1" colspan="1">
<em>Rv3354</em>
</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">28.312</td><td rowspan="1" colspan="1">2.83</td><td rowspan="1" colspan="1">9.237</td><td rowspan="1" colspan="1">102.347</td><td rowspan="1" colspan="1">4.35E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp117</td><td rowspan="1" colspan="1">Rv1664</td><td rowspan="1" colspan="1">
<em>pks9</em>
</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">32.262</td><td rowspan="1" colspan="1">2.86</td><td rowspan="1" colspan="1">7.168</td><td rowspan="1" colspan="1">72.449</td><td rowspan="1" colspan="1">3.57E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp118</td><td rowspan="1" colspan="1">Rv1366</td><td rowspan="1" colspan="1">
<em>Rv1366</em>
</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">64.513</td><td rowspan="1" colspan="1">3.43</td><td rowspan="1" colspan="1">42.380</td><td rowspan="1" colspan="1">84.022</td><td rowspan="1" colspan="1">9.72E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp119</td><td rowspan="1" colspan="1">Rv3765c</td><td rowspan="1" colspan="1">
<em>tcrX</em>
</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">74.502</td><td rowspan="1" colspan="1">4.19</td><td rowspan="1" colspan="1">69.446</td><td rowspan="1" colspan="1">75.001</td><td rowspan="1" colspan="1">6.29E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp120</td><td rowspan="1" colspan="1">Rv0454</td><td rowspan="1" colspan="1">
<em>Rv0454</em>
</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">75.750</td><td rowspan="1" colspan="1">4.13</td><td rowspan="1" colspan="1">75.001</td><td rowspan="1" colspan="1">82.088</td><td rowspan="1" colspan="1">1.87E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp121</td><td rowspan="1" colspan="1">Rv2315c</td><td rowspan="1" colspan="1">
<em>Rv2315c</em>
</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">84.037</td><td rowspan="1" colspan="1">3.37</td><td rowspan="1" colspan="1">7.814</td><td rowspan="1" colspan="1">102.354</td><td rowspan="1" colspan="1">4.37E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp122</td><td rowspan="1" colspan="1">Rv0242c</td><td rowspan="1" colspan="1">
<em>fabG4</em>
</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">88.975</td><td rowspan="1" colspan="1">5.10</td><td rowspan="1" colspan="1">86.329</td><td rowspan="1" colspan="1">90.948</td><td rowspan="1" colspan="1">5.91E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp123</td><td rowspan="1" colspan="1">Rv0959</td><td rowspan="1" colspan="1">
<em>Rv0959</em>
</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">91.863</td><td rowspan="1" colspan="1">4.47</td><td rowspan="1" colspan="1">90.948</td><td rowspan="1" colspan="1">93.408</td><td rowspan="1" colspan="1">4.13E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp124</td><td rowspan="1" colspan="1">Rv1961</td><td rowspan="1" colspan="1">
<em>Rv1961</em>
</td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">95.169</td><td rowspan="1" colspan="1">4.46</td><td rowspan="1" colspan="1">94.529</td><td rowspan="1" colspan="1">96.191</td><td rowspan="1" colspan="1">5.17E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp125</td><td rowspan="1" colspan="1">Rv0043c</td><td rowspan="1" colspan="1">
<em>Rv0043c</em>
</td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">44.080</td><td rowspan="1" colspan="1">3.98</td><td rowspan="1" colspan="1">36.743</td><td rowspan="1" colspan="1">44.128</td><td rowspan="1" colspan="1">2.51E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp126</td><td rowspan="1" colspan="1">Rv1400c</td><td rowspan="1" colspan="1">
<em>lipI</em>
</td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">78.088</td><td rowspan="1" colspan="1">3.52</td><td rowspan="1" colspan="1">75.905</td><td rowspan="1" colspan="1">85.637</td><td rowspan="1" colspan="1">3.92E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp127</td><td rowspan="1" colspan="1">Rv1552</td><td rowspan="1" colspan="1">
<em>frdA</em>
</td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">92.557</td><td rowspan="1" colspan="1">3.95</td><td rowspan="1" colspan="1">74.904</td><td rowspan="1" colspan="1">96.503</td><td rowspan="1" colspan="1">3.60E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp128</td><td rowspan="1" colspan="1">Rv3058c</td><td rowspan="1" colspan="1">
<em>Rv3058c</em>
</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">55.278</td><td rowspan="1" colspan="1">4.32</td><td rowspan="1" colspan="1">6.007</td><td rowspan="1" colspan="1">76.812</td><td rowspan="1" colspan="1">2.40E&#x02212;04</td></tr><tr><td rowspan="1" colspan="1">hp129</td><td rowspan="1" colspan="1">Rv1507A</td><td rowspan="1" colspan="1">
<em>Rv1507A</em>
</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">85.527</td><td rowspan="1" colspan="1">4.44</td><td rowspan="1" colspan="1">80.477</td><td rowspan="1" colspan="1">87.364</td><td rowspan="1" colspan="1">6.91E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp130</td><td rowspan="1" colspan="1">Rv3776</td><td rowspan="1" colspan="1">
<em>Rv3776</em>
</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">94.132</td><td rowspan="1" colspan="1">4.63</td><td rowspan="1" colspan="1">91.701</td><td rowspan="1" colspan="1">94.411</td><td rowspan="1" colspan="1">1.16E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp131</td><td rowspan="1" colspan="1">Rv3495c</td><td rowspan="1" colspan="1">
<em>lprN</em>
</td><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">57.375</td><td rowspan="1" colspan="1">4.87</td><td rowspan="1" colspan="1">57.348</td><td rowspan="1" colspan="1">58.270</td><td rowspan="1" colspan="1">3.71E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp132</td><td rowspan="1" colspan="1">Rv1084</td><td rowspan="1" colspan="1">
<em>Rv1084</em>
</td><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">10.714</td><td rowspan="1" colspan="1">4.90</td><td rowspan="1" colspan="1">3.337</td><td rowspan="1" colspan="1">14.209</td><td rowspan="1" colspan="1">2.73E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp133</td><td rowspan="1" colspan="1">Rv2378c</td><td rowspan="1" colspan="1">
<em>mbtG</em>
</td><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">23.841</td><td rowspan="1" colspan="1">3.02</td><td rowspan="1" colspan="1">21.915</td><td rowspan="1" colspan="1">27.750</td><td rowspan="1" colspan="1">3.72E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp134</td><td rowspan="1" colspan="1">Rv0056</td><td rowspan="1" colspan="1">
<em>rplI</em>
</td><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">39.978</td><td rowspan="1" colspan="1">4.00</td><td rowspan="1" colspan="1">38.651</td><td rowspan="1" colspan="1">42.427</td><td rowspan="1" colspan="1">3.83E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp135</td><td rowspan="1" colspan="1">Rv3540c</td><td rowspan="1" colspan="1">
<em>ltp2</em>
</td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1">101.772</td><td rowspan="1" colspan="1">4.43</td><td rowspan="1" colspan="1">80.056</td><td rowspan="1" colspan="1">115.391</td><td rowspan="1" colspan="1">3.14E&#x02212;03</td></tr><tr><td rowspan="1" colspan="1">hp136</td><td rowspan="1" colspan="1">Rv0694</td><td rowspan="1" colspan="1">
<em>lldD1</em>
</td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1">115.365</td><td rowspan="1" colspan="1">3.91</td><td rowspan="1" colspan="1">102.867</td><td rowspan="1" colspan="1">167.395</td><td rowspan="1" colspan="1">3.70E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp137</td><td rowspan="1" colspan="1">Rv1791</td><td rowspan="1" colspan="1">
<em>PE19</em>
</td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1">119.825</td><td rowspan="1" colspan="1">4.38</td><td rowspan="1" colspan="1">115.497</td><td rowspan="1" colspan="1">128.620</td><td rowspan="1" colspan="1">2.38E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp138</td><td rowspan="1" colspan="1">Rv2733c</td><td rowspan="1" colspan="1">
<em>Rv2733c</em>
</td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1">133.591</td><td rowspan="1" colspan="1">3.40</td><td rowspan="1" colspan="1">115.471</td><td rowspan="1" colspan="1">138.012</td><td rowspan="1" colspan="1">3.90E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp139</td><td rowspan="1" colspan="1">Rv0182c</td><td rowspan="1" colspan="1">
<em>sigG</em>
</td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1">136.470</td><td rowspan="1" colspan="1">4.68</td><td rowspan="1" colspan="1">136.304</td><td rowspan="1" colspan="1">138.007</td><td rowspan="1" colspan="1">1.61E&#x02212;02</td></tr><tr><td rowspan="1" colspan="1">hp140</td><td rowspan="1" colspan="1">Rv1413</td><td rowspan="1" colspan="1">
<em>Rv1413</em>
</td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1">138.012</td><td rowspan="1" colspan="1">4.90</td><td rowspan="1" colspan="1">135.586</td><td rowspan="1" colspan="1">139.435</td><td rowspan="1" colspan="1">5.84E&#x02212;03</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140614513545936"><a target="object" rel="noopener" href="/pmc/articles/PMC10468300/table/jkad147-T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="jkad147-tblfn1"><p class="p p-first-last">&#x0201c;<em>hp</em>QTL&#x0201d; is the name for each host-pathogen QTL of genome-wide significance (<em>P</em> &#x02264; 0.05). &#x0201c;Trait&#x0201d; refers to the Rv number of the bacterial mutant whose fitness profile identified the <em>hp</em>QTL in the host genome. &#x0201c;Gene&#x0201d; is the bacterial gene annotation of the mutant if one is available on Mycobrowser. &#x0201c;Chr&#x0201d; is the host chromosome on which the QTL is located, and &#x0201c;Position (Mb)&#x0201d; is the position of the QTL in megabases. &#x0201c;LOD&#x0201d; is the maximum association score between the mutant fitness profile and the host locus. &#x0201c;Start (Mb)&#x0201d; and &#x0201c;End (Mb)&#x0201d; are the lower and upper boundaries of the Bayesian 95% confidence interval, as calculated by R/qtl2. &#x0201c;<em>P</em>-value&#x0201d; denotes the significance of the association between the bacterial mutant fitness and the host genetic locus.</p></div></div></div></article><article data-type="fig" id="ob-jkad147-F5"><div class="fm-sec"><div class="fm-flexbox"><div class="fm-citation"><div>G3 (Bethesda). 2023 Sep; 13(9): jkad147. </div><div><span class="fm-vol-iss-date">Published online 2023 Jul 5. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fg3journal%2Fjkad147" target="_blank" rel="noopener noreferrer">10.1093/g3journal/jkad147</a></span></div></div></div><div class="half_rhythm permissions"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140614519882560_cl">Copyright/License</a><span class="fm-reuse"><a href="/pmc/about/copyright/" target="_blank" rel="noopener noreferrer">Request permission to reuse</a></span><div class="hide" id="idm140614519882560_cl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2023. Published by Oxford University Press on behalf of The Genetics Society of America.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div></div></div><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="jkad147-F5"><h3>Fig. 5.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140614519872256"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10468300_jkad147f5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jkad147f5.jpg" title="Click on image to zoom" class="tileshop" data-src="/pmc/articles/PMC10468300/bin/jkad147f5.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140614519872256"><a target="object" rel="noopener" href="/pmc/articles/PMC10468300/figure/jkad147-F5/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><p>A chromosome 6 QTL hotspot highlights a genomic region controlling fitness of multiple bacterial mutants. QTL mapping of Tn mutants lacking a) <em>mak</em>, b) <em>espR</em>, c) <em>perM</em>, and d) <em>rip2</em>. After 10,000 permutation tests, the solid threshold represents <em>P</em> = 0.05, and the dashed threshold represents <em>P</em> = 0.20. e) Chromosome 6 mapping overlap of the 4 Tn mutant fitness profiles that identified the QTL hotspot. f&#x02013;i) Boxplots representing the fitness of Tn mutants lacking <em>mak</em>, <em>espR</em>, <em>perM</em>, and <em>rip2</em> within BXD mice with <em>B</em> (navy) or <em>D</em> (lilac) haplotypes at the QTL position. BXD genotypes for which a haplotype state could not be called with 95% confidence were not included.</p></div></div></article><article data-type="fig" id="ob-jkad147-F6"><div class="fm-sec"><div class="fm-flexbox"><div class="fm-citation"><div>G3 (Bethesda). 2023 Sep; 13(9): jkad147. </div><div><span class="fm-vol-iss-date">Published online 2023 Jul 5. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fg3journal%2Fjkad147" target="_blank" rel="noopener noreferrer">10.1093/g3journal/jkad147</a></span></div></div></div><div class="half_rhythm permissions"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140614524235120_cl">Copyright/License</a><span class="fm-reuse"><a href="/pmc/about/copyright/" target="_blank" rel="noopener noreferrer">Request permission to reuse</a></span><div class="hide" id="idm140614524235120_cl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2023. Published by Oxford University Press on behalf of The Genetics Society of America.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div></div></div><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="jkad147-F6"><h3>Fig. 6.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140614524228720"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10468300_jkad147f6.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jkad147f6.jpg" title="Click on image to zoom" class="tileshop" data-src="/pmc/articles/PMC10468300/bin/jkad147f6.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140614524228720"><a target="object" rel="noopener" href="/pmc/articles/PMC10468300/figure/jkad147-F6/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><p>Independence tests identify 2 putative causal variants. Scan2 independence tests from R/qtl were used to identify whether more than 1 causal host factor may underlie the chromosome 6 hotspot. The QTL mapped by a) <em>mak</em>, b) <em>espR</em>, c) <em>perM</em>, and d) <em>rip2</em> Tn mutants were remapped, incorporating the haplotype of the BXD mice at the locus of each other QTL as a covariate in the mapping analysis. The original QTL mapped by each mutant fitness profile are represented as dashed lines. The solid horizontal threshold represents <em>P</em> = 0.05 while the dashed horizontal threshold represents <em>P</em> = 0.20.</p></div></div></article><article data-type="fig" id="ob-jkad147-F7"><div class="fm-sec"><div class="fm-flexbox"><div class="fm-citation"><div>G3 (Bethesda). 2023 Sep; 13(9): jkad147. </div><div><span class="fm-vol-iss-date">Published online 2023 Jul 5. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1093%2Fg3journal%2Fjkad147" target="_blank" rel="noopener noreferrer">10.1093/g3journal/jkad147</a></span></div></div></div><div class="half_rhythm permissions"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140614506454960_cl">Copyright/License</a><span class="fm-reuse"><a href="/pmc/about/copyright/" target="_blank" rel="noopener noreferrer">Request permission to reuse</a></span><div class="hide" id="idm140614506454960_cl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; The Author(s) 2023. Published by Oxford University Press on behalf of The Genetics Society of America.</div><div class="license half_rhythm">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div></div></div><!--fig mode="anchored" f5--><div class="fig  iconblock anchored whole_rhythm" id="jkad147-F7"><h3>Fig. 7.</h3><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140614506440928"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10468300_jkad147f7.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is jkad147f7.jpg" title="Click on image to zoom" class="tileshop" data-src="/pmc/articles/PMC10468300/bin/jkad147f7.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140614506440928"><a target="object" rel="noopener" href="/pmc/articles/PMC10468300/figure/jkad147-F7/?report=objectonly">Open in a separate window</a></div><!--caption a7--><div class="caption"><p>A bioinformatic pipeline highlights putative candidate genes underlying the QTL on chromosome 6. a) A graphic summarizing the bioinformatic pipeline used to prioritize host gene candidates within each QTL. b) A heatmap representative of the per-gene outcome of 5 distinct criteria: (1) whether or not D2 possesses a missense or nonsense mutation in comparison to B6 according to the Wellcome Sanger Institute Mouse Genomes Project (Sanger MGP) (<a href="#jkad147-B1" rid="jkad147-B1" class=" bibr popnode">Adams <em>et al</em>. 2015</a>), (2) whether the gene is significantly enriched in genetically diverse mice progressively infected with <em>Mtb</em> (<a href="#jkad147-B2" rid="jkad147-B2" class=" bibr popnode">Ahmed <em>et al</em>. 2020</a>), (3) whether the gene is significantly enriched in Rhesus Macaques progressively infected with <em>Mtb</em> (<a href="#jkad147-B2" rid="jkad147-B2" class=" bibr popnode">Ahmed <em>et al</em>. 2020</a>), (4) whether the gene is significantly upregulated or downregulated in TB patient blood (<a href="#jkad147-B70" rid="jkad147-B70" class=" bibr popnode">Zak <em>et al</em>. 2016</a>), and (5) whether the gene is differentially expressed in mouse lungs across variable host genotypes, <em>Mtb</em> genotypes, and doses (<a href="#jkad147-B42" rid="jkad147-B42" class=" bibr popnode">Moreira-Teixeira <em>et al</em>. 2020</a>). If the answer is no, the space remains blank. Green indicates yes, and yellow indicates yes but cautions that the Rhesus Macaque gene is not a high confidence homolog. Only protein-coding genes with a D2 SNP (as per Sanger MGP) that met at least 1 of the criteria are included in this visual.</p></div></div></article></div></div>
  
  
      <script src="/corehtml/pmc/jatsreader/ptpmc_3.22/js/libs.min.js"> </script><script src="/corehtml/pmc/jatsreader/ptpmc_3.22/js/jr.min.js"> </script>
      
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
</body>
</html>
